\chapter{Impact of Phenylalanines Outside the Dimer Interface on
Phosphotriesterase Stability and Function}
\label{chap:dimer}
\begin{refsection}

\section{Introduction}

\subsection{Organophosphates (OPs) Poisoning and Treatments}

Organophosphates (OPs) are a synthetic class of small molecule widely used as a
diversity of pesticides and chemical weapons \cite{Perezgasga2012,Ross2013b}.
While pesticides can be absorbed through the skin or swallowed, chemical
weapons such as VX, Sarin, soman, and cyclosarin are used in war.  Adverse
health effects, such as respiratory disorders, dermal damages, neurological
deficit, or memory disruption are reported after exposure to OPs
\cite{Ross2013b}. The toxicity of OPs are related to irreversible inactivation
of acetylcholinesterase (AChE, acetycholine acetylhydrolase, E.C. 3.1.1.7)
\cite{Ross2013b}. AChE is an enzyme that degrades the neurotransmitter,
acetylcholine (ACh), at the synaptic region of nervous system. (Figure
\ref{fig:ache}) The hydrolysis of the carboxyl ester results in an acyl-enzyme
and free choline, and the acyl-enzyme undergoes nucleophilic attack with
assistance of His440. The resultant releases an acetic acid and regenerating
the free enzyme \cite{Ross2013b}.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig2_10}
    \caption[Mechanism of ACh hydrolysis catalyzed by AChE.]{Mechanism of ACh
        hydrolysis catalyzed by AChE \cite{Ross2013b}} \label{fig:ache}
\end{figure}
% --------------------------

After the ACh is hydrolyzed, the synaptic transmission would be terminated.
However, the inhibition of AChE leads to hyper-stimulation from toxic
accumulation of acetylcholine \cite{Soreq2001}. The inhibition occurs in two
steps: (i) short term reversible enzyme inactivation; (ii) slow irreversible
inhibition that generates the enzyme-inhibitor complex \cite{Ross2013b}.

\subsection{Phosphotriesterase}

Phosphotriesterase (PTE, E.C. 3.1.8.1) is an enzyme isolated from \emph{P.
dimuta} capable of detoxifying organophosphates (OPs) \cite{
Lewis1988,Chen2007a,Mulbry1989,Benning2001a,Omburo1992a,Benning1995,Naqvi2014}.
OPs, which include pesticides\cite{Amaroli2013b,Ross2013b} and chemical
warfares agents \cite{Tsai2012b,Colovic2013b}, cause hyper-stimulation inside
synapses of the nerve system and when covalently-bounded to the active site of
acetylcholinesterase (AChE) \cite{Lopez-Granero2013c}. PTE hydrolyzes OPs via a
S\textsubscript{n}2-like reaction, preventing the binding with AChE and
subsequent inactivation \cite{Ghanem2005a}. Structurally, PTE is composed of two
($\beta$/$\alpha$)\textsubscript{8} TIM-barrel subunits, each with a
metallo-active site, and functional as a dimer.

\subsection{Active Site of Phosphotriesterase (PTE)}

Various strategies have been employed to engineer PTE including rational
mutagenesis \cite{Chen-Goodspeed2001a,Jackson2009a}, computational
design \cite{Pavelka2009,Yang2014a}, directed evolution \cite{Roodveldt2005}, and
incorporation of unnatural amino acids \cite{Yang2014a,Baker2011b}. Using
site-directed mutagenesis, Raushel group studied the impacts of
individual residues in the PTE active site. They were able to alter PTE
specificity by generating single mutations G60A, I106A, F132A, and
S308A in wild-type PTE \cite{Chen-Goodspeed2001a}. Notably, The mutation of
G60A led to 2 orders of magnitude of reduced catalytic efficiency on dimethyl
and diethyl \emph{p}-nitrophenyl phosphate. The steric constraints were
relieved by the substitution with smaller size of amino acid. The results
demonstrated the strategy for modulating PTE specificity via mutations in PTE
active site. 

Solving the structure of a PTE containing both \ch{Cd} and
\ch{Zn} in the PTE structure was studied with NMR \cite{Benning2001a}. The
$\alpha$ metal is coordinated to the enzyme by His55 and His57 from the
end of $\beta$-strand 1 as well as by Asp301 from $\beta$-strand 8. The
$\beta$-site metal is coordinated through His201 and His230. In addition, the
metals are coordinated through the bridging ligands identified as a
carboxylated Lys169.

The reaction catalyzed by PTE was described as an SN2-like mechanism. The
metals in the active site are coordinated to the majority of the residues from
the C-terminal loops. The active site pocket is designated to \emph{small},
\emph{large}, and \emph{leaving group} pockets. The leaving group pocket
consists of residues including Trp131, Phe132, Phe306, and Tyr309; small group
pocket are Gly60, Ile106, Leu303, Ser308; the large pocket are Typ131, Phe132,
Phe306, Tyr309.
% --------------------------
\begin{figure}[h!] 
    \centering 
    \includegraphics[width=0.5\textwidth]{fig2_03}
    \caption[Metal
        coordination of resting state of PTE (PDB 1HZY). Bond distances from
        metal to ligand are D301--$\alpha$ = 2.2\AA, H57--$\alpha$ = 2.1\AA,
        H55--$\alpha$ = 1.8\AA, K169-–$\alpha$ = 2.1\AA, K169-–$\beta$ =
        2.0\AA, \ce{H2O}-–$\beta$ = 2.1\AA, H230-–$\beta$ = 2.1\AA,
    H201-–$\beta$ = 2.2\AA.]{Metal coordination of resting state of PTE (PDB
        1HZY). Bond distances from metal to ligand are D301--$\alpha$ = 2.2\AA,
        H57--$\alpha$ = 2.1\AA, H55--$\alpha$ = 1.8\AA, K169-–$\alpha$ =
        2.1\AA, K169-–$\beta$ = 2.0\AA,
    \ce{H2O}-–$\beta$ = 2.1\AA, H230-–$\beta$ = 2.1\AA, H201-–$\beta$ = 2.2\AA.} 
    \label{fig:pte-active-site-chap2}
\end{figure}
% --------------------------

\subsection{Stability of Phosphotriesterase (PTE)}

Our previous studies successfully utilized non-natural amino acid,
\emph{p}-fluorophenylalanine (\emph{p}FF), to improve PTE stability and
functionality \cite{Baker2011b,Yang2014a}. Baker \emph{et al.} demonstrated the
altered \emph{p}FF-PTE activity on OP and non-OP substrates.  Upon elevated
temperatures, \emph{p}FF-PTE exhibited enhanced residual activity. To further
stabilize PTE via its dimer interface, Yang \emph{et al.} removed the clash
from F104A with the computational simulation of Rosetta. The \emph{p}FF-F104A
demonstrated the extended shelf life in comparison to wild-type
PTE \cite{Yang2014a}.

To stabilize PTE, Rochu \emph{et al.} found that PTE was only in active state
with dimeric conformation \cite{Rochu2002b}. Upon thermo- or chemical-
denaturation, the PTE intermediate (I\textsubscript{2}) lost both its structure
and function \cite{Rochu2002b,Grimsley1997b}. In the previous studies, residues of
mutagenesis have focused on the binding pocket or dimer interface 
\cite{Chen-Goodspeed2001a,Rochu2002b,Grimsley1997b}. However, there are examples
from directed evolution experiments, which have led to mutants outside these
regions. Tawfik and coworkers have employed directed evolution method to
increase the soluble expression of protein. This has led to the combination of
three mutations, K185R, D208G, and R319S on the surface of the protein,
suggesting that residues beyond active site and dimer interface may play
crucial role. 

\subsection{Side-chain Effects}
\label{sec:side-chain}

Several residues across the PTE sequence have been studied for enhancement of
stability \cite{Baker2011b,Yang2014a} or changes of
selectivities\cite{Bigley2013b,Chen-Goodspeed2001a,Pavelka2009}. Rauchel group
using the alanine scanning method to locate key residues for PTE hydrolysis
efficiency \cite{Chen-Goodspeed2001a}. As Ile106, Ser306 were swapped with
alanine, they found the mutations increased substrates hydrolysis
efficiency, including R\textsubscript{P}-enantiomers containing a phenyl
substituent. Due to the sterically hindering effect, they suggested that small
pocket of PTE dictated the chiral preference for the
S\textsubscript{P}-enantiomers. Interestingly, enlarging the large pocket site
did not significantly increase the hydrolysis efficiency. However, they did
note that H254A in the large pocket sites might interacted differently via
Asp301 as they were binding with $\alpha$ metal. 

In this article, we would like to investigate the effects of phenylalanines
outside the dimer interface of phosphotriesterase. While Baker \latin{et al.}
demonstrated how unnatural amino acid (\emph{p}-fluoro-phenylalanine)
stabilized PTE, we mutate phenylalanines positions to study the individual
effect on PTE. With the comparison of paraoxon hydrolysis of F51L, F150M,
F216L, F304L, F306L, F327L, F335M, and F357L, we also report and evaluate the
performance of computational modeling tool, Rosetta, for the future screening
and analysis of PTE.

\subsection{Scope of Work}

Here, we investigate the role of the non-dimer interface phenylalanines on PTE
function and stability as well as employ Rosetta to predict how these mutations
will behave. Our studies demonstrate that three residues F304L, F327L, and
F335M are important for PTE stability and activity. Notably, F306L is
identified and confirmed by experiments to improve stability.

\section{Methods}

\subsection{General}

\emph{DpnI} and dNTP were purchased from Roche. Pfu DNA polymerase was
purchased from Thermo Scientific (Waltham, MA). All other chemicals including
\ce{NaCl}, \ce{CoCl2}, Tris-HCl, tryptone, yeast extract, paraoxon, ampicillin,
chloramphenicol, sodium phosphates monobasic, sodium phosphate dibasic,
imidazole, methanol, and  isopropyl-$\beta$-D-thiogalactopyranoside were
purchased from Sigma (St. Louis, MO) or VWR (Radnor, PA). DNA sequence was
confirmed by Eurofins MWG Operon.  96-well plates were purchased from Thermo
Fisher Scientific (Waltham, MA). FPLC column was purchased from G.E Healthcare
(Piscataway, NJ). 

\subsection{Rosetta Design of Phosphotriesterase}

A symmetric starting model of wild type PTE from the B chain of PDB structure
1HZY \cite{Benning2001a} was built using the Rosetta suite of macromolecular
modeling tools \cite{Leaver-Fay2011,Leaver-Fay2013a,Song2011,Shapovalov2011}.
Three positions in the wild-type PTE sequence were mutated (K185R, D208G, and
R319S) based on S5PTE \cite{Roodveldt2005}. Both active site \ce{Zn^{2+}} ions
were replaced with \ce{Co^{2+}} to reflect the metal used in the experimentally
produced mutants.  Distance constraints between the cobalt cations and the
coordinating residues were taken from PDB structure 3A4J \cite{Jackson2009b}.
Torsional and partial charge parameters for the non-standard carboxylated
lysine residue (Lys 169) were calculated quantum mechanically using the
HF/6-31G(d) level of theory in Gaussian09 \cite{Frisch2009a} with an overall
charge of -1.  Rotamer libraries for the carboxylated lysine were generated
with the Rosetta MakeRotLib \cite{Renfrew2012b} protocol.  Models were
constructed for each of the point mutations: F51L, F150M, F216L, F304L, F306L,
F327L, F335M, and F357L using the Rosetta fixbb (fixed backbone design)
protocol with symmetry \cite{DiMaio2011a}. \ce{Co^{2+}} coordinating residues
were held fixed to their native rotamers. To propagate point mutation effects
throughout a mutant model, the Rosetta relax protocol was used to repack and
minimize the entire PTE structure with backbone flexibility. For each point
mutant, an ensemble of 500 relaxed decoys were generated. Interatomic distances
between \ce{Co^{2+}} and coordinating residues were enforced with harmonic
constraints.  The change in stability for a mutation was calculated as the
difference between the mutant and wild type ensemble averages of the total
Rosetta score. All protocols used here included the native rotamers and extra
rotamers sampling as additional parameters. All decoys were scored using the
\emph{talaris2013} score function \cite{Leaver-Fay2013a}.

\subsection{Sequence Alignment And Site-directed Mutagenesis}

The S5-PTE\cite{Griffiths2003} was used for protein sequence alignment via
BLAST (Figure \ref{fig:pte-alignment}). Three mutations, K185R, D208G, and
R319S, were introduced into the original PTE sequence to construct S5-PTE for
this study. Site-directed mutagenesis was carried out using the pQE30-PTE
plasmid was used as described \cite{Yang2014a,Baker2011b}. The PTE variants
plasmid were prepared as the followings: F51L, forward primer
(5$'$-TCT GAA GCG GGT \emph{CTG} ACA CTG ACT CAC G-3$'$), reverse primer
(5$'$-G AGA CTT CGC CCA \emph{GAC} TGT GAC TGA GTG-3$'$). F150M, forward primer
(5$'$-TC ACA CAG TTC \emph{ATG} CTG CGT GAG ATT CAA TAT GGC-3$'$), reverse primer
(5$'$-CAT CTC CTT GAG TGT GTC AAG \emph{TAC} GAC GCA CTC TA-3$'$). F216L, forward primer
(5$'$-AG GCC GCC ATT \emph{TTA} GAG TCC GAA GG-3$'$), reverse primer
(5$'$-CGG CGG TAA \emph{AAT} CTC AGG CTT CCG A-3$'$). F304L, forward primer
(5$'$-AT GAC TGG CTG \emph{CTG} GGG TTT TCG AGC TAT GTC-3$'$), reverse primer
(5$'$-CAA AGC TTA CTG ACC GAC \emph{GAC} CCC AAA AGC TC-3$'$). F306L, forward primer
(5$'$-TGG CTG TTC GGG \emph{CTG} TCG AGC TAT GTC ACC-3$'$), reverse primer
(5$'$-CTG ACC GAC AAG CCC \emph{GAC} AGC TCG ATA CAG-3$'$). F327L, forward primer
(5$'$-AC GGG ATG GCC \emph{TTA} ATT CCA CTG AG-3$'$), reverse primer
(5$'$-CCC TAC CGG \emph{AAT} TAA GGT GAC TCT C-3$'$). F335M, forward primer
(5$'$-G AGA GTG ATC CCA \emph{CTG} CTA CGA GAG AAG G-3$'$), reverse primer
(5$'$-C TCT CAC TAG GGT \emph{GAC} GAT GCT CTC TTC C-3$'$). F357L, forward primer
(5$'$-T AAC CCG GCG CGG \emph{TTA} TTG TC ACC GAC CTT GC-3$'$), reverse primer
(5$'$-GA TTG GGC CGC GCC \emph{AAT} AAC AGT GGC TGG AAC-3$'$). The polymerase
chain reaction (PCR) parameters were set as follow for 18 cycles: initial
denaturation in \SI{95}{\celsius} for 30 seconds, sequential denaturation in
\SI{95}{\celsius} for 30 seconds, annealing in \SI{55}{\celsius} for 1 minute,
and extension in \SI{68}{\celsius} for 4 minutes. The mixture was then
incubated \SI{37}{\celsius} overnight with DpnI to digest methylated parent DNA
strands, which lack the desired mutation. DNA sequence was further confirmed by
Eurofins MWG Operon.

\subsection{Bio-synthesis of PTE And Variants}

Mutant and wild type plasmids were transformed into
chemical-competent \emph{E.  coli} phenylalanine auxotrophic strains (AF-IQ
cells) \cite{Yang2014a}. Cells were plated on agar plates containing
\SI{200}{\ug\per\mL} ampicillin, \SI{34}{\ug\per\mL} chloramphenicol. A single
colony was picked and grown in LB with \SI{200}{\ug\per\mL} ampicillin and
\SI{34}{\ug\per\mL} chloramphenicol at \SI{37}{\celsius} incubation at 300
r.p.m. for 16 hours.  Afterwards, \SI{250}{\mL} of LB medium bearing the same
antibiotics for large-scale expression was innoculated 1:50 with the overnight
culture and incubated at \SI{37}{\celsius} and 300 r.p.m. While optical density
reached 1.0 at 600 nm, the expression media were supplemented with
\SI{1}{\milli\Molar} isopropyl-$\beta$-D-thiogalactopyranoside (IPTG) to induce
protein expression.  \SI{1}{\milli\Molar} of \ce{CoCl2} was added in both pre-
and post-induction medium.  After three hours incubation at \SI{37}{\celsius}
and 300 r.p.m., the cells were harvested by an centrifuge of 4000 r.p.m.
(Beckman Coulter, Jersey City, NJ. F10 rotor) at \SI{4}{\celsius} for 15
minutes and then resuspended with \SI{20}{\milli\Molar} Tris-HCl,
\SI{500}{\milli\Molar} \ce{NaCl}, \SI{5}{\milli\Molar} imidazole, 10\% glycerol
and \SI{1}{\micro\Molar} \ce{CoCl2} (pH 8.0). Cell lysate was immediately
sonicated at 400 kJ for 2.5 minutes at \SI{4}{\celsius} (Qsonica, Newtown, CT),
and then a clarification spin was performed (20000 r.p.m, \SI{4}{\celsius}, 30
minutes).  Clarified supernatants were loaded into a \SI{5}{\mL} His Trap
column (G.E Healthcare, Piscataway, NJ) using AKTA FPLC purifier (G.E.
Healthcare, Piscataway, NJ).  Protein was eluted by using 30\% elution buffer B
(\SI{20}{\milli\Molar} Tris-HCl, \SI{500}{\milli\Molar} sodium chloride,
\SI{500}{\milli\Molar} imidazole,\SI{100}{\micro\Molar} \ce{CoCl2}, pH 8.0) at
\SI{4}{\celsius}.  The purified samples were then transferred into a 3.5K MWCO
dialysis SnakeSkin (Life Technologies, Carlsbad, CA) for buffer exchange using
\SI{12}{\L} \SI{20}{\milli\Molar} phosphate buffer (pH 8.0,
\SI{100}{\micro\Molar} \ce{CoCl2}) on a stirred plate at \SI{4}{\celsius}.
Dialyzed protein was subjected to kinetic assays immediately. The purity of
protein was determined by sodium dodecyl sulfate polyacrylamide gel
electrophoresis (SDS-PAGE).

\subsection{Enzyme Kinetics}

The protein was diluted to a final concentration of \SI{30}{\nano\Molar} in
\SI{20}{\milli\Molar} sodium phosphate (pH 8.0, \SI{100}{\micro\Molar}
\ce{CoCl2}) by using the extinction coefficient \SI{29575}{\per\Molar\per\cm}
for all proteins with Nano-Drop (Waltham, MA) \cite{Gasteiger2005, Pace1995}.
Reactions were monitored spectrophotometrically (Synergy H1, BioTek, Winooski
VT) at \SI{405}{\nm} for paraoxon (coefficient = \SI{17000}{\per\Molar\per\cm})
\cite{Baker2011b} in a 96-well plate. Reactions for paraoxon
(\SIrange{13}{104}{\micro\Molar}) was carried out in 0.2\% methanol at room
temperature.  K\textsubscript{M} and k\textsubscript{cat} values were
determined by a Lineweaver-Burk plot \cite{Baker2011b}. The below equation was used
(Eq.~\ref{eqn:MM-chap2}):
% --------------------------
\begin{equation} 
    \frac{1}{v} =
    \frac{K\textsubscript{M}}{V\textsubscript{max}}\times\frac{1}{S} +
    \frac{1}{V\textsubscript{max}} 
    \label{eqn:MM-chap2}
\end{equation}
% --------------------------

where S represents substrate concentration; K\textsubscript{M} represents the
substrate concentration at which the reaction rate is half of
V\textsubscript{max}. The data reported was the average of three trials and the
error represented the standard deviation of those trials. Residual activities
and shelf life measurements were conducted with the same batch of proteins
(\SI{30}{\nano\Molar} in \SI{20}{\milli\Molar} sodium phosphate, \SI{100}{\micro\Molar}
\ce{CoCl2}, pH 8.0). For residual activity assays, proteins at
\SI{35}{\celsius}, \SI{45}{\celsius}, and \SI{55}{\celsius} were cooled back to
room temperature for one hour and then assessed for activity on paraoxon
(\SIrange{13}{104}{\micro\Molar}, 0.2\% methanol).  Half-life experiments were
carried out using proteins (\SI{30}{\nano\Molar} in \SI{20}{\milli\Molar}
sodium phosphate, \SI{100}{\micro\Molar} \ce{CoCl2}, pH 8.0) that kept under
room temperature for
1, 2, 3, and 7 days. After incubation, activity for paraoxon
(\SIrange{13}{104}{\micro\Molar}, 0.2\% methanol) was assessed at room
temperature. 

\subsection{Thermo-stability and Secondary Structure of Phosphotriesterase}

\subsubsection{Differential Scanning Calorimetry of PTE Variants}

Differential scanning calorimetry (Nano-DSC, TA instrument, USA) was performed
by using \SI{600}{\micro\L} (\SI{0.1}{\mg\per\mL}) of protein right after
dialysis into \SI{20}{\milli\Molar} sodium phosphate buffer
(\SI{100}{\micro\Molar} \ce{CoCl2}, pH 8.0).  Measurements were conducted at a
scan rate of \SI{1}{\celsius\per\minute} from \SI{20}{\celsius} to
\SI{70}{\celsius}.  Signals was blanked with buffer under
the same condition.  The observed scan was then analyzed by using
three-scaled model in NanoAnalyze software (TA instrument, USA). Cp and
T\textsubscript{m} were determined by fitting to a three state model built in
NanoAnalyze \cite{Privalov1986}. The equation is shown below:
% --------------------------
\begin{equation} 
    C\textsubscript{p} = \kappa\textsubscript{B}
    \Bigg(\frac{\epsilon}{\kappa\textsubscript{B}T}\Bigg) \times
    \frac{e^{\beta\kappa}}{\big[e^{\beta\kappa}+1\big]^{2}}
    \label{eqn:dsc-chap2}
\end{equation}
% --------------------------

\subsubsection{Circular Dichroism}

Circular Dichroism (CD) spectra were recorded on a JASCO J-815 Spectropolarimeter
(Easton, MD) using Spectra Manager software \cite{Kataev1985}. Temperature was
controlled at \SI{25}{\celsius} using a Fisher Isotemp Model 3016S water bath.
Proteins concentrations were \SI{10}{\micro\Molar} in \SI{20}{\milli\Molar}
phosphate buffer (pH 8.0, \SI{100}{\micro\Molar} \ce{CoCl2}). The sample volume
was \SI{600}{\micro\liter}. As a blank, \SI{20}{\milli\Molar} phosphate buffer
was used. The signal was converted into mean residue molar ellipticities (deg $\times$
cm\textsuperscript{2} $\times$ dmol\textsuperscript{-1}) using the following formula
\cite{Kelly2005} (Eq.~\ref{eqn:CD-chap2}): 
% --------------------------
\begin{equation}
    θmrw = MRW(θobs) / (10 * c * l)
    \label{eqn:CD-chap2}
\end{equation}
% --------------------------

where MRW is the mean residue weight of phosphotriesterase
(\si{\gram\per\mol}), $\theta$obs is the observed ellipticities (mdeg),
\emph{l} is the path length (cm), \emph{c} is the concentration in
\SI{}{\Molar}. Spectra was recorded from \SIrange{190}{250}{\nm} with a scan
speed of \SI{1}{\nano\meter\per\minute}.  The data presented is an average of
three scans.

\section{Results}

\subsection{Variants Expression And Purification}

To explore the impact of phenylalanines outside the dimer interface on
functions and stability, eight residues were identified for mutagenesis. We
conducted a sequence alignments of the \emph{Flavobacterium} S5-PTE and
organophosphorus hydrolase (OPH) proteins (Figure \ref{fig:pte-alignment}). Of
the eight phenylalanines F51, F150, F216, F304, F306, F327, F335, and F357, two
were conserved across the OPH proteins, while the remainder were substituted
with aliphatic amino acids, such as Met and Leu, or charged residue (Arg), or
aromatic tyrosine in the homologous sequence (Figure \ref{fig:pte-alignment}).
As we were interested in probing the importance of Phe as an aromatic side
chain, we chose to substitute it with Met or Leu as they share similar
sizes and properties with Phe \cite{Richards1974,McDaniel1988}. Methionine and
leucine exhibit a van der Waals volume of \SI{124}{\angstrom}, similar to the
aromatic phenylalanine (\SI{135}{\angstrom}) without the
aromaticity \cite{Barnes2007,Richards1974}. In general, Phe was substituted by
aliphatic Met or Leu observed in the alignment.  For residues that demonstrated
homologous substitutions to charged or aromatic amino acids, we selected
leucine. This led to construction of the following eight mutations: F51L,
F150M, F216L, F304L, F306L, F327L, F335M, and F357L. 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig2_04}
    \caption[The sequences alignment for phosphotriesterase. Stains, including
        \emph{Brevundimonas.}, \emph{Flavobaterium.}, \emph{Rhizobiacae.}, and
        \emph{Photorhabdus.}, were used for alignment. Highlighted in yellow
        represent the phenylalanine residues outside the dimer interface. The
        alignment was conducted via \emph{blastp} program. Expected threshold
    was 10, and matrix was BLOSUM62.]{The sequences alignment for
        phosphotriesterase. Stains, including \emph{Brevundimonas.},
        \emph{Flavobaterium.}, \emph{Rhizobiacae.}, and \emph{Photorhabdus.},
        were used for alignment. Highlighted in yellow represent the
        phenylalanine residues outside the dimer interface. The alignment was
        conducted via \emph{blastp} program. Expected threshold was 10, and
    matrix was BLOSUM62.}
    \label{fig:pte-alignment}
\end{figure}
% --------------------------

\subsection{Rosetta Design of Phosphotriesterase Variants}

Rosetta was employed to evaluate the stability and function of the eight PTE
variants. The wild-type PTE was constructed based on the crystal structure
(1HZY). For each variant, 1000 decoys were generated for each variant, which
followed the normal distribution (Figure \ref{fig:rosetta-box-plot}). In
comparison to wild-type PTE, which possessed a Rosetta Energy Unit (REU) of
-1119.93 $\pm$ 5.42, three variants, F304L, F327L, and F335L, demonstrated
values of -1107.29 $\pm$ 5.69, -1112.77 $\pm$ 6.27, and -1111.89 $\pm$ 6.66,
respectively (Figure \ref{fig:rosetta-box-plot}). Among eight variants, only
F306L exhibited improved REU value -1121.28 $\pm$ 5.60 (Figure
\ref{fig:rosetta-box-plot}). The remaining variants, F51L, F150M, F216L, and
F357L, showed minor changes of REU scores when compared with wild-type
(|$\Delta$REU| < 2) (Figure \ref{fig:rosetta-box-plot}).
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig2_02}
    \caption[Rosetta energy units (REU) comparison of wild-type PTE and
    variants. Reported REUs are scored by using Rosetta suite. 1000 decoys are
generated for each variant.]{Rosetta energy units (REU) comparison of wild-type
PTE and variants. Reported REUs are scored by using Rosetta suite. 1000 decoys
are generated for each variant.}
    \label{fig:rosetta-box-plot}
\end{figure}
% --------------------------

\subsection{Activity of PTE Variants}

To assess function, the kinetics of PTE and its variants were determined on
paraoxon. Activity was evaluated by measuring the increase of
\emph{p}-nitrophenol after hydrolysis at 405
nm \cite{Baker2011b,Yang2014a,Carr1996b,Cho2004b}. At \SI{25}{\celsius},
wild-type PTE exhibited a k\textsubscript{cat}/K\textsubscript{M} =
\SI{170000}{\per\Molar\per\second} consistent with previously published work
(Table \ref{tab:kinetics-chap2-result}, Figure \ref{fig:activity-chart}) 
\cite{Yang2014a,Baker2011b}. As anticipated from Rosetta, F304L, F327L, F335M
demonstrated loss in activity, exhibiting 32\%, 69\%, and 60\% in
k\textsubscript{cat}/K\textsubscript{M} relative to wild-type (Table
\ref{tab:kinetics-chap2-result}, Figure \ref{fig:activity-chart}) Two variants, F216L
and F306L, demonstrated improved activity with
k\textsubscript{cat}/K\textsubscript{M} values 147\% and 142\% respectively to
wild-type PTE (Table \ref{tab:kinetics-chap2-result}, Figure
\ref{fig:activity-chart}) While F306L was predicted by Rosetta to be more
stable, F216L possessed similar REU to the wild-type PTE (Figure
\ref{fig:rosetta-box-plot}). As the Rosetta simulations reflected stability of
variants relative to wild-type PTE, residual activity experiments were carried
out.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig2_05}
    \caption[The activity comparison of PTE and the eight variants at different
        temperatures. Proteins were incubated at \SIlist{35;45;55}{\celsius}
        for one hour. After cooling down to room temperatures, samples were
        assayed with the substrate, paraoxon, at 405 nm. All
        k\textsubscript{cat}/K\textsubscript{M} values were normalized to
        wild-type PTE at \SI{25}{\celsius}.]{The activity comparison of PTE and
            the eight variants at different temperatures. Proteins were
            incubated at \SIlist{35;45;55}{\celsius} for one hour. After
            cooling down to room temperatures, samples were assayed with the
            substrate, paraoxon, at 405 nm. All
            k\textsubscript{cat}/K\textsubscript{M} values were normalized to
            wild-type PTE at \SI{25}{\celsius}.}
    \label{fig:activity-chart}
\end{figure}
% --------------------------

To determined the function at elevated temperatures, the residual activities
were assessed after one hour incubation of the proteins at
\SIlist{35;45;55}{\celsius}. As expected, the wild-type PTE illustrated a
gradual decrease in activity consistent with prior studies (Table
\ref{tab:kinetics-chap2-result}, Figure
\ref{fig:activity-chart}) \cite{Yang2014a,Baker2011b}. All the variants
demonstrated a loss in activities at elevated temperatures. Notably, the three
variants, F304L, F327L, and F335M, revealed a rapid decrease in residual
activity, especially at \SI{45}{\celsius} and \SI{55}{\celsius} (Table
\ref{tab:kinetics-chap2-result}, Figure \ref{fig:activity-chart}). At
\SI{45}{\celsius}, F327L and F335M exhibited
k\textsubscript{cat}/K\textsubscript{M} of
\SI{68000\pm12000}{\per\Molar\per\second} and
\SI{33000\pm15000}{\per\Molar\per\second} respectively, while F304L completely
lost its activity to hydrolyze paraoxon (Figure \ref{fig:activity-chart}, Table
\ref{tab:kinetics-chap2-result}). All residual activity was lost at \SI{55}{\celsius}
for both F327L and F335M (Figure \ref{fig:activity-chart}).  These thermo
active profiles affirmed the Rosetta predictions where F304L possessed a
substantial loss in residual activity at elevated temperatures (Figure
\ref{fig:rosetta-box-plot}). While both F216L and F306L exhibited improved
activity at \SI{25}{\celsius} relative to wild-type PTE, F306L revealed a
consistently better activity across all elevated temperatures (Figure
\ref{fig:activity-chart}, Table \ref{tab:kinetics-chap2-result}). This overall
improved residual activity over wild-type PTE supported Rosetta simulations
where F306L was identified as the most stable (Figure
\ref{fig:rosetta-box-plot}).
% --------------------------
\begin{table}[htbp]
    \centering
    \begin{tabular}{llllll}
    \hline
    protein                 &  & \SI{25}{\celsius} & \SI{35}{\celsius} &
    \SI{45}{\celsius} & \SI{55}{\celsius} \\ 
    \hline
    
    \multirow{2}{*}{PTE}    & k\textsubscript{cat}/K\textsubscript{M} & 1.70 $
    \pm$ 0.20 & 1.20 $\pm$ 0.23 & 1.00 $\pm$ 0.11 & 0.65 $\pm$ 0.10 \\
    & k\textsubscript{cat} & 2.3 $\pm$ 0.5 & 2.0 $\pm$ 0.7 & 1.4 $\pm$ 0.5 & 1.3
    $\pm$ 0.5 \\
    \hline
    \multirow{2}{*}{F51L}  & k\textsubscript{cat}/K\textsubscript{M} & 1.40
    $\pm$ 0.19 & 1.20 $\pm$ 0.20 & 1.10 $\pm$ 0.20 & 0.70 $\pm$ 0.09 \\
    & k\textsubscript{cat} & 3.0 $\pm$ 1.1 & 2.6 $\pm$ 0.5 & 1.2 $\pm$ 0.3 &
    0.8 $\pm$ 0.3 \\
    \hline
    \multirow{2}{*}{F150M} & k\textsubscript{cat}/K\textsubscript{M} &
    1.30 $\pm$ 0.08 & 1.20 $\pm$ 0.10 & 0.90 $\pm$ 0.08 & 0.50 $\pm$ 0.07 \\
    & k\textsubscript{cat} & 2.5 $\pm$ 1.0 & 2.0 $\pm$ 1.0 & 1.4 $\pm$ 0.3 &
    1.0 $\pm$ 0.4 \\
    \hline
    \multirow{2}{*}{F216L} & k\textsubscript{cat}/K\textsubscript{M}
    & 2.50 $\pm$ 0.13 & 1.30 $\pm$ 0.10 & 1.00 $\pm$ 0.08 & 0.90 $\pm$ 0.20 \\
    & k\textsubscript{cat} & 2.5 $\pm$ 0.6 & 2.4 $\pm$ 0.6 & 1.8 $\pm$ 0.5 &
    1.7 $\pm$ 0.6 \\
    \hline
    \multirow{2}{*}{F304L}  & k\textsubscript{cat}/K\textsubscript{M} & 0.55
    $\pm$ 0.13 & 0.50 $\pm$ 0.10 & n.a. & n.a. \\
    & k\textsubscript{cat} & 0.6 $\pm$ 0.3 & 0.2 $\pm$ 0.1& n.a. & n.a. \\
    \hline
    \multirow{2}{*}{F306L}  & k\textsubscript{cat}/K\textsubscript{M} & 2.43
    $\pm$ 0.23 & 1.72 $\pm$ 0.04 & 1.38 $\pm$ 0.36 & 0.90 $\pm$ 0.12 \\
    & k\textsubscript{cat} & 2.1 $\pm$ 0.6 & 1.8 $\pm$ 0.5 & 1.7 $\pm$ 0.5 &
    1.2 $\pm$ 0.6 \\
    \hline
    \multirow{2}{*}{F327L}  & k\textsubscript{cat}/K\textsubscript{M} & 1.17
    $\pm$ 0.13 & 1.10 $\pm$ 0.19 & 0.68 $\pm$ 0.12 & n.a. \\
    & k\textsubscript{cat} & 1.5 $\pm$ 0.6 & 1.3 $\pm$ 0.5 & 0.8 $\pm$ 0.3 &
    n.a. \\
    \hline
    \multirow{2}{*}{F335M}  & k\textsubscript{cat}/K\textsubscript{M} & 1.02
    $\pm$ 0.13 & 0.78 $\pm$ 0.10 & 0.33 $\pm$ 0.15 & n.a. \\
    & k\textsubscript{cat} & 1.3 $\pm$ 0.5 & 0.6 $\pm$ 0.5 & 0.3 $\pm$ 0.2 &
    n.a. \\
    \hline
    \multirow{2}{*}{F357L}  & k\textsubscript{cat}/K\textsubscript{M} & 1.91
    $\pm$ 0.25 & 1.50 $\pm$ 0.13 & 1.10 $\pm$ 0.09 & 0.56 $\pm$ 0.09 \\
    & k\textsubscript{cat} & 2.5 $\pm$ 0.7 & 1.8 $\pm$ 0.3 & 1.5 $\pm$ 0.5 &
    0.4 $\pm$ 0.2 \\

    \hline
    \multicolumn{6}{l}{n.a = not available; 
        k\textsubscript{cat}/K\textsubscript{M}:
        $\times$10\textsuperscript{5}\SI{}{\per\Molar\per\second};
        k\textsubscript{cat}: \SI{}{\per\second}.}            
    \end{tabular}
    \caption[Paraoxon hydrolysis efficiency summary of PTE and its variants.
    Residual activities were performed after incubation at
\SIlist{35;45;55}{\celsius}.] {Paraoxon hydrolysis efficiency summary of PTE
    and its variants. Residual activities were performed after incubation at
    \SIlist{35;45;55}{\celsius}.} \label{tab:kinetics-chap2-result}
\end{table}
% --------------------------

\subsection{Structure and Thermodynamics Stability of PTE Variants}

Far UV wavelength scans of PTE and variants were evaluated to assess the impact
of mutations on secondary structure (Figure \ref{fig:cd-chap2-result}, Table
\ref{tab:cd-chap2-result}). The wavelength scan for wild-type PTE at
\SI{25}{\celsius} exhibited a double minima at 208 and 222 nm of -811 and -923
deg$\times$cm\textsuperscript{2}$\times$dmol\textsuperscript{-1}, respectively,
consistent with previous studies \cite{Yang2014a,Baker2011b}. The variants
possessed a slight loss in structure relative to wild-type PTE with the
exception of F216L, which possessed a lightly more negative signature of -822
and -963 deg$\times$cm\textsuperscript{2}$\times$dmol\textsuperscript{-1} at
208 and 222 nm, respectively (Figure \ref{fig:cd-chap2-result}, Table
\ref{tab:cd-chap2-result}). In general, the variants demonstrated similar
conformations, indicating that the mutations did not significantly alter the
PTE structure. 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.5\textwidth]{fig2_06}
    \caption[The CD wavelength scan overlay of variants relative to wild-type
    PTE. Proteins concentrations were \SI{10}{\micro\Molar} in
\SI{20}{\milli\Molar} phosphate buffer (\SI{100}{\micro\Molar} \ce{CoCl2}, pH
8.0). Sample volume was \SI{600}{\micro\liter}.]{The CD wavelength scan overlay
    of variants relative to wild-type PTE. Proteins concentrations were
    \SI{10}{\micro\Molar} in \SI{20}{\milli\Molar} phosphate buffer
    (\SI{100}{\micro\Molar} \ce{CoCl2}, pH 8.0). Sample volume was
    \SI{600}{\micro\liter}.}
    \label{fig:cd-chap2-result}
\end{figure}
% --------------------------

% --------------------------
\begin{table}[htbp]
    \centering
    \begin{tabular}{lll}
    \hline

    protein & $\theta$ 208 nm & $\theta$ 222 nm\\
    \hline

    PTE & -811  & -923 \\
    F51L & -694 &  -814 \\
    F150M & -560 & -810 \\
    F216L & -822 & -963 \\
    F304L & -684 & -749 \\
    F306L & -719 & -787 \\
    F327L & -683 & -731 \\
    F335M & -702 & -791 \\
    F357L & -735 & -785 \\

    \hline      
    \end{tabular}
    \caption[The mean residue molar ellipticity of PTE and its variants at 208
    and 222 nm.] {The mean residue molar ellipticity of PTE and its variants at
        208 and 222 nm.} 
    \label{tab:cd-chap2-result}
\end{table}
% --------------------------

To determine the thermodynamic stability of wild-type PTE and variants, DSC was
employed.  The wild-type PTE exhibited two endothermic transitions at
\SI{50.94\pm0.03}{\celsius} (T\textsubscript{m}1) and
\SI{63.54\pm0.10}{\celsius} (T\textsubscript{m}2) as expected in affirmation of
previous studies (Figure \ref{fig:dsc-chap2-result}, Table
\ref{tab:dsc-chap2-result}) \cite{Baker2011b}. While all variants possessed two
transitions, similar to wild-type PTE with slight changes in
T\textsubscript{m}, F51L and F357L demonstrated substantial changes in
T\textsubscript{m}2 with differences of \SI{9.56}{\celsius} and
\SI{5.86}{\celsius}, respectively (Figure \ref{fig:dsc-chap2-result}). As these
proteins all possessed a 3-state transitions \cite{Yang2014a}, we chose to
focus on the first transition in order to assess stability according to
Lamazares \emph{et al.} \cite{Lamazares2015}.

% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.9\textwidth]{fig2_07}
    \caption[The DSC comparison of T\textsubscript{m}1 and T\textsubscript{m}2.
    \SI{600}{\micro\L} (\SI{0.1}{\mg\per\mL}) of protein was used for
    measurements. The measurements were conducted at a scan rate of
    \SI{1}{\celsius\per\minute} from \SI{20}{\celsius} to \SI{70}{\celsius}.
    Signals was blanked with buffer under the same condition.]{The DSC
        comparison of T\textsubscript{m}1 and T\textsubscript{m}2.
        \SI{600}{\micro\L} (\SI{0.1}{\mg\per\mL}) of protein was used for
        measurements. The measurements were conducted at a scan rate of
        \SI{1}{\celsius\per\minute} from \SI{20}{\celsius} to
        \SI{70}{\celsius}.  Signals was blanked with buffer under the same
    condition.} 
    \label{fig:dsc-chap2-result}
\end{figure}
% --------------------------

After obtaining the PTE thermogram, the analysis from 3-state model
demonstrated the enthalpies including $\Delta$H\textsubscript{1(cal)},
$\Delta$H\textsubscript{vH1}, $\Delta$H\textsubscript{2(cal)}, and
$\Delta$H\textsubscript{vH2} of wild-type PTE and variants (Table
\ref{tab:dsc-chap2-result}). Wild-type PTE demonstrated a larger unfolding enthalpy
$\Delta$H\textsubscript{vH1} at \SI{62.4}{kcal\per\mole}. Notably, in
comparison to wild-type, F304L, F306L, F327L, and F335L exhibited increased
$\Delta$H\textsubscript{vH1} at \SI{107.0}{kcal\per\mole},
\SI{96.9}{kcal\per\mole}, \SI{90.5}{kcal\per\mole}, and
\SI{87.2}{kcal\per\mole}, respectively (Table \ref{tab:dsc-chap2-result}). The
significant change of enthalpy due to a single mutation was also discovered in
Sauer group \cite{Hecht1984a}. Due to factors including hydrophobicity, packing
interactions, and introduction or removal of hydrogen bonds, the thermal
stability would depend on the contents of mutations. To obtain PTE folding
cooperativity profile from the model, the ratio of
$\Delta$H\textsubscript{(cal)}/$\Delta$H\textsubscript{vH} was calculated among
wild-type PTE and variants. Wild-type exhibited the ratio of 0.96, and all PTE
variants demonstrated the values lower than wild-type PTE. While Grimsley
\emph{et al.} demonstrated the irreversibility of PTE unfolding, the ratio of
wild-type PTE would be expected to be values smaller than
1 \cite{Grimsley1997b,Privalov2009,Honda2000,Sancho2013}. In comparison to
wild-type PTE $\Delta$H\textsubscript{(cal)}/$\Delta$H\textsubscript{vH} ratio
and unfolding profile, all variants underwent the same irreversible
process.

% --------------------------
\begin{table}[htbp]
    \centering
    \begin{tabular}{lllclclc}
    \hline

    protein & T\textsubscript{m}1 (\si{\celsius}) & T\textsubscript{m}2
    (\si{\celsius}) & $\Delta$H\textsubscript{1(cal)} &
    $\Delta$H\textsubscript{vH1} & $\Delta$H\textsubscript{2(cal)} &
    $\Delta$H\textsubscript{vH2} &
    $\Delta$H\textsubscript{1(cal)}/$\Delta$H\textsubscript{vH1} \\
    \hline

    PTE & 50.94 $\pm$ 0.03  & 63.54 $\pm$ 0.10 & 59.9 & 62.4 & 11.0 & 15.9 & 0.96\\
    F51L & 49.46 $\pm$ 0.50 & 53.98 $\pm$ 0.13 & 10.9 & 60.1 & 13.6 & 54.9 & 0.18\\
    F150M & 49.33 $\pm$ 1.14 & 60.26 $\pm$ 1.00 & 31.0 & 35.5 & 68.4 & 75.8 & 0.87\\
    F216L & 50.80 $\pm$ 0.40 & 63.90 $\pm$ 0.05 & 15.0 & 34.8 & 25.0 & 126.7 & 0.43\\
    F304L & 53.72 $\pm$ 0.04 & 62.35 $\pm$ 0.05 & 94.2 & 107.0 & 7.4 & 10.3 & 0.80\\
    F306L & 52.03 $\pm$ 0.03 & 63.13 $\pm$ 0.03 & 40.2 & 96.9 & 5.0 & 10.2 & 0.42\\
    F327L & 50.16 $\pm$ 0.03 & 62.79 $\pm$ 0.50 & 47.1 & 90.5 & 10.3 & 13.5 & 0.52\\
    F335M & 50.07 $\pm$ 0.08 & 62.03 $\pm$ 0.03 & 26.4 & 87.2 & 11.9 & 15.4 & 0.30\\
    F357L & 48.90 $\pm$ 0.15 & 57.68 $\pm$ 0.35 & 10.7 & 37.4 & 9.7 & 67.2 & 0.20\\

    \hline
    \multicolumn{8}{l}{$\Delta$H\textsubscript{1,2(cal)}: kcal/mol;
    $\Delta$H\textsubscript{vH1,2}: kcal/mol}
    \end{tabular}
    \caption[Differential scanning calorimetry results of PTE and its variants.
        The melting temperatures, enthalpies, and van't Hoff enthalpies were
    analyzed via NanoAnalyze three-scaled model (TA instrument).] {Differential
        scanning calorimetry results of PTE and its variants. The melting
        temperatures, enthalpies, and van't Hoff enthalpies were analyzed via
        NanoAnalyze three-scaled model (TA instrument).}
        \label{tab:dsc-chap2-result}
\end{table}
% --------------------------

\section{Discussion}

The importance of phenylalanines in the dimer interface resulted in the PTE
stabilization. With the buried total surface of \SI{3200}{\angstrom^{2}} in the
interface ,F65, F72, F73, F104, F132, F149, and F179 were involved in stacking
interactions \cite{Toone2009}. For example, F72 is interacted with W69 from the
same monomer as well as F149 from the opposite subunit of
PTE \cite{Benning2001a}. Another example demonstrated the removal of one of the
stacking forces, F104A, led to the significant decrease of activity on
paraoxon \cite{Yang2014a}.

However, the phenylalanines outside the dimer interface may not be necessary
for stability. In fact, Schwehm \emph{et al.} demonstrated the effects of
hydrophobicity from the solvent-exposed side chains. While it was argued that
the increased hydrophobicity at the solvent-exposed residue destabilized
proteins \cite{Pakula1990,Mollah2003,Herrmann1997}, Schwehm chose to conduct 47
phenylalanine substitutions on the surface of nuclease. Results demonstrated
the content-dependency among these substitutions. 

From our study, we affirm this as five (F51, F150, F216, F306, F357) of the
eight phenylalanine substitutions did not negatively impact stability and
function. In fact, F306L demonstrated the improved stability and activity
relative to wild-type PTE. 

Overall, Rosetta predicted that substitution of phenylalanines outside the
dimer interface maintained stability relative to wild-type PTE with the
exception of F304L, F327L, and F335M, which led to a loss in activity at
elevated temperatures. Surprisingly, Rosetta also predicted improved
functionality for the F306L variant, which also was confirmed by experimental
results. Below, we discuss these four variants in details.

% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig2_08}
    \caption[(A) RMSD overlay of F304L (red), F306L (green), and F327L (blue)
        relative to wild-type PTE. The large peak deviation corresponds to loop
        7 region. (B) Overlay of the structures of F304L (red),  F306L (green),
        and F327L (blue) relative to wild-type PTE (grey).] {(A) RMSD overlay
            of F304L (red), F306L (green), and F327L (blue) relative to
            wild-type PTE. The large peak deviation corresponds to loop 7
            region. (B) Overlay of the structures of F304L (red),  F306L
        (green), and F327L (blue) relative to wild-type PTE (grey).}
        \label{fig:rmsd}
\end{figure}
% --------------------------

\subsection{Significance of F304L, F327L, F335M}

To investigate how the mutations F304L, F327L, and F335M impacted structure,
leading to the experimental loss in function under elevated temperatures, we
assessed the RMSD of all variants relative to wild-type PTE (Figure
\ref{fig:rmsd}). The 304L variant revealed deviations in the 75-100
region ($\alpha$ helix 3 region) and 200-300 region (loop 7) when compared to
wild-type PTE (Figure \ref{fig:rmsd}). Notably, F327L exhibited a significant
disturbance in the loop 7 region; the overlay of F327L with wild-type PTE
illustrated the difference of structure in this region (Figure \ref{fig:rmsd}).
Previously, the Tawfik group demonstrated that the loop 7 region were important
for hydrolysis of wild-type PTE \cite{Afriat-Jurnou2012}. With the deletion of
loop 7, they demonstrated a 100-fold loss of PTE activity. Finally while F335M
did not demonstrate a measurable RMSD relative to wild-type PTE, the mutation
affected the overall hydrogen-bond pattern (Figure \ref{fig:hbond-plot}). It
led to significant impact on the large pocket, H254, H257, and M317, of PTE
active site (Figure \ref{fig:hbond-plot}). 

\subsection{Improved Function and Stability by F306L}

Unlike the above mentioned variants, which led to a loss in activity at
elevated temperatures, F306L exhibited improved function and stability in
affirmation of Rosetta predictions. The RMSD overlay revealed the largest
deviation in loop 7 region by F306L  (Figure \ref{fig:rmsd}). This increase of
RMSD of loop 7 allowed PTE to accommodate substrate and hold the compartment
for reaction. While F327L had also demonstrated change in this region, the
magnitude of deviation was greater F306L by 70\%.  While the loop 7 was also
significantly altered in the case of F327L, when aligning the structure, the
F327L caused loop 7 to move in the opposite direction away from the substrate,
rendering the PTE active site open, leading to the loss in function (Figure
\ref{fig:rmsd}). 

% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig2_09}
    \caption[The comparison of wild-type PTE and F335M. (A) The hydrogen-bond
    patterns of wild-type PTE and F335M across all residues. (B) The
    probabilities of hydrogen-bond of wild-type PTE and F335M. (C) Overlay of
    the structures of F335M (red) relative to wild-type PTE (grey).] {The
        comparison of wild-type PTE and F335M. (A) The hydrogen-bond patterns
        of wild-type PTE and F335M across all residues. (B) The probabilities
        of hydrogen-bond of wild-type PTE and F335M. (C) Overlay of the
        structures of F335M (red) relative to wild-type PTE (grey). }
        \label{fig:hbond-plot}
\end{figure}
% --------------------------

In support of the improvement of F306 position, Raushel group also showed that
the mutations into His and Tyr altered the hydrolysis
efficiency \cite{Pavelka2009}. Previously, Gopal \latin{et al.} generated PTE
variants, F306A and F306Y, which enhanced activity on VX \cite{Gopal2000}. In
comparison to \emph{Torpedo californica} (TAChE), PTE active site was not
originally accommodated to hydrolyze VX. With the goal of increased hydrophobic
and large binding pocket as well as improved electrostatic environment for
leaving group, the experiments demonstrated the feasibility of designed active
site at F306.  

Our results affirm the F306 affected the hydrolysis of paraoxon. The RMSD
results demonstrated the impact of F306L from neighboring residues (Figure
\ref{fig:rmsd}). Upon elevated temperatures at \SIlist{35;45;55}{\celsius},
F306L exhibited improved stability in comparison to wild-type PTE. The
thermogram also indicated the additional enthalpy of \SI{34.5}{kcal\per\mole}
at $\Delta$H\textsubscript{vH1}. Notably, Rosetta REU score of F306L also
predicted the stability in relative to wild-type and other variants. Here, we
demonstrate the successful application of Rosetta for predicting impacts of
phenylalanine mutations outside the dimer interface of wild-type PTE. Having
the cutoff value of 5 from |$\Delta$REU|, Rosetta successfully identify crucial
residues as well as variants with improved stability and function. Valuable
insight into structure is provided to further understand the experimental
results. 

%This is the end of chap2

\printbibliography[heading=subbibliography]

\end{refsection}
