\chapter{Effects of Phenylalanines Outside Dimer Interface of Phosphotriesterase}
\label{chap:dimer}
\begin{refsection}

\section{Introduction}

\subsection{Phosphotriesterase}

PTE is a homodimeric protein composed of two monomers, each of which contains a
metallo-active site. Phosphotriesterase (PTE) are enzymes, which hydrolyze
organophosphates (OPs) as well as synthetic esters (Figure
\ref{fig:pte-structure})\cite{Ghanem2005a}. The proenzyme form of PTE contains
29 amino acids signal peptide at the N-terminus. It is originally found as a
39kDa monomeric form in the solution\cite{Mulbry1989}. Later, the proenzyme of
PTE is engineered and expressed in the form of mature protein from \latin{E.
coli}. A ($\beta$/$\alpha$)\textsubscript{8} TIM-barrel structure forms the
monomeric PTE\cite{Roodveldt2005,Seibert2005}. The globular monomer is roughly
51\AA $\times$ 55\AA $\times$ 51\AA.  OPs are a synthetic class of small molecule
that irreversibly inactivate acetylcholinesterase (AChE), disrupting
neural transmission. AChE is an enzyme that degrades the neurotransmitter,
acetylcholine, at the neuromuscular junction in the cholinergic nervous system.
After the acetylcholine is hydrolyzed, the synaptic transmission would be
terminated. Inhibition of AChE lead to hyper-stimulation from toxic
accumulation of acetylcholine\cite{Soreq2001}. Army also adapted this protein
for chemical weapons neutralization \cite{Yang2014a}.

\subsection{Dimer Interface of Phosphotriesterase}

\subsection{Protein Stability}

Protein stabilities have been studied in decades. Shoichet \latin{et al.}
determined the relationship between stability and functionality of T4 lysozyme
via mutagenesis\cite{Shoichet1995}. 

\subsection{Metal Ions Effects At Active Site}

\subsection{Side-chain Effects}
\label{sec:side-chain}

\section{Methods}

\subsection{General}

\emph{DpnI} and dNTP were purchased from Roche. All other chemicals, including
\ch{NaCl}, sodium phosphates monobasic, sodium phosphate dibasic, were
purchased from Sigma or VWR. DNA primers for mutagenesis were purchased from
Fisher Scientific. DNA sequence was confirmed by Eurofins MWG Operon.  96-well
plates were purchased from Thermo Fisher Scientific (Waltham,
MA)\cite{Yang2014a}.

\subsection{Rosetta Design of Phosphotriesterase}

A symmetric starting model of wild type PTE from the B chain of PDB structure
1HZY\cite{Benning2001a} was built using the Rosetta suite of macromolecular
modeling tools\cite{Leaver-Fay2011}. Three positions in the
wild-type PTE sequence were mutated (K185R, D208G, and R319S) based on
S5PTE\cite{Roodveldt2005}. Both active site \ch{Zn^{2+}} ions were replaced
with \ch{Co^{2+}} to reflect the metal used in the experimentally produced
mutants.  Distance constraints between the cobalt cations and the coordinating
residues were taken from PDB structure 3A4J\cite{Jackson2009b}.  Torsional and
partial charge parameters for the non-standard carboxylated lysine residue (Lys
169) were calculated quantum mechanically using the HF/6-31G(d) level of theory
in Gaussian09\cite{Frisch2009} with an overall charge of -1.  Rotamer libraries
for the carboxylated lysine were generated with the Rosetta
MakeRotLib\cite{Renfrew2012b} protocol.  Models were constructed for each of
the point mutations: F51L, F150M, F216L, F304L, F306L, F327L, F335M, and F357L
using the Rosetta fixbb (fixed backbone design) protocol with
symmetry\cite{DiMaio2011a}.  \ch{Co^{2+}} coordinating residues were held fixed
to their native rotamers. To propagate point mutation effects throughout a
mutant model, the Rosetta relax protocol was used to repack and minimize the
entire PTE structure with backbone flexibility. For each point mutant, an
ensemble of 500 relaxed decoys were generated. Interatomic distances between
\ch{Co^{2+}} and coordinating residues were enforced with harmonic constraints.
The change in stability for a mutation was calculated as the difference between
the mutant and wild type ensemble averages of the total Rosetta score. All
protocols used here included the native rotamers and extra rotamers sampling as
additional parameters. All decoys were scored using the talaris2013 score
function\cite{Leaver-Fay2013a}.

\subsection{Variants of Phosphotriesterase}

Purified protein product was assayed for concentration by way of a Thermo

\subsection{Biosynthesis}

pQE30-PTE was used as described before\cite{Yang2014a}. The PTE variants plasmid
were prepared as the followings: F51L, forward primers
(5’-TCTGAAGCGGGTCTGACACTGACTCACG-3'), reverse primers
(5’-GAGACTTCGCCCAGACTGTGACTGAGTG-3’); F150M, forward primers
(5’TCACACAGTTCATGCTGCGTGAGATTCAATATGGC-3'), reverse primers
(5’-CATCTCCTTGAGTGTGTCAAGTACGACGCACTCTA-3’); F216L, forward primers
(5’-AGGCCGCCATTTTAGAGTCCGAAGG-3'), reverse primers
(5’-CGGCGGTAAAATCTCAGGCTTCCGA-3’); F304L, forward primers
(5'-ATGACTGGCTGCTGGGGTTTTCGAGCTATGTC-3'), reverse primers
(5’-CAAAGCTTACTGACCGACGACCCCAAAAGCTC-3’); F306L, forward primers
(5’-TGGCTGTTCGGGCTGTCGAGCTATGTCACC-3'), reverse primers
(5’-CTGACCGACAAGCCCGACAGCTCGATACAG-3’); F327L, forward primers
(5’-ACGGGATGGCCTTAATTCCACTGAG-3'), reverse primers
(5’-CCCTACCGGAATTAAGGTGACTCTC-3’); F335M, forward primers
(5’-GAGAGTGATCCCACTGCTACGAGAGAAGG-3'), reverse primers
(5’-GAGAGTGATCCCACTGCTACGAGAGAAGG-3’); F357L, forward primers
(5’-TAACCCGGCGCGGTTATTGTC ACCGACCTTGC-3'), reverse primers
(5’-GATTGGGCCGCGCCAATAACAGTGGCTGGAAC-3’). The polymerase chain reaction (PCR)
parameters were set as follow for 18 cycles: initial denaturation in
\SI{95}{\celsius} for 30 seconds, sequential denaturation in \SI{95}{\celsius}
for 30 seconds, annealing in \SI{55}{\celsius} for 1 minute, and extension in
\SI{68}{\celsius} for 4 minutes. The mixture was then incubated
\SI{37}{\celsius} overnight with DpnI to digest methylated parent DNA strands,
which lack the desired mutation. DNA sequence was further confirmed by Eurofins
MWG Operon. (See appendix for plasmid map)

\subsection{Protein Purification}

The purification was described previously in the section
\ref{sec:protein-expression-method}. All solutions used in the extraction and
purification of recombinant proteins \ldots delete content 5 CVs of buffer
prior to each injection.

\subsection{Enzyme Kinetics}

The protein was diluted to a final concentration of \SI{30}{\nano\Molar} in
\SI{20}{\milli\Molar} sodium phosphate (pH 8.0) by using the extinction
coefficient \SI{29280}{\per\Molar\per\cm}. Reactions were monitored
spectrophotometrically (Synergy H1, BioTek, Winooski VT) at \SI{405}{\nm} for
paraoxon (coefficient = \SI{17000}{\per\Molar\per\cm}).  Reactions for paraoxon
(\SIrange{13}{104}{\micro\Molar}) was done in 0.4\% methanol.
K\textsubscript{M} and k\textsubscript{cat} values were determined by a
Lineweaver-Burk plot.\cite{Baker2011b} The equation used is shown below
(Eq.~\ref{eqn:MM-chap2}): 
\begin{equation} 
    \frac{1}{v} =
    \frac{K\textsubscript{M}}{V\textsubscript{max}}\times\frac{1}{S} +
    \frac{1}{V\textsubscript{max}} 
    \label{eqn:MM-chap2}
\end{equation}
where S represents substrate concentration; K\textsubscript{M} represents the
substrate concentration at which the reaction rate is half of
V\textsubscript{max}. The data reported is the average of three trials and the
error represents the standard deviation of those trials.

\subsection{Thermo-stability and Secondary Structure of Phosphotriesterase}

\subsubsection{Nano-DSC}

The details are described in the section \ref{sec:dsc-method}. DSC (Nano-DSC,
TA instrument, USA) was preformed by using \SI{600}{\micro\L}
(\SI{0.1}{\mg\per\mL}) of protein right after dialysis. Measurements were
conducted at a scan rate of \SI{1}{\celsius\per\minute}. Signals was blanked with
buffer under the same condition.  The observed diagram was then analyzed by
using NanoAnalyze software.

\subsubsection{Circular Dichroism}

The details are described in the section \ref{sec:cd-method}. CD spectra were
recorded on a JASCO J-815 Spectropolarimeter (Easton, MD) using Spectra Manager
software. Temperature was controlled using a Fisher Isotemp Model 3016S water
bath. Proteins concentrations were \SI{10}{\micro\Molar} in
\SI{20}{\milli\Molar} phosphate buffer (pH 8.0). \SI{20}{\milli\Molar}
phosphate buffer was used for blanking signals. To calculate ellipticities, the
following formula was used(Eq.~\ref{eqn:CD-chap2}): 
\begin{equation}
    θmrw = MRW(θobs) / (10 * c * l)
    \label{eqn:CD-chap2}
\end{equation}
where \emph{MRW} is the mean residue weight of the specific phosphotriesterase,
θobs is the observed ellipticities (mdeg), \emph{l} is the path length (cm),
\emph{c} is the concentration in \SI{}{\micro\Molar}. Spectra was recorded from
\SIrange{190}{250}{\nm} with a scan speed of \SI{1}{\nano\meter\per\minute}.

\section{Results}

\subsection{DNA Alignments And PTE Variants}

Sequence alignments of PTE from different strains is illustrated (Figure ) and
phenylalanines outside the dimer interface of PTE are highlighted. We located
those phenylalanines positions in the phosphotriesterase and identify different
OPH from different origins, including  \latin{Brevundimonas.},
\latin{Flavobaterium.}, \latin{Rhizobiacae.}, and \latin{Photorhabdus.}. Most
of phenylalanines are conservative among these variants. Phenylalanines share
the similar sizes and properties with methionine and leucine. While the
residues, F150 and F335, are conservative among strains, we mutate these two
into methionine. Other phenylalanine residues outside dimer interface,
individually, are mutated into leucines, including F51L, F216L, F304L, F306L,
F327L, and F357L. With respect to the codon usages, we design primer sets for
each of variants of PTE. With the site-directed mutagenesis,
each variant is mutated from the parent DNA and the sequences  confirmed.

\subsection{Variants Expression And Purification}

Mutant and wild type plasmids were transformed into \latin{E. coli} phenylalanine
auxotrophic strain (AF-IQ cells) and expressed and purified with the method
discussed  previously.\cite{Yang2014a} Over-expression of proteins were
confirmed by SDS-PAGE gel. Variants were dialysed in \SI{20}{\milli\Molar}
phosphate buffer (pH 8.0) after purification and concentrations of proteins
were determined by using Nano-Drop.

\subsection{Rosetta Simulation}

% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=1.0\textwidth]{fig2_02}
    \caption[Rosetta simulation results of PTE.]{Rosetta simulation results of PTE.}
    \label{fig:rosetta-pte-chap2}
\end{figure}
% --------------------------

% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=1.0\textwidth]{fig2_01}
    \caption[RMSD comparison of PTE and its variants.]{RMSD comparison of PTE
    and its variants.} \label{fig:rmsd-pte-chap2}
\end{figure}
% --------------------------

\subsection{Variants Enzyme Kinetics}

Hydrolysis activity was evaluated by measuring the increase of
\emph{p}-nitrophenol. To assess function, we determined the Michaelis–Menten
kinetics of PTE and its variants with paraoxon. At \SI{25}{\celsius},
wild-type PTE exhibited the hydrolysis activity of
k\textsubscript{cat}/K\textsubscript{M} = \SI{170000}{\per\Molar\per\second};
F216L and F306L ware slightly higher (k\textsubscript{cat}/K\textsubscript{M} =
\SI{250000}{\per\Molar\per\second}, \SI{243000}{\per\Molar\per\second}). 

After the one hour incubation at \SIlist{35;45;55}{\celsius}, the residual
activities of wild-type and PTE variants decreased. While wild-type PTE retains
only 38\% initial activity, F51L exhibited sightly higher
k\textsubscript{cat}/K\textsubscript{M} at 50\%. We are comparing hydrolysis of
paraoxon among variants thermo-stabilities from \SI{25}{\celsius} to
\SI{55}{\celsius} with \SI{10}{\celsius} increment. The wild-type PTE lost 41\%
of its activity after \SI{45}{\celsius} incubation and only exhibited
\SI{65000}{\per\Molar\per\second} after treated at \SI{55}{\celsius}. The
results followed the pattern published before.\cite{Yang2014a} 

Previously, Gopal \latin{et al.} extensively studied the residue F306 of
PTE.\cite{Gopal2000} With the intension of enhanced activity of VX, F306A and
F306Y were introduced in PTE sequence. After we identify one of the Phe
residues outside the dimer interface, F306,  and choose to mutate it into Leu.
We compare the result of F306L with other variants. It seems that the mutation
increase the hydrolysis of paraoxon by 43\% compared with wild-type PTE at
\SI{25}{\celsius}. While the majority of variants retain their hydrolysis
efficiency, three variants, however, lose more than 30\% of
k\textsubscript{cat}/K\textsubscript{M}. F304L exhibited only 32\% of wild-type
PTE activity at \SI{25}{\celsius} (k\textsubscript{cat}/K\textsubscript{M} =
\SI{55000}{\per\Molar\per\second}. F327L and F335M, at the same time, reduce
the paraoxon hydrolysis efficiency to 69\% and 60\%, respectively.

Focusing on the thermo-stability, we then compare the variants activities at
\SIlist{35;45;55}{\celsius}. While F216L loses almost half (~48\%) of its
\SI{25}{\celsius} activity, the rest of seven variants retain more than 70\% of
their own \SI{25}{\celsius} activities. However, upon elevated temperature at
\SI{55}{\celsius}, three variants, F304L, F327L, and F335M, were inactivated in
comparison with other variants.  

% --------------------------
\begin{table}[h!]
\centering
    \begin{tabular}{llllll}
    \hline
%%
    protein                 &  & \SI{25}{\celsius} & \SI{35}{\celsius} &
    \SI{45}{\celsius} & \SI{55}{\celsius} \\ 
    \hline
%%
    \multirow{2}{*}{PTE}    & k\textsubscript{cat}/K\textsubscript{M} & 1.70 $
    \pm$ 0.13 & 0.76 $\pm$ 0.11 & 0.72 $\pm$ 0.12 & 0.46 $\pm$ 0.18 \\
    
    & k\textsubscript{cat} & 2.1 $\pm$ 0.4 & 1.3 $\pm$ 0.1 & 1.4 $\pm$ 0.1 & 0.9
    $\pm$ 0.1 \\
%%
    \multirow{2}{*}{F51L}  & k\textsubscript{cat}/K\textsubscript{M} & 3.27
    $\pm$ 0.11 & 2.42 $\pm$ 0.10 & 1.84 $\pm$ 0.21 & 0.80 $\pm$ 0.09 \\ 
    
    & k\textsubscript{cat} & 6.0 $\pm$ 1.1 & 5.6 $\pm$ 0.1 & 4.0 $\pm$ 1.1 &
    2.0 $\pm$ 0.9 \\
%%
    \multirow{2}{*}{F150M} & k\textsubscript{cat}/K\textsubscript{M} &
    0.23 $\pm$ 0.04 & 0.21 $\pm$ 0.03 & n.a & n.a \\ 
    
    & k\textsubscript{cat} & 0.01 $\pm$ 0.0 & 0.1 $\pm$ 0.0 & n.a & n.a \\
%%
    \multirow{2}{*}{F216L} & k\textsubscript{cat}/K\textsubscript{M}
    & 2.23 $\pm$ 0.15 & 1.94 $\pm$ 0.18 & 1.49 $\pm$ 0.20 & 1.11 $\pm$ 0.09 \\
    & k\textsubscript{cat} & 3.3 $\pm$ 0.3 & 3.3 $\pm$ 0.6 & 2.6 $\pm$ 1.0 &
    2.0 $\pm$ 0.7 \\
%%
    \multirow{2}{*}{F304L}  & k\textsubscript{cat}/K\textsubscript{M} & 3.27
    $\pm$ 0.11 & 2.42 $\pm$ 0.10 & 1.84 $\pm$ 0.21 & 0.80 $\pm$ 0.09 \\ 
    
    & k\textsubscript{cat} & 6.0 $\pm$ 1.1 & 5.6 $\pm$ 0.1 & 4.0 $\pm$ 1.1 &
    2.0 $\pm$ 0.9 \\
%%
    \multirow{2}{*}{F306L}  & k\textsubscript{cat}/K\textsubscript{M} & 3.27
    $\pm$ 0.11 & 2.42 $\pm$ 0.10 & 1.84 $\pm$ 0.21 & 0.80 $\pm$ 0.09 \\ 
    
    & k\textsubscript{cat} & 6.0 $\pm$ 1.1 & 5.6 $\pm$ 0.1 & 4.0 $\pm$ 1.1 &
    2.0 $\pm$ 0.9 \\
%%
    \multirow{2}{*}{F327L}  & k\textsubscript{cat}/K\textsubscript{M} & 3.27
    $\pm$ 0.11 & 2.42 $\pm$ 0.10 & 1.84 $\pm$ 0.21 & 0.80 $\pm$ 0.09 \\ 
    
    & k\textsubscript{cat} & 6.0 $\pm$ 1.1 & 5.6 $\pm$ 0.1 & 4.0 $\pm$ 1.1 &
    2.0 $\pm$ 0.9 \\
%%
    \multirow{2}{*}{F335L}  & k\textsubscript{cat}/K\textsubscript{M} & 3.27
    $\pm$ 0.11 & 2.42 $\pm$ 0.10 & 1.84 $\pm$ 0.21 & 0.80 $\pm$ 0.09 \\ 
    
    & k\textsubscript{cat} & 6.0 $\pm$ 1.1 & 5.6 $\pm$ 0.1 & 4.0 $\pm$ 1.1 &
    2.0 $\pm$ 0.9 \\
%%
    \multirow{2}{*}{F357L}  & k\textsubscript{cat}/K\textsubscript{M} & 3.27
    $\pm$ 0.11 & 2.42 $\pm$ 0.10 & 1.84 $\pm$ 0.21 & 0.80 $\pm$ 0.09 \\ 
    
    & k\textsubscript{cat} & 6.0 $\pm$ 1.1 & 5.6 $\pm$ 0.1 & 4.0 $\pm$ 1.1 &
    2.0 $\pm$ 0.9 \\

    \hline
    \multicolumn{6}{l}{n.a = not available; 
        k\textsubscript{cat}/K\textsubscript{M}:
        $\times$10\textsuperscript{5}\SI{}{\per\Molar\per\second};
        k\textsubscript{cat}: \SI{}{\per\second}.}            
    \end{tabular}
    \caption[Paraoxon hydrolysis efficiency summary of PTE, F104A,
    \emph{p}FF-PTE, and \emph{p}FF-F104A. Residual activities were preformed
after incubation at \SIlist{35;45;55}{\celsius}.]{Paraoxon hydrolysis
    efficiency summary of PTE, F104A, \emph{p}FF-PTE, and \emph{p}FF-F104A.
    Residual activities were preformed after incubation at
    \SIlist{35;45;55}{\celsius}.} 
    \label{tab:kinetics-chap2-result}
\end{table}
% --------------------------

\subsection{Thermo-stability and CD of PTE Variants}

\section{Discussion}

\printbibliography[heading=subbibliography]

\end{refsection}
