\chapter{Impact of Phenylalanines Outside the Dimer Interface on
Phosphotriesterase Stability and Function}
\label{chap:dimer}
\begin{refsection}

\section{Introduction}

\subsection{Organophosphates (OPs) Poisoning and Treatments}

Organophosphates (OPs) are a synthetic class of small molecules widely used in
pesticides (Figure \ref{fig:pesticides}) and chemical weapons
\cite{Perezgasga2012,Ross2013b} (Figure \ref{fig:chem-weapon}). In general,
pesticides such as paraoxon, parathion, diazinon, and chlorpyrifos can be
absorbed through the skin or swallowed. Chemical weapons such as VX, sarin,
tabun, soman, and cyclosarin used in war can be readily aerosolized, and the
primary entry is the respiratory system \cite{Colovic2013b} (Figure
\ref{fig:chem-weapon}).  Adverse health effects, such as respiratory disorders,
dermal damages, neurological deficit, or memory disruption are reported after
exposure to OPs \cite{Ross2013b,Colovic2013b}.  The toxicity of OPs are related
to irreversible inactivation of acetylcholinesterase (AChE, acetycholine
acetylhydrolase, E.C.  3.1.1.7) \cite{Ross2013b}. 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig2_22}
    \caption[Examples of OP pesticides: methyl parathion, methyl paraoxon,
    malaoxon, malathion, azinphos methyl, diazinon, chlorpyrifos, dichlorvos,
and fenitrothion.] {Examples of OP pesticides: methyl parathion, methyl
    paraoxon, malaoxon, malathion, azinphos methyl, diazinon, chlorpyrifos,
    dichlorvos, and fenitrothion \cite{Colovic2013b}.} \label{fig:pesticides}
\end{figure}
% --------------------------
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig2_20}
    \caption[Examples of chemical weapon agents: VX, sarin, tabun, soman, and
    cyclosarin.] {Examples of chemical weapon agents: VX, sarin, tabun, soman,
        and cyclosarin \cite{Colovic2013b}.} \label{fig:chem-weapon}
\end{figure}
% --------------------------

AChE is an enzyme that degrades the neurotransmitter, acetylcholine (ACh), at
the synaptic region of the nervous system. The hydrolysis
of the carboxyl ester results in an acyl-enzyme and free choline, and the
acyl-enzyme undergoes nucleophilic attack with assistance of H440 (Figure
\ref{fig:ache}). After release of choline, acetylated S200 is then converted
back to serine through release of an acetic acid, regenerating the free enzyme
\cite{Ross2013b}. After ACh is hydrolyzed, synaptic transmission is terminated.
However, the inhibition of AChE leads to hyper-stimulation from toxic
accumulation of ACh \cite{Soreq2001}. The inhibition occurs in two steps: (1)
short-term reversible enzyme inactivation and (2) slow irreversible inhibition
that generates an enzyme-inhibitor complex (Figure \ref{fig:op-ache})
\cite{Ross2013b}.

% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig2_10}
    \caption[Mechanism of acetylcholine (ACh) hydrolysis catalyzed by
        acetycholine acetylhydrolase (AChE). Choline (R-OH) is formed after the
    hydrolysis.] {Mechanism of acetylcholine (ACh) hydrolysis catalyzed by
        acetycholine acetylhydrolase (AChE). Choline (R-OH) is formed after the
        hydrolysis \cite{Ross2013b}.} \label{fig:ache}
\end{figure}
% --------------------------
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.9\textwidth]{fig2_21}
    \caption[Mechanism of AChE inhibition induced by OPs. EOH: serine side
    chain from AChE enzyme (E).]{Mechanism of AChE inhibition induced by OPs.
        EOH: serine side chain from AChE enzyme (E) \cite{Colovic2013b}.}
    \label{fig:op-ache}
\end{figure}
% --------------------------

Currently, multiple treatments for OP-induced intoxication have been developed
\cite{Colovic2013b} including: resuscitation \cite{MahdiBalaliMood2012}, oxygen
supply \cite{MahdiBalaliMood2012}, or the administration of symptomatic drugs
\cite{Colovic2013b,Robenshtok2002,Marrs2003}. Symptomatic drugs are
administered to reduce the symptoms from  OP-induced intoxication
\cite{Colovic2013b}. For example, atropine (Figure \ref{fig:op-drug}) reduces
the effects of the accumulated ACh on receptors by acting as a competitive
agonist of acetylcholine receptors \cite{Colovic2013b,Robenshtok2002}. The
other frequently used symptomatic treatment is anticonvulsives (diazepam,
Figure \ref{fig:op-drug}), which reacts at $\gamma$-aminobutyric acid A
(GABA-A) receptor \cite{Marrs2003}, leading to an increase of chloride ions
across the neuronal cell membrane and a decrease of convulsion. 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.5\textwidth]{fig2_11}
    \caption[Frequently used drugs for symptomatic treatment of OP
    intoxication: atropine and diazepam.]{Frequently used drugs for symptomatic
    treatment of OP intoxication: atropine and diazepam.} \label{fig:op-drug}
\end{figure}
% --------------------------

Other treatments, such as pralidoxime, methoxime, trimedoxime, obidoxime, and
asoxime, reactivate AChE by cleaving OP moiety from the active site (Figure
\ref{fig:oxime-exapmle}) \cite{Colovic2013b}. The hydroxyiminomethyl moiety of
these oxime reactivators undergoes a nucleophilic attack toward the OP of the
phosphorylated AChE (Figure \ref{fig:oxime}), reversing the inhibition from
OPs.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig2_23}
    \caption[Examples of oxime reactivators: pralidoxime, asoxime, obidoxime,
    trimedoxime, methoxime.]{Examples of oxime reactivators: pralidoxime,
        asoxime, obidoxime, trimedoxime, methoxime \cite{Colovic2013b}.}
        \label{fig:oxime-exapmle}
\end{figure}
% --------------------------
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig2_12}
    \caption[Regeneration of inhibited AChE activity by oxime
    reactivators.]{Regeneration of inhibited AChE activity by oxime
        reactivators \cite{Colovic2013b}.}
        \label{fig:oxime}
\end{figure}
% --------------------------

These examples employ small molecules to address OP toxicity. However, in order to
remediate OP, stoichiometric amounts of the drugs are required. In addition,
adverse reactions have also been reported due to treatment using such drugs
\cite{Robenshtok2002,MahdiBalaliMood2012a}. For example, the usage of atropine
may lead to tachycardia, hypertension, or mydriasis \cite{Robenshtok2002}.  To
avoid these limitations, the use of an enzymatic approach has been developed as
an alternative strategy
\cite{Tsai2010a,Cherny2013,Chen-Goodspeed2001a,Rochu2002b,Grimsley1997b,Roodveldt2005,Griffiths2003c}.

\subsection{Paraoxonase (PON)}

Serum paraoxonase (PON1, A esterase, EC 3.1.1.2) is a mammalian enzyme with
hydrolase activity toward OPs \cite{Khersonsky2005}. PON1 usually resides on
high density cholesterol (HDL) and is also involved in the prevention of
atherosclerosis \cite{Khersonsky2005}. The Tawfik group have demonstrated that the
primary activity of PONs serves as a lactonase with promiscuous paraoxonase
activities \cite{Khersonsky2005,Gupta2011}. To overcome the slow rates of
hydrolysis of cyclosarin (GF) and soman (GD), Gupta \latin{et al.} have demonstrated
that PON1 4E9 variant is capable of GF hydrolysis through directed evolution
\cite{Gupta2011}. Using mammalian serum paraoxonase PON1 (rePON1) as a starting point,
four rounds of mutagenesis and screening has been performed, leading to the final
construct of 4E9 (L69G, S111T, H115W, H134R, F222S, T332S) \cite{Gupta2011}.
Randomly picked colonies are individually grown in 96-well plates. After
incubation, the cells are collected and lysed with lysozyme. Using GF, the
initial rate of hydrolysis is measured for evaluation of mutants.  The 4E9
mutant exhibits a 135-fold increase of activity toward GF relative to
wild-type rePON1 (Figure \ref{fig:paraoxonase}).
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig2_29}
    \caption[AChE activity was assayed after the incubation of GF (40 nM) and
    4E9 (L69G, S111T, H115W, H134R, F222S, T332S), 3D8 (L69G, H115W, H134R,
    M196V, F222S, T332S), 3B3 (N41D, S110P, L240S, H243R, F264L, N324D, T332A)
    and  mammalian serum paraoxonase PON1 (rePON1) at the concentrations
noted.]{AChE activity was assayed after the incubation of GF (40 nM) and 4E9
    (L69G, S111T, H115W, H134R, F222S, T332S), 3D8 (L69G, H115W, H134R, M196V,
    F222S, T332S), 3B3 (N41D, S110P, L240S, H243R, F264L, N324D, T332A) and
    mammalian serum paraoxonase PON1 (rePON1) at the concentrations noted
    \cite{Gupta2011}.} \label{fig:paraoxonase}
\end{figure}
% --------------------------

\subsection{The Family of Organophosphorus Hydrolase (OPH)}

Organophosphate hydrolase (OPH, E.C. 3.1.8.1) refers to those enzymes that
hydrolyze paraoxon and other P-esters (P-O bonds) substrates efficiently
\cite{Hanusa2011}. The gene has been originally identified from two soil bacteria,
\emph{Flavobacterium} and \emph{Pseudomonas} \cite{Harper1988,McDaniel1988a}.
Using ion-exchange chromatography, Brown \latin{et al.} isolates the gene
from \emph{Flavobacterium} bacteria that has been identified in soil
\cite{Brown1980}. In addition, \emph{Flavobacterium} OPH shows maximum
activity between pH 8-10 after the purification (Figure \ref{fig:first-oph}). 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.6\textwidth]{fig2_24}
    \caption[Effect of pH on OPH activity. The initial rate of hydrolysis of
    paraoxon, 17.3 pM  (closed circle) and parathion, 15.4 pM (open circle) was
    determined in \SI{50}{\milli\Molar} glycine-NaOH buffer at various pH
    values.  Each point is the mean of three determinations.] {Effect of pH on
        OPH activity.  The initial rate of hydrolysis of paraoxon, 17.3 pM
        (closed circle) and parathion, 15.4 pM (open circle) was determined in
        \SI{50}{\milli\Molar} glycine-NaOH buffer at various pH values. Each
        point is the mean of three determinations \cite{Brown1980}.}
    \label{fig:first-oph}
\end{figure}
% --------------------------

The gene for OPH (organophosphate-degrading; \emph{opd})
codes for a pro-enzyme containing a leader peptide that attached to the
membrane \cite{Hanusa2011}. Serdar \latin{et al.} have demonstrated that OPH is
synthesized as a 365 amino acids precursor \cite{Serdar1989}. When the 29 amino
acids leader sequence is removed, the recombinant enzyme is found as a
soluble, mature enzyme that maintains activity (Figure \ref{fig:pte-precusor}). 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig2_25}
    \caption[Nucleotide sequence of the 0.60 kb PstI fragment of pCMS1
    containing a portion of parathion hydrolase (OPH) gene. The sequences were
determined in both directions. The experimentally determined N-terminal amino
acid sequences (precursor) are underlined.] {Nucleotide sequence of the 0.60 kb
    PstI fragment of pCMS1 containing a portion of parathion hydrolase (OPH)
    gene. The sequences were determined in both directions. The experimentally
    determined N-terminal amino acid sequences (precursor) are underlined
    \cite{Serdar1989}.}
    \label{fig:pte-precusor}
\end{figure}
% --------------------------

Other species, including \emph{Agrobacterium radiobacter} and
\emph{Sulfolobus}, have been found to possess a related OPH capable of
hydrolyzing phosphotriesters \cite{Horne2002,Porzio2007}. Horne \latin{et al.}
have found that the gene from \emph{Agrobacterium} is more than 90\% identical
to PTE, exhibiting high OPH activity \cite{Horne2002}. While the
\emph{Sulfolobus} proteins shares 34\% identity with PTE, it exhibits
relatively little OPH activity \cite{Porzio2007}.

\subsection{Phosphotriesterase (PTE)}

As described in the previous chapter, phosphotriesterase (PTE, E.C. 3.1.8.1) is
an enzyme isolated from \emph{Pseudomonas dimuta} capable of enzymatically
detoxifying OPs \cite{
Lewis1988,Chen2007a,Mulbry1989,Benning2001a,Omburo1992a,Benning1995,Naqvi2014}.
PTE hydrolyzes OPs via a SN\textsubscript{2}-like reaction (Figure
\ref{fig:pte-mechanism}), preventing the binding with AChE and subsequent
inactivation \cite{Ghanem2005a}. The resulting metabolites are more soluble,
and therefore can be eliminated via urine \cite{Colovic2013b}. Various
strategies have been employed to engineer PTE including rational mutagenesis
\cite{Chen-Goodspeed2001a,Jackson2009a}, computational design
\cite{Pavelka2009,Yang2014a}, directed evolution \cite{Roodveldt2005}, and
incorporation of fluorinated amino acids (Chapter \ref{chap:uaa}, Section
\ref{sec:uaa-intro}) \cite{Yang2014a,Baker2011b}. 

\subsection{Active Site of Phosphotriesterase (PTE)}

PTE containing \ch{Cd^{2+}} or \ch{Zn^{2+}} in the active site have been
investigated via NMR \cite{Benning2001a}. The $\alpha$ metal is coordinated to
the enzyme by H55 and H57 from the end of $\beta$-strand 1 as well as by D301
from $\beta$-strand 8 (Figure \ref{fig:pte-active-site-chap2}). The
$\beta$-site metal is coordinated through H201 and H230. In addition, the
metals are coordinated through the bridging carboxylated K169 (Figure
\ref{fig:pte-active-site-chap2}). The structural geometry has bond distances of
D301--$\alpha$ = 2.2\AA, H57--$\alpha$ = 2.1\AA, H55--$\alpha$ = 1.8\AA,
K169-–$\alpha$ = 2.1\AA, K169-–$\beta$ = 2.0\AA, \ce{H2O}-–$\beta$ = 2.1\AA,
H230-–$\beta$ = 2.1\AA, and H201-–$\beta$ = 2.2\AA\xspace \cite{Hanusa2011}.
% --------------------------
\begin{figure}[htbp] 
    \centering 
    \includegraphics[width=0.5\textwidth]{fig2_03}
    \caption[Metal
        coordination of resting state of PTE (PDB 1HZY). Bond distances from
        metal to ligand are D301--$\alpha$ = 2.2\AA, H57--$\alpha$ = 2.1\AA,
        H55--$\alpha$ = 1.8\AA, K169-–$\alpha$ = 2.1\AA, K169-–$\beta$ =
        2.0\AA, \ce{H2O}-–$\beta$ = 2.1\AA, H230-–$\beta$ = 2.1\AA,
    H201-–$\beta$ = 2.2\AA.]{Metal coordination of resting state of PTE (PDB
        1HZY). Bond distances from metal to ligand are D301--$\alpha$ = 2.2\AA,
        H57--$\alpha$ = 2.1\AA, H55--$\alpha$ = 1.8\AA, K169-–$\alpha$ =
        2.1\AA, K169-–$\beta$ = 2.0\AA,
        \ce{H2O}-–$\beta$ = 2.1\AA, H230-–$\beta$ = 2.1\AA, H201-–$\beta$ =
        2.2\AA\xspace \cite{Hanusa2011}.} \label{fig:pte-active-site-chap2}
\end{figure}
% --------------------------

The metals in the active site are coordinated to the majority of the residues
from the C-terminal end of PTE. As described in Chapter 1, the active site
region is designated into \emph{small}, \emph{large}, and \emph{leaving group}
pockets (Figure \ref{fig:pte-structure} C). Below I discuss in detail each
pocket and their roles in function.

\subsubsection{Small Pocket of PTE Active Site}

The small group pocket consists of residues G60, I106, L303, and S308
\cite{Hanusa2011} (Figure \ref{fig:small-pocket}). Since a variety of OP
analogues can be accommodated by PTE \cite{Hong1999}, the Raushel group have
investigated the impact of individual residues in the PTE active site by using
site-directed mutagenesis \cite{Chen-Goodspeed2001a}. The specificity of
wild-type PTE and mutants have been characterized with 16 organophosphate
triesters containing \emph{p}-nitrophenol as the leaving group (Figure
\ref{fig:op-library}). 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig2_19}
    \caption[The small pocket of PTE active site, including G60A, I106, L303,
    and S308A (PDB 1HZY). The image is generated using UCSF Chimera.] {The
        small pocket of PTE active site, including G60A, I106, L303, and S308A
        (PDB 1HZY). The image is generated using UCSF Chimera.}
        \label{fig:small-pocket}
\end{figure}
% --------------------------
% --------------------------
\begin{table}[htbp]
    \centering
    \caption[Values of k\textsubscript{cat}/K\textsubscript{M}
    (M\textsuperscript{-1} s\textsuperscript{-1}) for the Hydrolysis of
    phosphotriesters by PTE. The values are obtained by a fit of the data at pH
    9.0 and \SI{25}{\celsius}. PTE concentration is \SI{20}{\nano\Molar} in 50
    mM HEPES buffer, pH 8.5.] {Values of
        k\textsubscript{cat}/K\textsubscript{M} (M\textsuperscript{-1}
        s\textsuperscript{-1}) for the Hydrolysis of phosphotriesters by PTE.
        The values are obtained by a fit of the data at pH 9.0 and
        \SI{25}{\celsius}. PTE concentration is \SI{20}{\nano\Molar} in 50 mM
        HEPES buffer, pH 8.5.}
    \begin{tabular}{lllll}
    \hline

    protein & I & V & VIII & X \\
    \hline

    wild-type PTE & 1.2e7 & 6.4e7 & 4.8e6 & 1.7e7 \\
    G60A & 1.0e7 & 4.2e7 & 1.9e5 & 9.6e4 \\

    \hline  
    \end{tabular}
    \label{tab:pte-g60a}
\end{table}
% --------------------------

Variant G60A exhibited a 340-fold decrease in k\textsubscript{cat} for
substrate (X), but an only 2-fold reduction for substrate (VIII) relative to
that of the wild-type enzyme (Table \ref{tab:pte-g60a}). However, the kinetic
constants of G60A with substrate (I) and (V) were identical to wild-type PTE
\cite{Chen-Goodspeed2001a} (Table \ref{tab:pte-g60a}). Notably, the mutation of
G60A led to 2 orders of magnitude of reduced catalytic efficiency on dimethyl
and diethyl \emph{p}-nitrophenyl phosphate (Table \ref{tab:pte-g60a}). The steric
constraints were relieved by the substitution with smaller size of amino acid. 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.4\textwidth]{fig2_13}
    \caption[Structures of the organophosphates used in the substrate
    library.]{Structures of the organophosphates used in the substrate library
        \cite{Chen-Goodspeed2001a}.}
        \label{fig:op-library}
\end{figure}
% --------------------------

To further demonstrate the significance of residue G60, Tsai \latin{et al.}
investigated the hydrolysis rate of G60A toward racemic sarin (GB)
\cite{Tsai2010}. It exhibited a similar k\textsubscript{cat}/K\textsubscript{M}
value as wild-type PTE for the first phase, and a 5-fold slower for the second
phase \cite{Tsai2010} (Figure \ref{fig:g60a-gb}). Using gas chromatographic
analysis, Tsai \latin{et al.} revealed that the faster phase corresponded to
the degradation of R\textsubscript{P} enantiomers, followed by
S\textsubscript{P} enantiomers degradation in the second phase \cite{Tsai2010}.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.5\textwidth]{fig2_14}
    \caption[Hydrolysis of \SI{250}{\micro\Molar} racemic GB followed by ITC.
        All reactions were initiated by injection of \SI{5}{\micro\liter} of GB
        into a volume of \SI{200}{\micro\liter}. (A) Hydrolysis of racemic GB by
        \SI{30}{\nano\Molar} wild-type PTE as a function of time. The data are
        fit to single exponential (solid-line); (B) Hydrolysis of GB by
        \SI{60}{\nano\Molar} G60A as a function of time. The data are fit to
    the sum of two exponentials (solid-line).]{Hydrolysis of
        \SI{250}{\micro\Molar} racemic GB followed by ITC.  All reactions were
        initiated by injection of \SI{5}{\micro\liter} of GB into a volume
        of \SI{200}{\micro\liter}. (A) Hydrolysis of racemic GB by
        \SI{30}{\nano\Molar} wild-type PTE as a function of time. The data are
        fit to single exponential (solid-line); (B) Hydrolysis of GB by
        \SI{60}{\nano\Molar} G60A as a function of time. The data are fit to
        the sum of two exponentials (solid-line) \cite{Tsai2010}.}
    \label{fig:g60a-gb}
\end{figure}
% --------------------------

In addition, variant S308A enlarged the size of the small pocket of PTE active site
\cite{Chen-Goodspeed2001a}. Among the 16 substrates (Figure
\ref{fig:op-library}), S308A exhibited increased activity up to 9-fold for
the R\textsubscript{P}-enantiomers of (IV), (VII), and (IX) (Table
\ref{tab:pte-s308a}). However, for chiral substrates (II), (III), and (VI),
S308A displayed minimal effects \cite{Chen-Goodspeed2001a,Jeong2014a}. This
improvement was also discovered in I106A, while catalytic constants for the
R\textsubscript{P}-enantiomers were increased \cite{Chen-Goodspeed2001a}.
% --------------------------
\begin{table}[htbp]
    \centering
    \caption[Values of k\textsubscript{cat}/K\textsubscript{M}
    (M\textsuperscript{-1} s\textsuperscript{-1}) for the Hydrolysis of
    phosphotriesters by S308A. The values are obtained by a fit of the data at pH
    9.0 and \SI{25}{\celsius}. PTE concentration is \SI{20}{\nano\Molar} in 50
    mM HEPES buffer, pH 8.5.] {Values of
        k\textsubscript{cat}/K\textsubscript{M} (M\textsuperscript{-1}
        s\textsuperscript{-1}) for the Hydrolysis of phosphotriesters by S308A.
        The values are obtained by a fit of the data at pH 9.0 and
        \SI{25}{\celsius}. PTE concentration is \SI{20}{\nano\Molar} in 50 mM
        HEPES buffer, pH 8.5.}
    \begin{tabular}{lllllll}
    \hline

    protein & (Rp)-IV & (Sp)-IV & (Rp)-VII & (Sp)-VII & (Rp)-IX & (Sp)-IX \\
    \hline

    wild-type PTE & 1.0e6 & 9.3e7 & 3.7e6 & 7.6e7 & 5.2e6 & 1.8e8 \\
    S308A & 3.6e6 & 5.1e7 & 3.4e7 & 1.6e8 & 1.7e6 & 2.3e7 \\

    \hline  
    \end{tabular}
    \label{tab:pte-s308a}
\end{table}
% --------------------------

\subsubsection{Large Pocket of PTE Active Site}

The large group pocket consists of residues H254, H257, L271, and M317
\cite{Hanusa2011} (Figure \ref{fig:large-pocket}). In comparison with small
pocket of active site, the enlargement of these residues through mutation
(H254A, H257A, L271A, or M317A) have demonstrate a relatively small effect on
either the R\textsubscript{P}- or S\textsubscript{P}-enantiomers
\cite{Chen-Goodspeed2001a}. While the Raushel group have investigated the
hydrolysis on the same set of 16 organophosphate triesters (Figure
\ref{fig:op-library}), reductions have been limited to 1 order of magnitude
\cite{Chen-Goodspeed2001a}.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig2_26}
    \caption[The large pocket of PTE active site, including H254A, H257A,
    L271A, and M317A (PDB 1HZY). The image is generated using UCSF Chimera.]
    {The large pocket of PTE active site, including H254A, H257A, L271A, and
        M317A (PDB 1HZY). The image is generated using UCSF Chimera.}
    \label{fig:large-pocket}
\end{figure}
% --------------------------

In follow-up, Raushel and coworkers chose to mutate H254 through site-directed
mutagenesis in order to investigate hydrogen bonding at the active site
\cite{Aubert2004b} (Figure \ref{fig:large-pocket}). After H254 was mutated to
alanine and asparagine, the hydrolysis of paraoxon decreased by 1-2 orders of
magnitude. In addition, the hydrolysis efficiency of H254A and H254N
toward diethyl \emph{p}-chlorophenyl phosphate were increased by 6 and 3-fold,
respectively (Table \ref{tab:GF}).
% --------------------------
\begin{table}[htbp]
    \centering
    \caption[Values of k\textsubscript{cat}/K\textsubscript{M}
        (M\textsuperscript{-1} s\textsuperscript{-1}) for the hydrolysis of
    \emph{p}-chlorophenyl phosphate by PTE. The kinetic parameters are
determined in \SI{50}{\milli\Molar} HEPES buffer (pH 9).] {Values of
    k\textsubscript{cat}/K\textsubscript{M} (M\textsuperscript{-1}
    s\textsuperscript{-1}) for the hydrolysis of \emph{p}-chlorophenyl
    phosphate by PTE. The kinetic parameters are determined in
    \SI{50}{\milli\Molar} HEPES buffer (pH 9).}
    \begin{tabular}{ll}
    \hline

    protein & k\textsubscript{cat}/K\textsubscript{M} (M\textsuperscript{-1}
    s\textsuperscript{-1})  \\ 
    \hline

    wild-type PTE & 220 \\
    H254A & 1400 \\
    H254N & 640 \\

    \hline  
    \end{tabular}
    \label{tab:GF}
\end{table}
% --------------------------

To investigate the role of large pocket of PTE, Bigley \latin{et
al.} have mutated H254 and H257 via error-PCR and screened the hydrolysis
efficiency toward VX \cite{Tsai2010,Bigley2013}. After construction of
the library, variant QF (H254Q/H257F) is isolated, demonstrating 100-fold
improvement of k\textsubscript{cat}/K\textsubscript{M} against the chiral
centers in VX \cite{Tsai2010} (Figure \ref{fig:pte-vx}, Table
\ref{tab:qf-pte}).
% --------------------------
\begin{table}[htbp]
    \centering
    \caption[Values of k\textsubscript{cat}/K\textsubscript{M}
    (M\textsuperscript{-1} s\textsuperscript{-1}) for the Hydrolysis of VX by
    PTE. The values are obtained by a fit of the data at pH 9.0 and
    \SI{30}{\celsius}. PTE concentration is \SI{20}{\nano\Molar} in 50 mM
HEPES, pH 8.0, \SI{100}{\micro\Molar} \ch{CoCl2}.] {Values of
    k\textsubscript{cat}/K\textsubscript{M} (M\textsuperscript{-1}
    s\textsuperscript{-1}) for the Hydrolysis of VX by PTE. The values are
    obtained by a fit of the data at pH 9.0 and \SI{30}{\celsius}. PTE
    concentration is \SI{20}{\nano\Molar} in 50 mM HEPES, pH 8.0,
    \SI{100}{\micro\Molar} \ch{CoCl2} \cite{Bigley2013}.}
    \begin{tabular}{lll}
    \hline

    protein & k\textsubscript{cat}/K\textsubscript{M} & stereochemical preference \\
    \hline

    wild-type PTE & 3.0e2 & 1:1 \\
    QF & 3.0e4 & 12:1 (Sp) \\

    \hline  
    \end{tabular} 
    \label{tab:qf-pte}
\end{table}
% --------------------------
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig2_15}
    \caption[Representative time courses for the complete hydrolysis of
    \SI{160}{\micro\Molar} racemic VX by (a) QF (\SI{850}{\nano\Molar}); (b)
wild-type PTE (\SI{27}{\micro\Molar})]{Representative time courses for the
    complete hydrolysis of \SI{160}{\micro\Molar} racemic VX by (a) QF
    (\SI{850}{\nano\Molar}); (b) wild-type PTE (\SI{27}{\micro\Molar})
    \cite{Bigley2013}.}
    \label{fig:pte-vx}
\end{figure}
% --------------------------

\subsubsection{Leaving Group of PTE Active Site}

The leaving group pocket of PTE active site consists of the aromatic residues
W131, F132, F306, and Y309 \cite{Hanusa2011} (Figure \ref{fig:leaving-group}).
In the PTE structure, these residues interact with the leaving group of the
substrate. Raushel and coworkers have demonstrated that the F132A mutant
exhibited a more than 5-fold increase in k\textsubscript{cat} for the
R\textsubscript{P}-enantiomers toward (IV), (VII), and (IX) from those 16
substrates \cite{Chen-Goodspeed2001a} (Figure \ref{fig:op-library}). Variant
Y309A has demonstrated reductions in k\textsubscript{cat}/K\textsubscript{M} up
to 27-fold in comparison to the wild-type PTE \cite{Chen-Goodspeed2001a}, while
F306A resulted in decreases in the kinetic constants by 1 to 3
orders of magnitude \cite{Chen-Goodspeed2001a} (Table \ref{tab:pte-leaving}). To
investigate the impact of F306, Gopal \latin{et al.} have introduced F306Y,
which displayed roughly a 1.3-fold increased activity toward demeton-S-methyl
in comparison of almost abolished activity by F306A \cite{Gopal2000} (Table
\ref{tab:pte-leaving}). 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.5\textwidth]{fig2_27}
    \caption[The leaving group pocket of PTE active site, including F132A,
    Y309A, and F306A (PDB 1HZY). The image is generated using UCSF Chimera.]
    {The leaving group pocket of PTE active site, including  F132A, Y309A, and
        F306A (PDB 1HZY). The image is generated using UCSF Chimera.}
    \label{fig:leaving-group}
\end{figure}
% --------------------------
% --------------------------
\begin{table}[htbp]
    \centering
    \caption[Values of k\textsubscript{cat}/K\textsubscript{M}
    (M\textsuperscript{-1} s\textsuperscript{-1}) for the Hydrolysis of
    phosphotriesters by F132A, F306A, and Y309A. The values are obtained by a
    fit of the data at pH 9.0 and \SI{25}{\celsius}. PTE concentration is
    \SI{20}{\nano\Molar} in 50 mM HEPES buffer, pH 8.5.] {Values of
        k\textsubscript{cat}/K\textsubscript{M} (M\textsuperscript{-1}
        s\textsuperscript{-1}) for the Hydrolysis of phosphotriesters by F132A,
        F306A, and Y309A.  The values are obtained by a fit of the data at pH
        9.0 and \SI{25}{\celsius}. PTE concentration is \SI{20}{\nano\Molar} in
    50 mM HEPES buffer, pH 8.5.}
    \begin{tabular}{lllllll}
    \hline

    protein & (Rp)-IV & (Sp)-IV & (Rp)-VII & (Sp)-VII & (Rp)-IX & (Sp)-IX \\
    \hline

    wild-type PTE & 1.0e6 & 9.3e7 & 3.7e6 & 7.6e7 & 5.2e6 & 1.8e8 \\
    F132A & 1.3e7 & 6.1e7 & 1.8e7 & 5.0e7 & 2.3e7 & 1.2e8 \\
    F306A & 3.2e4 & 5.1e6 & 7.2e5 & 1.3e7 & 1.5e4 & 1.5e6 \\
    Y309A & 8.7e5 & 8.7e6 & 1.2e6 & 9.8e6 & 2.3e5 & 8.1e6 \\

    \hline  
    \end{tabular} 
    \label{tab:pte-leaving}
\end{table}
% --------------------------

These are multiple mutants that demonstrate enhanced activity or selectivity.
Overall, several groups were able to alter PTE specificity by generating
mutations G60A, F132A, H254Q/H257F, F306A, or Y309A, in wild-type PTE
\cite{Chen-Goodspeed2001a,Hanusa2011,Aubert2004b,Chen-Goodspeed2001a,Bigley2013}.
These results demonstrated the strategy for modulating PTE specificity via
mutations in the active site. However, due to the limitation of rational
design, alternative strategies have been developed to accommodate multiple
mutations PTE.

\subsection{Computational Approach for PTE Design}

Computational calculation or simulation have been also developed to generate
multiple mutations in PTE \cite{Tsai2010a,Cherny2013} Tsai \latin{et al.}
have demonstrated that two PTE mutants, GWT and YT, hydrolyzes the Sp
enantiomer of GB more efficiently via molecular simulation \cite{Tsai2010a}.
Molecular dynamics (MD) is a simulation that investigates the physical
movements of atoms and molecules. The atoms and molecules are allowed to
interact for a designated period of time, leading to the dynamical evolution
\cite{Gordon1999b}.  MD trajectories are used on these structures via the
Assisted Model Building with Energy Refinement (AMBER 9) suite of programs
\cite{Tsai2010a}. Both GWT and YT exhibited Rp/Sp ratio of 1/33 and 1/44,
respectively, while wild-type PTE showed a ratio of 22 \cite{Tsai2010a} (Figure
\ref{fig:gwt-gb}). 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.65\textwidth]{fig2_36}
    \caption[Bar graph illustrating the values of
    k\textsubscript{cat}/K\textsubscript{M} for the Rp and Sp enantiomers of
GB.]{Bar graph illustrating the values of
    k\textsubscript{cat}/K\textsubscript{M} for the Rp and Sp enantiomers of GB
    \cite{Tsai2010a}.} \label{fig:gwt-gb}
\end{figure}
% --------------------------

\subsection{Directed Evolution of PTE}

In the previous studies, residues for mutagenesis have focused on the binding
pocket or dimer interface \cite{Chen-Goodspeed2001a,Rochu2002b,Grimsley1997b}.
However, there are examples from directed evolution experiments, which have led
to mutants outside these regions. Tawfik and coworkers have employed directed
evolution to enhance the esterase activity of protein by applying a
layer of soft agar supplemented with 2-naphthyl acetate (2-NA, 0.5 mM) as the
substrate and Fast Red (\SI{1.3}{\mg\per\mL}) as indicator (Figure
\ref{fig:s5}). Roughly 10000 colonies have been screened, and the 150 positive
resultants transferred to liquid medium for assays.  This has led to
the combination of three mutations, K185R, D208G, and R319S on the surface of
the protein, suggesting that residues beyond active site and dimer interface
may play crucial role \cite{Roodveldt2005}.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig2_18}
    \caption[Evolution of higher functional expression of PTE. The paraoxonase
    activity per mg of cells measured in crude lysates (bars, left ordinate)
--- normalized according to total protein expression for each clone --- and
percentage of protein found in the supernatant, as determined by densitometry
from SDS-PAGE gel (10\%)  (circles, right ordinate).]{Evolution of higher
    functional expression of PTE. The paraoxonase activity per mg of cells
    measured in crude lysates (bars, left ordinate) --- normalized according to
    total protein expression for each clone --- and percentage of protein found
    in the supernatant, as determined by densitometry from SDS-PAGE gel (10\%)
    (circles, right ordinate) \cite{Roodveldt2005}.}
    \label{fig:s5}
\end{figure}
% --------------------------

To screen PTE mutations, Griffiths \latin{et al.} have developed an \emph{in
vitro} compartmentalization (IVC) using water-in-oil emulsions
\cite{Griffiths2003c} (Figure \ref{fig:ivc}). First, a gene library encoding
protein variants, each with a common epitope tag, is linked to
streptavidin-coated beads carrying antibodies that bind the epitope tag
\cite{Griffiths2003c} (Figure \ref{fig:ivc} A). Afterwards, in a compartment
containing a gene encoding an enzyme, the products of reaction are attached to
the bead, and an anti-product antibodies can be employed to detect product
formation. The resulting IVC emulsions are analyzed by flow cytometry (Figure
\ref{fig:ivc} B). 

A library of \SI{3.4e7} mutated phosphotriesterase genes were
screened, and the percentage of clones with detectable activity using flow
cytometry reached 14\% by the sixth round of selection \cite{Griffiths2003c}.
The variant h5 containing I106T and F132L at the PTE active site demonstrated a
3.8-fold increase of hydrolysis rate and a
k\textsubscript{cat}/K\textsubscript{M} of \SI{1.8e8}{\per\Molar\per\second}.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig2_30}
    \caption[Creation of microbead-display libraries and selection for
    catalysis by compartmentalization. (A) Creation of microbead-display
libraries. A repertoire of genes encoding protein variants, each with a
common epitope tag, is linked to streptavidin-coated beads carrying antibodies
that bind the epitope tag at, on average, less than one gene per bead (1).
The beads are compartmentalized in a water-in-oil emulsion to give, on average,
less than one bead per compartment (2), and transcribed and translated in vitro
in the compartments. Consequently, in each compartment, the translated protein
(10-100 copies) becomes attached to the gene that encodes it via the bead (3).
The emulsion is broken (4) and the microbeads carrying the display library
are isolated (5). (B) Enzyme selection by compartmentalization.
Microbead-display libraries are compartmentalized in a water-in-oil emulsion
(1) and a soluble substrate attached to caged-biotin is added. The substrate is
converted to product only in compartments containing beads displaying active
enzymes (2). The emulsion is then irradiated to uncage the biotin (3). In a
compartment containing a gene encoding an enzyme, the product becomes attached
to the gene via the bead (4). In other compartments, in which the genes do not
encode an active enzyme, the intact substrate becomes attached to the gene. The
emulsion is broken (5) and the beads are incubated with anti-product antibodies
(6). Product-coated beads can then be enriched (together with the genes
attached to them) either by affinity purification or, after reacting with a
fluorescent labelled antibody, by flow cytometry.]{Creation of
    microbead-display libraries and selection for catalysis by
    compartmentalization. (A) Creation of microbead-display libraries. A
    repertoire of genes encoding protein variants, each with a common epitope
    tag, is linked to streptavidin-coated beads carrying antibodies that bind
    the epitope tag at, on average, less than one gene per bead (1).  The beads
    are compartmentalized in a water-in-oil emulsion to give, on average, less
    than one bead per compartment (2), and transcribed and translated in vitro
    in the compartments. Consequently, in each compartment, the translated
    protein (10-100 copies) becomes attached to the gene that encodes it via
    the bead (3).  The emulsion is broken (4) and the microbeads carrying the
    display library are isolated (5). (B) Enzyme selection by
    compartmentalization.  Microbead-display libraries are compartmentalized in
    a water-in-oil emulsion (1) and a soluble substrate attached to
    caged-biotin is added. The substrate is converted to product only in
    compartments containing beads displaying active enzymes (2). The emulsion
    is then irradiated to uncage the biotin (3). In a compartment containing a
    gene encoding an enzyme, the product becomes attached to the gene via the
    bead (4). In other compartments, in which the genes do not encode an active
    enzyme, the intact substrate becomes attached to the gene. The emulsion is
    broken (5) and the beads are incubated with anti-product antibodies (6).
    Product-coated beads can then be enriched (together with the genes attached
    to them) either by affinity purification or, after reacting with a
    fluorescent labelled antibody, by flow cytometry \cite{Griffiths2003c}.}
    \label{fig:ivc}
\end{figure}
% --------------------------

Using directed evolution, Tsai \latin{et al.} screened a \num{6e5} PTE library
and identified variants that demonstrated enhanced catalytic efficiency toward
cyclosarin analog (Sp-5) \cite{Tsai2012b} (Figure \ref{fig:gwt}). Using
GWT-PTE (H254G, H257W, L303T) as a template, a library was transformed into
\emph{E. coli} and plated. After transferring to a phosphate-free plate for
screening, \SI{0.5}{\milli\Molar} IPTG was added in the presence of
\SI{1}{\milli\Molar} Sp-5. The colonies with sizes larger than background were
selected. Using Sp-5 as the substrate, GWT-f3 and GWT-f1 variants demonstrated a
2000-fold and 6000-fold increase, respectively, relative to wild-type PTE
\cite{Tsai2012b} (Table \ref{tab:gwt}).  The mutant GWT-f3 contained three
additional changes in the amino acid sequence: I106C, F132I, and L271I, while
GWT-f1 contained H254G, H257W, L303T, M317L, K185R, and I274N \cite{Tsai2012b}. 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.9\textwidth]{fig2_32}
    \caption[(A) Screening of the randomized library using GWT as the parental
        template with Sp-5. The light green and red bars represent the relative
        catalytic activities of the GWT-f1 and GWT-f3 mutants, respectively.
        (B) Chemical structure of Sp-5.]{(A) Screening of the randomized library
            using GWT as the parental template with Sp-5. The light green and
            red bars represent the relative catalytic activities of the GWT-f1
            and GWT-f3 mutants, respectively.  (B) Chemical structure of Sp-5
            \cite{Tsai2012b}.}
    \label{fig:gwt}
\end{figure}
% --------------------------
% --------------------------
\begin{table}[htbp]
    \centering
    \caption[Values of k\textsubscript{cat}/K\textsubscript{M}
        (M\textsuperscript{-1} s\textsuperscript{-1}) for the hydrolysis of
    cyclosarin analog Sp-5 by PTE-GWT. The kinetic parameters are determined in \SI{50}{\milli\Molar} HEPES buffer (pH 9).] {Values of k\textsubscript{cat}/K\textsubscript{M}
        (M\textsuperscript{-1} s\textsuperscript{-1}) for the hydrolysis of
    cyclosarin analog Sp-5 by PTE-GWT. The kinetic parameters are determined in \SI{50}{\milli\Molar} HEPES buffer (pH 9) \cite{Tsai2012b}.}
    \begin{tabular}{llll}
    \hline

    wild-type PTE & GWT & GWT-f1 & GWT-f3 \\ 
    2.1e1 & 2.8e4 & 3.9e4 & 1.2e5 \\

    \hline  
    \end{tabular}
    \label{tab:gwt}
\end{table}
% --------------------------

Kaltenbach \latin{et al.} also demonstrated the multi-site mutations of
PTE through directed evolution \cite{Kaltenbach2015}. To investigate the
hydrolysis of 2-naphthyl hexanoate (2NH) and paraoxon, a PTE library was
generate for screening (Figure \ref{fig:2NH}). A screening of arylesterase
activity was performed on agar plates in the presence of a mixture of the
substrate 2NH and Fast Red (Figure \ref{fig:2NH}). Upon hydrolysis of 2NH, Fast
Red formed a red complex with the naphtholate \cite{Kaltenbach2015}. The other
screening used a fluorogenic phosphotriester as a surrogate for paraoxon. The
screening demonstrated that a 1000-fold loss in PTE activity, accompanied the
functional transition to an arylester hydrolysis \cite{Kaltenbach2015}
(Figure \ref{fig:2NH} B).
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.6\textwidth]{fig2_33}
    \caption[Activity and sequence changes of PTE over the evolution. (A)
    Overview of the experimental evolution. Libraries were generated and
    transformed into \emph{E. coli}. Proteins were expressed and screened for
    paraoxon and/or 2NH hydrolysis. Several thousand variants were screened per
    round. (B) Activity changes during the forward (screening for arylesterase
    hydrolysis) and reverse evolution (screening for re-increase in
phosphotriesterase hydrolysis).]{Activity and sequence changes of PTE over the
    evolution. (A) Overview of the experimental evolution. Libraries were
    generated and transformed into \emph{E. coli}. Proteins were expressed and
    screened for paraoxon and/or 2NH hydrolysis. Several thousand variants were
    screened per round. AE: specialized arylesterase. (B) Activity changes
    during the forward (screening for arylesterase hydrolysis) and reverse
    evolution (screening for re-increase in phosphotriesterase hydrolysis)
    \cite{Kaltenbach2015}} \label{fig:2NH}
\end{figure}
% --------------------------

\subsection{Stability of Phosphotriesterase (PTE)}

Structurally, PTE is composed of two ($\beta$/$\alpha$)\textsubscript{8}
TIM-barrel subunits, each with a metallo-active site and functional as a dimer
(Figure \ref{fig:pte-structure}). Our previous studies successfully integrated the
non-natural amino acid, \emph{p}-fluorophenylalanine (\emph{p}FF), via
residue-specific incorporation to improve PTE stability and functionality
\cite{Baker2011b,Yang2014a} (Figure \ref{fig:PJB}). Baker \emph{et al.} have
demonstrated altered \emph{p}FF-PTE activity on OP and non-OP substrates.
Upon elevated temperatures, \emph{p}FF-PTE exhibited enhanced residual
activity. To further stabilize PTE via its dimer interface, Yang \emph{et al.}
removed the clash from F104A by employing Rosetta; the \emph{p}FF-F104A
demonstrated the extended shelf life in comparison to wild-type PTE
\cite{Yang2014a} (Chapter \ref{chap:uaa}, Figure \ref{fig:kinetics-fig}).

To stabilize PTE, Grimsley \emph{et al.} found that PTE was only in the active
state as a dimer \cite{Grimsley1997b}. Upon thermo or chemical denaturation,
the PTE intermediate (I\textsubscript{2}) lost both its structure and function
\cite{Rochu2002b,Grimsley1997b} (Figure \ref{fig:pte-unfold}). The unfolding
was investigated by comparing the changes in the intrinsic tryptophan
fluorescence at 320 nm and the enzymatic activity, using urea as denaturant
(Figure \ref{fig:pte-unfold}). The midpoint of the denaturation curve occurred
at about \SI{3.77}{\Molar} urea. The intermediate was not active as PTE started
to unfold (Figure \ref{fig:pte-unfold}).
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.5\textwidth]{fig2_17}
    \caption[Relationship between equilibrium unfolding followed by
    fluorescence and enzymatic activity. Urea-induced unfolding of PTE
(\SI{125}{\micro\gram\per\mL}) was measured by fluorescence emission at 320
nm with excitation at 278 nm and by enzymatic activity (open circle). The
enzymatic activity was obtained from a least-squares fit of a two-state
unfolding transition.]{Relationship between equilibrium unfolding followed by
    fluorescence and enzymatic activity. Urea-induced unfolding of PTE
    (\SI{125}{\micro\gram\per\mL}) was measured by fluorescence emission (b) at
    320 nm with excitation at 278 nm and by enzymatic activity (open circle).
    The enzymatic activity was obtained from a least-squares fit of a two-state
    unfolding transition \cite{Grimsley1997b}.}
    \label{fig:pte-unfold}
\end{figure}
% --------------------------

\subsection{Scope of Work}

Here, we investigate the role of the non-dimer interface phenylalanines on PTE
function and stability as well as employ Rosetta to predict how these mutations
will behave. Our studies demonstrate that three residues F304L, F327L, and
F335M are important for PTE stability and activity. Notably, F306L is
identified and confirmed by experiments to improve stability even under
elevated temperatures.

\section{Methods}

\subsection{General}

\emph{DpnI} and dNTP were purchased from Roche. Pfu DNA polymerase was
purchased from Thermo Scientific (Waltham, MA). All other chemicals including
\ce{NaCl}, \ce{CoCl2}, Tris-HCl, tryptone, sodium dodecyl sulfate,
polyacrylamide, glycerol, methanol, yeast extract, paraoxon, ampicillin,
chloramphenicol, sodium phosphate monobasic, sodium phosphate dibasic,
imidazole, methanol, and  isopropyl-$\beta$-D-thiogalactopyranoside were
purchased from Sigma (St. Louis, MO) or VWR (Radnor, PA). DNA sequence was
confirmed by Eurofins MWG Operon.  96-well plates were purchased from Thermo
Fisher Scientific (Waltham, MA). FPLC column was purchased from G.E Healthcare
(Piscataway, NJ). 

\subsection{Rosetta Design of Phosphotriesterase}

A symmetric starting model of wild type PTE from the B chain of PDB structure
1HZY \cite{Benning2001a} was built using the Rosetta suite of macromolecular
modeling tools \cite{Leaver-Fay2011,Leaver-Fay2013a,Song2011,Shapovalov2011}.
Three positions in the wild-type PTE sequence were mutated (K185R, D208G, and
R319S) based on S5PTE \cite{Roodveldt2005}. Both active site \ce{Zn^{2+}} ions
were replaced with \ce{Co^{2+}} to reflect the metal used in the experimentally
produced mutants.  Distance constraints between the cobalt cations and the
coordinating residues were taken from PDB structure 3A4J \cite{Jackson2009b}.
Torsional and partial charge parameters for the non-standard carboxylated
lysine residue (Lys 169) were calculated quantum mechanically using the
HF/6-31G(d) level of theory in Gaussian09 \cite{Frisch2009a} with an overall
charge of -1.  Rotamer libraries for the carboxylated lysine were generated
with the Rosetta MakeRotLib \cite{Renfrew2012b} protocol.  Models were
constructed for each of the point mutations: F51L, F150M, F216L, F304L, F306L,
F327L, F335M, and F357L using the Rosetta fixbb (fixed backbone design)
protocol with symmetry \cite{DiMaio2011a}. \ce{Co^{2+}} coordinating residues
were held fixed to their native rotamers. To propagate point mutation effects
throughout a mutant model, the Rosetta relax protocol was used to repack and
minimize the entire PTE structure with backbone flexibility. For each point
mutant, an ensemble of 500 relaxed decoys were generated. Interatomic distances
between \ce{Co^{2+}} and coordinating residues were enforced with harmonic
constraints.  The change in stability for a mutation was calculated as the
difference between the mutant and wild type ensemble averages of the total
Rosetta score. All protocols used here included the native rotamers and extra
rotamers sampling as additional parameters. All decoys were scored using the
\emph{talaris2013} score function \cite{Leaver-Fay2013a}.

\subsection{Sequence Alignment And Site-directed Mutagenesis}

The S5-PTE \cite{Griffiths2003} was used for protein sequence alignment via
BLAST (Figure \ref{fig:pte-alignment}) \cite{Altschul1990a}. Three mutations,
K185R, D208G, and R319S, were introduced into the original PTE sequence to
construct S5-PTE for this study. Site-directed mutagenesis was carried out
using the pQE30-PTE plasmid as described \cite{Yang2014a,Baker2011b}. The PTE
variants were prepared using the following primers, with the mutations
underlined: F51L forward primer (5$'$-TCT GAA GCG GGT \underline{CTG} ACA CTG
ACT CAC G-3$'$), F51L reverse primer (5$'$-G AGA CTT CGC CCA \underline{GAC}
TGT GAC TGA GTG-3$'$).  F150M forward primer (5$'$-TC ACA CAG TTC
\underline{ATG} CTG CGT GAG ATT CAA TAT GGC-3$'$),  F150M reverse primer
(5$'$-CAT CTC CTT GAG TGT GTC AAG \underline{CAT} GAC GCA CTC TA-3$'$). F216L
forward primer (5$'$-AG GCC GCC ATT \underline{TTA} GAG TCC GAA GG-3$'$), F216L
reverse primer (5$'$-CGG CGG TAA \underline{AAT} CTC AGG CTT CCG A-3$'$).
F304L forward primer (5$'$-AT GAC TGG CTG \underline{CTG} GGG TTT TCG AGC TAT
GTC-3$'$), F304L reverse primer (5$'$-CAA AGC TTA CTG ACC GAC \underline{GAC}
CCC AAA AGC TC-3$'$). F306L forward primer (5$'$-TGG CTG TTC GGG
\underline{CTG} TCG AGC TAT GTC ACC-3$'$), F306L reverse primer (5$'$-CTG ACC
GAC AAG CCC \underline{GAC} AGC TCG ATA CAG-3$'$). F327L forward primer
(5$'$-AC GGG ATG GCC \underline{TTA} ATT CCA CTG AG-3$'$), F327L reverse primer
(5$'$-CCC TAC CGG \underline{AAT} TAA GGT GAC TCT C-3$'$).  F335M forward
primer (5$'$-G AGA GTG ATC CCA \underline{ATG} CTA CGA GAG AAG G-3$'$), F335M
reverse primer (5$'$-C TCT CAC TAG GGT \underline{CAT} GAT GCT CTC TTC C-3$'$).
F357L forward primer (5$'$-T AAC CCG GCG CGG \underline{TTA} TTG TC ACC GAC CTT
GC-3$'$), F357L reverse primer (5$'$-GA TTG GGC CGC GCC \underline{AAT} AAC AGT
GGC TGG AAC-3$'$). The polymerase chain reaction (PCR) parameters were set as
follow for 18 cycles: initial denaturation in \SI{95}{\celsius} for 30 seconds,
sequential denaturation in \SI{95}{\celsius} for 30 seconds, annealing in
\SI{55}{\celsius} for 1 minute, and extension in \SI{68}{\celsius} for 4
minutes. The mixture was then incubated \SI{37}{\celsius} overnight with
\SI{1}{\micro\L} DpnI to digest methylated parent DNA strands, which lack the
desired mutation. DNA sequence was further confirmed by Eurofins MWG Operon
(See Appendix for plasmid maps of each variant).

\subsection{Bio-synthesis of PTE And Variants}

Mutant and wild type plasmids were transformed into chemical-competent \emph{E.
coli} phenylalanine auxotrophic strains (AF-IQ cells) \cite{Yang2014a}.
\SI{100}{\micro\gram} plasmid DNA was added into \SI{1}{\mL} of AF-IQ cells,
incubated on ice for 10 minutes.  After heating the cells to \SI{37}{\celsius}
for 5 minutes, \SI{1}{\mL} (pre-warmed) LB was added, incubated
\SI{37}{\celsius} for an hour. Cells were plated on agar plates containing
\SI{200}{\ug\per\mL} ampicillin, \SI{34}{\ug\per\mL} chloramphenicol. A single
colony was picked and grown in LB with \SI{200}{\ug\per\mL} ampicillin and
\SI{34}{\ug\per\mL} chloramphenicol at \SI{37}{\celsius} incubation at 300
r.p.m. for 16 hours.  Afterwards, \SI{250}{\mL} of LB medium bearing the same
antibiotics for large-scale expression was innoculated 1:50 with the overnight
culture and incubated at \SI{37}{\celsius} and 300 r.p.m. (ATR Biotech
Multitron shaker). While optical density reached 1.0 at 600 nm, the expression
media were supplemented with \SI{1}{\milli\Molar}
isopropyl-$\beta$-D-thiogalactopyranoside (IPTG) to induce protein expression.
\SI{1}{\milli\Molar} of \ce{CoCl2} was added in both pre- and post-induction
medium.  After three hours incubation at \SI{37}{\celsius} and 300 r.p.m., the
cells were harvested by an centrifuge of 4000 r.p.m.  (Beckman Coulter, Jersey
City, NJ. F10 rotor) at \SI{4}{\celsius} for 15 minutes and then resuspended
with \SI{20}{\milli\Molar} Tris-HCl,
\SI{500}{\milli\Molar} \ce{NaCl}, \SI{5}{\milli\Molar} imidazole, 10\% glycerol
and \SI{1}{\micro\Molar} \ce{CoCl2} (pH 8.0). Cell lysate was immediately
sonicated at 400 kJ for 2.5 minutes at \SI{4}{\celsius} (Qsonica, Newtown, CT),
and then a clarification spin was performed (20000 r.p.m, \SI{4}{\celsius}, 30
minutes).  Clarified supernatants were loaded into a \SI{5}{\mL} His Trap
column (G.E Healthcare, Piscataway, NJ) using AKTA FPLC purifier (G.E.
Healthcare, Piscataway, NJ).  Protein was eluted by using 30\% elution buffer B
(\SI{20}{\milli\Molar} Tris-HCl, \SI{500}{\milli\Molar} sodium chloride,
\SI{500}{\milli\Molar} imidazole,\SI{100}{\micro\Molar} \ce{CoCl2}, pH 8.0) at
\SI{4}{\celsius}.  The purified samples were then transferred into a 3.5K MWCO
dialysis SnakeSkin (Life Technologies, Carlsbad, CA) for buffer exchange using
\SI{12}{\L} \SI{20}{\milli\Molar} phosphate buffer (pH 8.0,
\SI{100}{\micro\Molar} \ce{CoCl2}) on a stirred plate at \SI{4}{\celsius}.
Dialyzed protein was subjected to kinetic assays immediately. The purity of
protein was determined by 12\% sodium dodecyl sulfate-polyacrylamide gel
electrophoresis (SDS-PAGE) and the purified yields were calculated from
SDS-PAGE gels and FPLC UV intensity (Figure \ref{fig:gel},
\ref{fig:yield-estimate}, Table \ref{tab:st1}). The molecular weights were
confirmed by MALDI-TOF (matrix-assisted laser desorption/ionization
time-of-flight) (Table \ref{tab:st2}).
% --------------------------
\begin{table}
    \caption{The comparison of variants purified yields}
    \centering
    \begin{tabular}{lll}
    \hline

    protein & area integrated (mAU)\textsuperscript{a} & purified yield (mg) \\
    \hline

    PTE & 98 & 1.83 \\
    F51L & 74 & 1.38 \\
    F150M & 70 & 1.31 \\
    F216L & 101 & 1.88 \\
    F304L & 83 & 1.55 \\
    F306L & 105 & 1.96 \\
    F327L & 70 & 1.84 \\
    F335M & 95 & 1.77\\
    F357L & 63 & 1.18\\

    \hline
    \multicolumn{3}{l}{\textsuperscript{a} Calculated via GE FPLC UNICORN software.}
    \end{tabular}
    \label{tab:st1}
\end{table}
% --------------------------
% --------------------------
\begin{table}
    \caption{MALDI-TOF comparisons of wild-type PTE and variants.}
    \centering
    \begin{tabular}{lll}
    \hline

    protein & expected molecular weight\textsuperscript{a} & observed molecular weight \\
    \hline

    PTE & 35537.66 & ---- \\
    F51L & 35503.64 & ---- \\
    F150M & 35521.68 & ---- \\
    F216L & 35503.64 & ---- \\
    F304L & 35503.64 & ---- \\
    F306L & 35503.64 & ---- \\
    F327L & 35503.64 & ---- \\
    F335M & 35521.68 & ---- \\
    F357L & 35503.64 & ---- \\

    \hline      
     \multicolumn{3}{l}{\textsuperscript{a}Expected molecular weights were
     calculated by ExPASy \cite{Tools2010}.} \end{tabular}
    \label{tab:st2}
\end{table}
% --------------------------

\subsection{Enzyme Kinetics}

The protein was diluted to a final concentration of \SI{30}{\nano\Molar} in
\SI{20}{\milli\Molar} sodium phosphate (pH 8.0, \SI{100}{\micro\Molar}
\ce{CoCl2}) by using the extinction coefficient \SI{29575}{\per\Molar\per\cm}
for all proteins with Nano-Drop (Waltham, MA) \cite{Gasteiger2005, Pace1995}.
Reactions were monitored spectrophotometrically (Synergy H1, BioTek, Winooski
VT) at \SI{405}{\nm} for paraoxon (coefficient = \SI{17000}{\per\Molar\per\cm})
\cite{Baker2011b} in a 96-well plate. Reactions for paraoxon
(\SIrange{13}{104}{\micro\Molar}) was carried out in 0.2\% methanol at room
temperature.  K\textsubscript{M} and k\textsubscript{cat} values were
determined by a Lineweaver-Burk plot \cite{Baker2011b}. The below equation was
used (Eq.~\ref{eqn:MM-chap2}):
% --------------------------
\begin{equation} \frac{1}{v} =
    \frac{K\textsubscript{M}}{V\textsubscript{max}}\times\frac{1}{S} +
    \frac{1}{V\textsubscript{max}} \label{eqn:MM-chap2} \end{equation}
% --------------------------

where S represents substrate concentration; K\textsubscript{M} represents the
substrate concentration at which the reaction rate is half of
V\textsubscript{max}. The data reported was the average of three trials and the
error represented the standard deviation of those trials. Residual activities
and shelf life measurements were conducted with the same batch of proteins
(\SI{30}{\nano\Molar} in \SI{20}{\milli\Molar} sodium phosphate,
\SI{100}{\micro\Molar} \ce{CoCl2}, pH 8.0). For residual activity assays,
proteins at \SI{35}{\celsius}, \SI{45}{\celsius}, and \SI{55}{\celsius} were
heated for one hour, cooled back to room temperature for one hour, and then
assessed for activity on paraoxon (\SIrange{13}{104}{\micro\Molar}, 0.2\%
methanol).  Half-life experiments were carried out using proteins
(\SI{30}{\nano\Molar} in \SI{20}{\milli\Molar} sodium phosphate,
\SI{100}{\micro\Molar} \ce{CoCl2}, pH 8.0) that were kept under room
temperature for 1, 2, 3, and 7 days. After incubation, activity for paraoxon
(\SIrange{13}{104}{\micro\Molar}, 0.2\% methanol) was assessed at room
temperature. 

\subsection{Thermostability and Secondary Structure of Phosphotriesterase}

\subsubsection{Differential Scanning Calorimetry of PTE Variants}

Differential scanning calorimetry (Nano-DSC, TA instrument, USA) was performed
by using \SI{600}{\micro\L} (\SI{0.1}{\mg\per\mL}) of protein right after
dialysis into \SI{20}{\milli\Molar} sodium phosphate buffer
(\SI{100}{\micro\Molar} \ce{CoCl2}, pH 8.0).  Measurements were conducted at a
scan rate of \SI{1}{\celsius\per\minute} from \SI{20}{\celsius} to
\SI{70}{\celsius}. Each signal was blanked with buffer under the same condition.
The observed scan was then analyzed by using three-scaled model in NanoAnalyze
software (TA instrument, USA). Cp and T\textsubscript{m} were determined by
fitting to a three state model built in NanoAnalyze \cite{Privalov1986}. The
equation is shown below:
% --------------------------
\begin{equation} C\textsubscript{p} = \kappa\textsubscript{B}
    \Bigg(\frac{\epsilon}{\kappa\textsubscript{B}T}\Bigg) \times
    \frac{e^{\beta\kappa}}{\big[e^{\beta\kappa}+1\big]^{2}}
    \label{eqn:dsc-chap2} \end{equation}
% --------------------------

where C\textsubscript{p} is the heat capacity (\si{\J\per\mol}),
$\kappa$\textsubscript{B} is the Boltzmann constant (\si{\J\per\kelvin}), T is
the temperature (\si{\kelvin}), and $\epsilon$ is the energy (\si{J}).

\subsubsection{Circular Dichroism}

Circular dichroism (CD) spectra were recorded on a JASCO J-815
Spectropolarimeter (Easton, MD) using Spectra Manager software
\cite{Kataev1985}. Temperature was controlled at \SI{25}{\celsius} using a
Fisher Isotemp Model 3016S water bath.  Proteins concentrations were
\SI{10}{\micro\Molar} in \SI{20}{\milli\Molar} phosphate buffer (pH 8.0,
\SI{100}{\micro\Molar} \ce{CoCl2}). The sample volume was
\SI{600}{\micro\liter}. As a blank, \SI{20}{\milli\Molar} phosphate buffer was
used. The signal was converted into mean residue molar ellipticities (deg
$\times$ cm\textsuperscript{2} $\times$ dmol\textsuperscript{-1}) using the
following formula \cite{Kelly2005} (Eq.~\ref{eqn:CD-chap2}): 
% --------------------------
\begin{equation} θmrw = MRW(θobs) / (10 * c * l) \label{eqn:CD-chap2}
\end{equation}
% --------------------------

where MRW is the mean residue weight of phosphotriesterase
(\si{\gram\per\mol}), $\theta$obs is the observed ellipticities (mdeg),
\emph{l} is the path length (cm), \emph{c} is the concentration in
\SI{}{\Molar}. Spectra was recorded from \SIrange{190}{250}{\nm} with a scan
speed of \SI{1}{\nano\meter\per\minute}.  The data presented is an average of
three scans.

\section{Results}

\subsection{PTE Variant Expression And Purification}

To explore the impact of phenylalanines outside the dimer interface on
function and stability, eight residues were identified for mutagenesis. We
conducted a sequence alignment of the \emph{Flavobacterium} S5-PTE and
organophosphorus hydrolase (OPH) proteins (Figure \ref{fig:pte-alignment}).
Using bacterial S5-PTE sequence as a template, strains, including
\emph{Brevundimonas} (ID:AIB55573.1), \emph{Flavobaterium} (ID:AAV39527.1),
\emph{Rhizobiacae} (ID:WP-022594966.1), and \emph{Photorhabdus}
(ID:C7BN74-PHOAA) were identified via BLAST BLOSUM62 \cite{Styczynski2008}.  Of
the eight phenylalanines, F51, F150, F216, F304, F306, F327, F335, and F357,
two were conserved across the OPH proteins, while the remainder were
substituted with aliphatic amino acids, such as methionine and leucine, charged
residue (arginine), or aromatic isostere tyrosine in the homologous sequence
(Figure \ref{fig:pte-alignment}). To investigate the significance of
phenylalanine as an aromatic ring, we chose to mutate it to methionine or
leucine as they share alike sizes and properties with phenylalanine
\cite{Richards1974,McDaniel1988}.  Methionine and leucine exhibited a van der
Waals volume of \SI{124}{\angstrom}, similar to phenylalanine
(\SI{135}{\angstrom}) without the aromaticity \cite{Barnes2007,Richards1974}.
In general, phenylalanine was substituted to the corresponding aliphatic
methionine or leucine observed in the alignment (Figure
\ref{fig:pte-alignment}).  For residues that demonstrated homologous
substitutions to charged or aromatic amino acids, we selected leucine. This led
to construction of the following eight mutations: F51L, F150M, F216L, F304L,
F306L, F327L, F335M, and F357L (Figure \ref{fig:pte-alignment}).  
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig2_04}
    \caption[The sequences alignment for phosphotriesterase using S5
        sequence. Strains, including \emph{Brevundimonas},
        \emph{Flavobaterium}, \emph{Rhizobiacae}, and \emph{Photorhabdus},
        were used for alignment. Highlighted in yellow represent the
        phenylalanine residues outside the dimer interface. The alignment was
        conducted via \emph{blastp} program (NCBI). Expected threshold was 10,
    and matrix was BLOSUM62.]{The sequences alignment for phosphotriesterase
        using S5 sequence.\cite{Roodveldt2005} Srtains, including
        \emph{Brevundimonas}, \emph{Flavobaterium}, \emph{Rhizobiacae}, and
        \emph{Photorhabdus}, were used for alignment. Highlighted in yellow
        represent the phenylalanine residues outside the dimer interface. The
        alignment was conducted via \emph{blastp} program (NCBI). Expected
    threshold was 10, and matrix was BLOSUM62.}
    \label{fig:pte-alignment}
\end{figure}
% --------------------------

All genes for site-directed mutagenesis were generated by following the codon
usage of \emph{E. coli} as well as the codons of mutation sites
\cite{Sivashanmugam2009b}: AUG for methionine, and UUA or CUG for leucine.
After FPLC purification, wild-type PTE purity was confirmed by using
12\% sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE)
(Figure \ref{fig:gel}). A single band at 37kD appeared on the gel
across four elutions. Approximately \SI{1.83}{\mg} of wild-type PTE was estimated
by analyzing intensity of bands on ImageJ \cite{Abramoff2004}. The amount of
wild-type protein was compared with UV trace (280nm) from FPLC (Figure
\ref{fig:yield-estimate}). All other variants yields were normalized in
comparison of wild-type UV trace and yield, leading to a 1\%, 2\%, and 7\%
increase of yields in F216L, F306L, and F327L, respectively (Table
\ref{tab:st1}). Approximately 25\%, 28\%, 15\%, 4\%, and 36\% decrease in
yields were estimated in F51L, F150M, F304L, F335M, and F357L, respectively
(Table \ref{tab:st1}). 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.5\textwidth]{fig2_34}
    \caption[12\% sodium dodecyl sulfate-polyacrylamide gel electrophoresis
    (SDS-PAGE) of wild-type PTE after purification. Ladder indicates the size
of 37kD. E1, E2, E3, and E4 represent the samples of elution 1, 2, 3, and
4.]{12\% sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE)
of wild-type PTE after purification. Ladder indicates the size of 37kD. E1, E2,
E3, and E4 represent the samples of elution 1, 2, 3, and 4.}
\label{fig:gel}
\end{figure}
% --------------------------
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig2_35}
    \caption[The UV trace of PTE variants after FPLC purification. All variants
    are stacked on wild-type PTE. Baseline of each variants are normalized. The
histogram is generated by using GE UNICORN software.]{The UV trace of PTE
    variants after FPLC purification. All variants are stacked on wild-type
    PTE. Baseline of each variants are normalized. The histogram is generated
by using GE UNICORN software.}
\label{fig:yield-estimate}
\end{figure}
% --------------------------

\subsection{Rosetta Design of Phosphotriesterase Variants}

Rosetta was employed to evaluate the stability and function of the eight PTE
variants. The wild-type PTE was constructed based on the crystal structure
(1HZY). For each variant, 1000 decoys were generated for each variant, which
followed the normal distribution (Figure \ref{fig:rosetta-box-plot}). In
comparison to wild-type PTE, which possessed a Rosetta Energy Unit (REU) of
-1119.93 $\pm$ 5.42, three variants, F304L, F327L, and F335M, demonstrated
values of -1107.29 $\pm$ 5.69, -1112.77 $\pm$ 6.27, and -1111.89 $\pm$ 6.66,
respectively, indicating a destabilization (Figure \ref{fig:rosetta-box-plot}).
Among eight variants, only F306L exhibited improved REU value -1121.28 $\pm$
5.60 (Figure \ref{fig:rosetta-box-plot}). The remaining variants, F51L, F150M,
F216L, and F357L, showed minor changes of REU scores when compared with
wild-type ($\Delta$REU $\leq$ 2) (Figure \ref{fig:rosetta-box-plot}).
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig2_02}
    \caption[Rosetta energy units (REU) comparison of wild-type PTE and
    variants. Reported REUs are scored by using Rosetta suite. 1000 decoys are
generated for each variant.]{Rosetta energy units (REU) comparison of wild-type
PTE and variants. Reported REUs are scored by using Rosetta suite. 1000 decoys
are generated for each variant. Experiments performed in collaboration with
Bonneau Lab.} \label{fig:rosetta-box-plot}
\end{figure}
% --------------------------

\subsection{Activity of PTE Variants}

To assess function, the kinetics of PTE and its variants were determined on
paraoxon. Activity was evaluated by measuring the increase of
\emph{p}-nitrophenol after hydrolysis at 405 nm
\cite{Baker2011b,Yang2014a,Carr1996b,Cho2004b} (See Appendix). At
\SI{25}{\celsius}, wild-type PTE exhibited a
k\textsubscript{cat}/K\textsubscript{M} = \SI{170000}{\per\Molar\per\second}
consistent with previously published work \cite{Yang2014a,Baker2011b} (Figure
\ref{fig:activity-chart}, Table \ref{tab:kinetics-chap2-result}). As
anticipated from Rosetta, F304L, F327L, F335M demonstrated loss in activity,
exhibiting 32\%, 69\%, and 60\% in
k\textsubscript{cat}/K\textsubscript{M} relative to wild-type (Figure
\ref{fig:activity-chart}, Table \ref{tab:kinetics-chap2-result}) Two variants,
F216L and F306L, demonstrated improved activity with
k\textsubscript{cat}/K\textsubscript{M} values of 147\% and 142\%, respectively
relative to wild-type PTE (Figure \ref{fig:activity-chart}, Table
\ref{tab:kinetics-chap2-result}). While F306L was predicted by Rosetta to be
more stable, F216L possessed similar REU to the wild-type PTE (Figure
\ref{fig:rosetta-box-plot}). As the Rosetta simulations reflected stability of
variants relative to wild-type PTE, residual activity experiments were carried
out.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig2_05}
    \caption[The activity comparison of PTE and the eight variants at different
        temperatures. Proteins were incubated at \SIlist{35;45;55}{\celsius}
        for one hour. After cooling down to room temperatures, samples were
    assayed with the substrate, paraoxon, at 405 nm. All
k\textsubscript{cat}/K\textsubscript{M} values were normalized to wild-type PTE
at \SI{25}{\celsius}.]{The activity comparison of PTE and the eight variants at
    different temperatures. Proteins were incubated at
    \SIlist{35;45;55}{\celsius} for one hour. After cooling down to room
    temperatures, samples were assayed with the substrate, paraoxon, at 405 nm.
    All k\textsubscript{cat}/K\textsubscript{M} values were normalized to
    wild-type PTE at \SI{25}{\celsius}.} \label{fig:activity-chart}
\end{figure}
% --------------------------

To determined the function at elevated temperatures, the residual activities
were assessed after one hour incubation of the proteins at
\SIlist{35;45;55}{\celsius} (See Appendix). As expected, the wild-type PTE
illustrated a gradual decrease in activity, consistent with prior studies
(Figure \ref{fig:activity-chart}, Table \ref{tab:kinetics-chap2-result})
\cite{Yang2014a,Baker2011b}. All the variants demonstrated a loss in activities
at elevated temperatures. Notably, the three variants, F304L, F327L, and F335M,
revealed a rapid decrease in residual activity, especially at \SI{45}{\celsius}
and \SI{55}{\celsius} (Figure \ref{fig:activity-chart}, Table
\ref{tab:kinetics-chap2-result}). At \SI{45}{\celsius}, F327L and F335M
exhibited k\textsubscript{cat}/K\textsubscript{M} of
\SI{68000\pm12000}{\per\Molar\per\second} and
\SI{33000\pm15000}{\per\Molar\per\second}, respectively, while F304L completely
lost its activity to hydrolyze paraoxon (Figure \ref{fig:activity-chart}, Table
\ref{tab:kinetics-chap2-result}). All residual activity was lost at
\SI{55}{\celsius} for both F327L and F335M (Figure \ref{fig:activity-chart}).
These thermo active profiles affirmed the Rosetta predictions where F304L
possessed a substantial loss in residual activity at elevated temperatures
(Figure \ref{fig:rosetta-box-plot}). While both F216L and F306L exhibited
improved activity at \SI{25}{\celsius} relative to wild-type PTE, F306L
revealed a consistently better activity across all elevated temperatures
(Figure \ref{fig:activity-chart}, Table \ref{tab:kinetics-chap2-result}). This
overall improved residual activity over wild-type PTE supported Rosetta
simulations where F306L was identified as the most stable (Figure
\ref{fig:rosetta-box-plot}).
% --------------------------
\begin{table}[htbp]
    \centering
    \caption[Paraoxon hydrolysis efficiency summary of PTE and its variants.
    Residual activities were performed after incubation at
\SIlist{35;45;55}{\celsius}.] {Paraoxon hydrolysis efficiency summary of PTE
    and its variants. Residual activities were performed after incubation at
    \SIlist{35;45;55}{\celsius}.}
    \begin{tabular}{llllll}
    \hline
    protein                 &  & \SI{25}{\celsius} & \SI{35}{\celsius} &
    \SI{45}{\celsius} & \SI{55}{\celsius} \\ 
    \hline
    
    \multirow{2}{*}{PTE}    & k\textsubscript{cat}/K\textsubscript{M} & 1.70 $
    \pm$ 0.20 & 1.20 $\pm$ 0.23 & 1.00 $\pm$ 0.11 & 0.65 $\pm$ 0.10 \\
    & k\textsubscript{cat} & 2.3 $\pm$ 0.5 & 2.0 $\pm$ 0.7 & 1.4 $\pm$ 0.5 & 1.3
    $\pm$ 0.5 \\
    \hline
    \multirow{2}{*}{F51L}  & k\textsubscript{cat}/K\textsubscript{M} & 1.40
    $\pm$ 0.19 & 1.20 $\pm$ 0.20 & 1.10 $\pm$ 0.20 & 0.70 $\pm$ 0.09 \\
    & k\textsubscript{cat} & 3.0 $\pm$ 1.1 & 2.6 $\pm$ 0.5 & 1.2 $\pm$ 0.3 &
    0.8 $\pm$ 0.3 \\
    \hline
    \multirow{2}{*}{F150M} & k\textsubscript{cat}/K\textsubscript{M} &
    1.30 $\pm$ 0.08 & 1.20 $\pm$ 0.10 & 0.90 $\pm$ 0.08 & 0.50 $\pm$ 0.07 \\
    & k\textsubscript{cat} & 2.5 $\pm$ 1.0 & 2.0 $\pm$ 1.0 & 1.4 $\pm$ 0.3 &
    1.0 $\pm$ 0.4 \\
    \hline
    \multirow{2}{*}{F216L} & k\textsubscript{cat}/K\textsubscript{M}
    & 2.50 $\pm$ 0.13 & 1.30 $\pm$ 0.10 & 1.00 $\pm$ 0.08 & 0.90 $\pm$ 0.20 \\
    & k\textsubscript{cat} & 2.5 $\pm$ 0.6 & 2.4 $\pm$ 0.6 & 1.8 $\pm$ 0.5 &
    1.7 $\pm$ 0.6 \\
    \hline
    \multirow{2}{*}{F304L}  & k\textsubscript{cat}/K\textsubscript{M} & 0.55
    $\pm$ 0.13 & 0.50 $\pm$ 0.10 & n.a. & n.a. \\
    & k\textsubscript{cat} & 0.6 $\pm$ 0.3 & 0.2 $\pm$ 0.1& n.a. & n.a. \\
    \hline
    \multirow{2}{*}{F306L}  & k\textsubscript{cat}/K\textsubscript{M} & 2.43
    $\pm$ 0.23 & 1.72 $\pm$ 0.04 & 1.38 $\pm$ 0.36 & 0.90 $\pm$ 0.12 \\
    & k\textsubscript{cat} & 2.1 $\pm$ 0.6 & 1.8 $\pm$ 0.5 & 1.7 $\pm$ 0.5 &
    1.2 $\pm$ 0.6 \\
    \hline
    \multirow{2}{*}{F327L}  & k\textsubscript{cat}/K\textsubscript{M} & 1.17
    $\pm$ 0.13 & 1.10 $\pm$ 0.19 & 0.68 $\pm$ 0.12 & n.a. \\
    & k\textsubscript{cat} & 1.5 $\pm$ 0.6 & 1.3 $\pm$ 0.5 & 0.8 $\pm$ 0.3 &
    n.a. \\
    \hline
    \multirow{2}{*}{F335M}  & k\textsubscript{cat}/K\textsubscript{M} & 1.02
    $\pm$ 0.13 & 0.78 $\pm$ 0.10 & 0.33 $\pm$ 0.15 & n.a. \\
    & k\textsubscript{cat} & 1.3 $\pm$ 0.5 & 0.6 $\pm$ 0.5 & 0.3 $\pm$ 0.2 &
    n.a. \\
    \hline
    \multirow{2}{*}{F357L}  & k\textsubscript{cat}/K\textsubscript{M} & 1.91
    $\pm$ 0.25 & 1.50 $\pm$ 0.13 & 1.10 $\pm$ 0.09 & 0.56 $\pm$ 0.09 \\
    & k\textsubscript{cat} & 2.5 $\pm$ 0.7 & 1.8 $\pm$ 0.3 & 1.5 $\pm$ 0.5 &
    0.4 $\pm$ 0.2 \\

    \hline
    \multicolumn{6}{l}{n.a = no activity; 
        k\textsubscript{cat}/K\textsubscript{M}:
        $\times$10\textsuperscript{5}\SI{}{\per\Molar\per\second};
        k\textsubscript{cat}: \SI{}{\per\second}.}            
    \end{tabular}
    \label{tab:kinetics-chap2-result}
\end{table}
% --------------------------

\subsection{Structure and Thermodynamics Stability of PTE Variants}

Far UV wavelength scans of PTE and variants were evaluated to assess the impact
of mutations on secondary structure (Figure \ref{fig:cd-chap2-result}, Table
\ref{tab:cd-chap2-result}). The wavelength scan for wild-type PTE at
\SI{25}{\celsius} exhibited a double minima at 208 and 222 nm of -811 and -923
deg$\times$cm\textsuperscript{2}$\times$dmol\textsuperscript{-1}, respectively,
consistent with previous studies \cite{Yang2014a,Baker2011b}. The variants
possessed a slight loss in structure relative to wild-type PTE with the
exception of F216L, which possessed a lightly more negative signature of -822
and -963 deg$\times$cm\textsuperscript{2}$\times$dmol\textsuperscript{-1} at
208 and 222 nm, respectively (Figure \ref{fig:cd-chap2-result}, Table
\ref{tab:cd-chap2-result}). In general, the variants demonstrated similar
conformations, indicating that the mutations did not significantly alter the
PTE structure. 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.5\textwidth]{fig2_06}
    \caption[The CD wavelength scan overlay of variants relative to wild-type
    PTE. Proteins concentrations were \SI{10}{\micro\Molar} in
\SI{20}{\milli\Molar} phosphate buffer (\SI{100}{\micro\Molar} \ce{CoCl2}, pH
8.0). Sample volume was \SI{600}{\micro\liter}.]{The CD wavelength scan overlay
    of variants relative to wild-type PTE. Proteins concentrations were
    \SI{10}{\micro\Molar} in \SI{20}{\milli\Molar} phosphate buffer
    (\SI{100}{\micro\Molar} \ce{CoCl2}, pH 8.0). Sample volume was
    \SI{600}{\micro\liter}.}
    \label{fig:cd-chap2-result}
\end{figure}
% --------------------------

% --------------------------
\begin{table}[htbp]
    \centering
    \caption[The mean residue molar ellipticity of PTE and its variants at 208
    and 222 nm.] {The mean residue molar ellipticity of PTE and its variants at
    208 and 222 nm.}
    \begin{tabular}{lll}
    \hline

    protein & $\theta$ 208 nm$\ddag$ & $\theta$ 222 nm$\ddag$ \\
    \hline

    PTE & -811  & -923 \\
    F51L & -694 &  -814 \\
    F150M & -560 & -810 \\
    F216L & -822 & -963 \\
    F304L & -684 & -749 \\
    F306L & -719 & -787 \\
    F327L & -683 & -731 \\
    F335M & -702 & -791 \\
    F357L & -735 & -785 \\

    \hline
    \multicolumn{3}{l}{$\ddag$: units represent
        deg$\times$cm\textsuperscript{2}$\times$dmol\textsuperscript{-1}}
    \end{tabular} 
    \label{tab:cd-chap2-result}
\end{table}
% --------------------------

To determine the thermodynamic stability of wild-type PTE and variants, DSC was
employed.  Wild-type PTE exhibited two endothermic transitions at
\SI{50.94\pm0.03}{\celsius} (T\textsubscript{m1}) and \SI{63.54 \pm
0.10}{\celsius} (T\textsubscript{m2}) as expected in affirmation of previous
studies (Figure \ref{fig:dsc-chap2-result}, Table \ref{tab:dsc-chap2-result})
\cite{Baker2011b}. While all variants possessed two transitions, similar to
wild-type PTE with slight changes in T\textsubscript{m}, F51L and F357L
demonstrated substantial changes in T\textsubscript{m2} with differences of
\SI{9.56}{\celsius} and \SI{5.86}{\celsius}, respectively (Figure
\ref{fig:dsc-chap2-result}, Table \ref{tab:dsc-chap2-result}). Although the
unfolding of small proteins could be described by the two-state model
\cite{Jackson1998}, complex proteins populate intermediate conformations at
equilibrium \cite{Lamazares2015,Barrick1993,Gualfetti1999}. Grimsley \latin{et
al.} demonstrated that the PTE intermediate (I\textsubscript{2}) lost both
its structure and function \cite{Rochu2002b,Grimsley1997b} (Figure
\ref{fig:pte-unfold}). While Lamazares \latin{et al.}
showed that the stabilization of apo-flavodoxin could be achieved by
introducing mutations into protein; 11 individual single mutants showed
increased T\textsubscript{m1} values (from 2.6 to \SI{8.8}{\celsius})
relative to wild-type apo-flavodoxin \cite{Lamazares2015} via DSC.
Hypothesizing that thermal stabilization was generated through the first
transition, their results confirmed that the stabilizing mutations led to an
increased T\textsubscript{m1}. As these PTE variants proteins possessed a
3-state transitions with intermediate conformations
\cite{Rochu2002b,Grimsley1997b}, we chose to focus on the first transition in
order to assess stability according to Lamazares \emph{et al.}
\cite{Lamazares2015}.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.9\textwidth]{fig2_07}
    \caption[The DSC comparison of T\textsubscript{m1} and T\textsubscript{m2}.
    \SI{600}{\micro\L} (\SI{0.1}{\mg\per\mL}) of protein was used for
    measurements. The measurements were conducted at a scan rate of
    \SI{1}{\celsius\per\minute} from \SI{20}{\celsius} to \SI{70}{\celsius}.
    Signals was blanked with buffer under the same condition.]{The DSC
        comparison of T\textsubscript{m1} and T\textsubscript{m2}.
        \SI{600}{\micro\L} (\SI{0.1}{\mg\per\mL}) of protein was used for
        measurements. The measurements were conducted at a scan rate of
        \SI{1}{\celsius\per\minute} from \SI{20}{\celsius} to
        \SI{70}{\celsius}.  Signals was blanked with buffer under the same
    condition.} 
    \label{fig:dsc-chap2-result}
\end{figure}
% --------------------------

After obtaining the PTE thermogram, the analysis from 3-state model
demonstrated the enthalpies including $\Delta$H\textsubscript{1(cal)},
$\Delta$H\textsubscript{vH1}, $\Delta$H\textsubscript{2(cal)}, and
$\Delta$H\textsubscript{vH2} of wild-type PTE and variants (Table
\ref{tab:dsc-chap2-result}). While T\textsubscript{m} determined the transition
temperature, calorimetric enthalpy, $\Delta$H\textsubscript{1,2(cal)}, was
calculated by the integration of the endotherm curves. The van't
Hoff enthalpy, $\Delta$H\textsubscript{vH1,2}, represented the width of the
transition, allowing enthalpy comparison of cooperative units in the
system \cite{Privalov2009}. Wild-type PTE demonstrated a larger unfolding
enthalpy $\Delta$H\textsubscript{vH1} at \SI{62.4}{kcal\per\mole}. In
comparison to wild-type, F304L, F306L, F327L, and F335M exhibited increased
$\Delta$H\textsubscript{vH1} at \SI{107.0}{kcal\per\mole},
\SI{96.9}{kcal\per\mole}, 90.5 \SI{}{kcal\per\mole}, and
\SI{87.2}{kcal\per\mole}, respectively (Table \ref{tab:dsc-chap2-result}). The
significant change of enthalpy due to a single mutation was also discovered in
Sauer group \cite{Hecht1984a}. With a single A49V mutation in a phage
$\lambda$ repressor protein, the $\Delta$H\textsubscript{vH} demonstrated a
1.7-fold increase as measured by DSC \cite{Hecht1984a}. In contrast, the A66T
exhibited a 5-fold decrease of $\Delta$H\textsubscript{vH} relative to
wild-type protein \cite{Hecht1984a}. 

Due to factors including hydrophobicity, packing interactions, and introduction
or removal of hydrogen bonds, the thermal stability would depend on the
contexts of mutations. To obtain PTE folding cooperativity profile from the
model, the ratio of $\Delta$H\textsubscript{(cal)}/$\Delta$H\textsubscript{vH}
has been calculated among wild-type PTE and variants. This ratio is considered
as a measure of the validity of a studied process \cite{Privalov1986,Gill2010}.
For heat denaturation of a compact globular protein, this ratio is usually
close to 1.0 \cite{Privalov1986}. In contrast, when $\Delta$H\textsubscript{vH}
is more than $\Delta$H\textsubscript{(cal)}, the intermolecular co-operation,
such as aggregation, is shown \cite{Gill2010}. Rochu \latin{et al.} have
demonstrated that irreversible unfolding of PTE using capillary
electropherograms \cite{Rochu2002b}. Moreover, Grimsley \latin{et al.} showed
that PTE aggregates upon heat denaturation \cite{Grimsley1997b}. In this study,
wild-type PTE reveals a ratio of 0.96, consistent with expected values between
0 and 1 \cite{Grimsley1997b}.  While the PTE variants demonstrate ratios lower
than wild-type PTE (Table \ref{tab:dsc-chap2-result}), all are less than 1,
indicative of the irreversible
unfolding.\cite{Grimsley1997b,Privalov2009,Honda2000,Sancho2013}
% --------------------------
\begin{table}[htbp]
    \centering
    \caption[Differential scanning calorimetry results of PTE and its variants.
        The melting temperatures, enthalpies, and van't Hoff enthalpies were
    analyzed via NanoAnalyze three-scaled model (TA instrument).] {Differential
        scanning calorimetry results of PTE and its variants. The melting
        temperatures, enthalpies, and van't Hoff enthalpies were analyzed via
    NanoAnalyze three-scaled model (TA instrument).}
    \begin{tabular}{lllclclc}
    \hline

    protein & T\textsubscript{m}1 (\si{\celsius}) & T\textsubscript{m}2
    (\si{\celsius}) & $\Delta$H\textsubscript{1(cal)} &
    $\Delta$H\textsubscript{vH1} & $\Delta$H\textsubscript{2(cal)} &
    $\Delta$H\textsubscript{vH2} &
    $\Delta$H\textsubscript{1(cal)}/$\Delta$H\textsubscript{vH1} \\
    \hline

    PTE & 50.94 $\pm$ 0.03  & 63.54 $\pm$ 0.10 & 59.9 & 62.4 & 11.0 & 15.9 & 0.96\\
    F51L & 49.46 $\pm$ 0.50 & 53.98 $\pm$ 0.13 & 10.9 & 60.1 & 13.6 & 54.9 & 0.18\\
    F150M & 49.33 $\pm$ 1.14 & 60.26 $\pm$ 1.00 & 31.0 & 35.5 & 68.4 & 75.8 & 0.87\\
    F216L & 50.80 $\pm$ 0.40 & 63.90 $\pm$ 0.05 & 15.0 & 34.8 & 25.0 & 126.7 & 0.43\\
    F304L & 53.72 $\pm$ 0.04 & 62.35 $\pm$ 0.05 & 94.2 & 107.0 & 7.4 & 10.3 & 0.80\\
    F306L & 52.03 $\pm$ 0.03 & 63.13 $\pm$ 0.03 & 40.2 & 96.9 & 5.0 & 10.2 & 0.42\\
    F327L & 50.16 $\pm$ 0.03 & 62.79 $\pm$ 0.50 & 47.1 & 90.5 & 10.3 & 13.5 & 0.52\\
    F335M & 50.07 $\pm$ 0.08 & 62.03 $\pm$ 0.03 & 26.4 & 87.2 & 11.9 & 15.4 & 0.30\\
    F357L & 48.90 $\pm$ 0.15 & 57.68 $\pm$ 0.35 & 10.7 & 37.4 & 9.7 & 67.2 & 0.20\\

    \hline
    \multicolumn{8}{l}{$\Delta$H\textsubscript{1,2(cal)}: kcal/mol;
    $\Delta$H\textsubscript{vH1,2}: kcal/mol}
    \end{tabular}
    \label{tab:dsc-chap2-result}
\end{table}
% --------------------------

\section{Discussion}

Phenylalanines within the dimer interface of PTE have been have been
demonstrated to be important for stabilization and function \cite{Yang2014a,
Baker2011b,Toone2009}. With the buried total surface of
\SI{3200}{\angstrom^{2}} in the interface, F65, F72, F73, F104, F132, F149, and
F179 are involved in stacking interactions \cite{Toone2009} (Figure
\ref{fig:interface}). For example, Benning \latin{et al.} have shown that F72
interacts with W69 from the same monomer as well as F149 from the opposite
subunit of PTE \cite{Benning2001a}.  Moreover, our group have demonstrated that
the removal of one of the stacking forces, F104, leads to significant decrease
of activity \cite{Yang2014a}.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig2_31}
    \caption[The F65, F72, F73, F104, F132, F149, and F179 residues at the
    dimer interface of chain A of PTE (PDB 1HZY). The image is generated using
UCSF Chimera.]{The F65, F72, F73, F104, F132, F149, and F179 residues at the
dimer interface of PTE chain A (PDB 1HZY). The image is generated using UCSF
Chimera.}
    \label{fig:interface}
\end{figure}
% --------------------------

However, the phenylalanines outside the dimer interface may not be necessary
for stability \cite{Pakula1990a,Schwehm1998}. For example, Schwehm \latin{et
al.} have systematically mutated solvent-exposed side chains to phenylalanine in
Staphylococcal nuclease and investigated stability using guanidine
hydrochloride denaturation \cite{Schwehm1998}. The average loss in
stability ($\Delta$$\Delta$G\textsubscript{F-WT}) for the phenylalanine mutants
is 1.1 $\pm$ 1.2 kcal/mol \cite{Schwehm1998} (Figure \ref{fig:phe50}). To test
this hypothesis within the context of PTE, we have generated eight single
phenylalanine substitutions using computational and experimental studies.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig2_28}
    \caption[A histogram showing the frequency of the occurrence of a given
    change in the stability relative to wild type for the 50 different
phenylalanine mutants.]{A histogram showing the frequency of the occurrence of
    a given change in the stability relative to wild type for the 50 different
    phenylalanine mutants \cite{Schwehm1998}.}
    \label{fig:phe50}
\end{figure}
% --------------------------

Rosetta predicted that substitution of phenylalanines outside the dimer
interface maintained stability relative to wild-type PTE with the exception of
F304L, F327L, and F335M, which led to a loss in activity at elevated
temperatures. Surprisingly, Rosetta also predicted improved functionality for
the F306L variant, which also was confirmed by experimental results. Below, we
discuss these four variants in details.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig2_08}
    \caption[(A) RMSD overlay of F304L (red), F306L (green), and F327L (blue)
        relative to wild-type PTE. The large peak deviation corresponds to loop
        7 region. (B) Overlay of the structures of F304L (red),  F306L (green),
    and F327L (blue) relative to wild-type PTE (grey).] {(A) RMSD overlay of
        F304L (red), F306L (green), and F327L (blue) relative to wild-type PTE.
        The large peak deviation corresponds to loop 7 region. (B) Overlay of
        the structures of F304L (red),  F306L (green), and F327L (blue)
        relative to wild-type PTE (grey). Experiments performed in
    collaboration with Bonneau Lab.}
    \label{fig:rmsd}
\end{figure}
% --------------------------

\subsection{Significance of F304L, F327L, F335M}

To investigate how the mutations F304L, F327L, and F335M impacted structure,
leading to the experimental loss in function under elevated temperatures, we
assessed the RMSD of all variants relative to wild-type PTE (Figure
\ref{fig:rmsd}). The 304L variant revealed deviations in the 75-100
region ($\alpha$ helix 3 region) and 200-300 region (loop 7) when compared to
wild-type PTE (Figure \ref{fig:rmsd}). In particular, F327L exhibited a
significant disturbance in the loop 7 region; the overlay of F327L with
wild-type PTE illustrated the difference of structure in this region (Figure
\ref{fig:rmsd}).  Previously, the Tawfik group demonstrated that the loop 7
region was important for hydrolysis by wild-type PTE \cite{Afriat-Jurnou2012}.
With the deletion of loop 7, they observed a 100-fold loss of PTE activity,
consistent with our results. Finally while F335M did not demonstrate a
measurable RMSD relative to wild-type PTE, the mutation affected the overall
hydrogen-bond pattern (Figure \ref{fig:hbond-plot}). It led to significant
impact on the large pocket, H254, H257, and M317, of PTE active site (Figure
\ref{fig:hbond-plot}). 

\subsection{Improved Function and Stability by F306L}

Unlike the above mentioned variants which led to a loss in activity at elevated
temperatures, F306L exhibited improved function and stability in affirmation of
Rosetta predictions. At elevated temperature of \SI{35}{\celsius},
\SI{45}{\celsius}, and \SI{55}{\celsius}, F306L was more active than wild-type
PTE with an additional $\Delta$H\textsubscript{vH1} of \SI{34.5}{kcal\per\mole}
(Table \ref{tab:dsc-chap2-result}). The RMSD overlay revealed the largest deviation
in loop 7 region by F306L  (Figure \ref{fig:rmsd}). This increase of RMSD of
loop 7 allowed PTE to accommodate substrate and hold the compartment for
reaction. While the loop 7 was also significantly altered in the case of F327L,
when aligning the structure, the mutation caused loop 7 to move in the opposite
direction away from the substrate, rendering the PTE active site open, leading
to the loss in function (Figure \ref{fig:rmsd}).

% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig08.pdf}
    \caption[The comparison of wild-type PTE and F335M. (A) The hydrogen-bond
    patterns of wild-type PTE and F335M across all residues. (B) The
    probabilities of hydrogen-bond of wild-type PTE and F335M. (C) Overlay of
    the structures of F335M (red) relative to wild-type PTE (grey).] {The
        comparison of wild-type PTE and F335M. (A) The hydrogen-bond patterns
        of wild-type PTE and F335M across all residues. (B) The probabilities
        of hydrogen-bond of wild-type PTE and F335M. (C) Overlay of the
        structures of F335M (red) relative to wild-type PTE (grey). }
        \label{fig:hbond-plot}
\end{figure}
% --------------------------

In support of the significance of residue F306, the Raushel group
also showed that the mutations into Tyr altered the hydrolysis
efficiency \cite{Pavelka2009,Watkins1997}. Using diisopropyl fluorophosphate
(DFP) as substrate, F306Y demonstrated a 1.5-fold increase of
Vmax/K\textsubscript{M} relative to wild-type PTE \cite{Pavelka2009} (Table
\ref{tab:DFP}).  Moreover, Gopal \emph{et al.} generated variants F306Y that
exhibited enhanced activity on demeton-S-methyl \cite{Gopal2000} (Table
\ref{tab:gopal}). With the goal of increased hydrophobic and large binding
pocket as well as improved electrostatic environment for leaving group, these
experiments affirmed the importance of F306 mutation.  
% --------------------------
\begin{table}[htbp]
    \centering
    \caption[Hydrolysis of DFP with phosphotriesterase. 50 mM HEPES (pH 8.5)
    buffer is used for assays.] {Hydrolysis of DFP with phosphotriesterase. 50
        mM HEPES (pH 8.5) buffer is used for assays \cite{Watkins1997}.}
    \begin{tabular}{lll}
    \hline

    Protein & K\textsubscript{M} (\SI{}{\milli\Molar}) & Vmax (s\textsuperscript{-1}) \\
    \hline

    wild-type PTE & 0.57 & 290 \\
    F306Y & 1.5 & 990 \\
    F306H & 16 & 1400 \\

    \hline  
    \end{tabular}
    \label{tab:DFP}
\end{table}
% --------------------------
% --------------------------
\begin{table}[htbp]
    \centering
    \caption[Relative hydrolase activities (\% of wild type activity) present
    in crude extracts of OPH mutants are shown.] {Relative hydrolase activities
        (\% of wild type activity) present in crude extracts of OPH mutants are
        shown \cite{Gopal2000}.}

    \begin{tabular}{lll}
    \hline

    substrate & F306A & F306Y \\
    \hline

    demeton-S-methyl & 17 & 125 \\
    paraoxon & 10 & 8 \\

    \hline  
    \end{tabular}
    \label{tab:gopal}
\end{table}
% --------------------------

\section{Conclusion}

Here, we demonstrate the application of Rosetta for predicting impacts of
phenylalanine mutations outside the dimer interface of wild-type PTE. Rosetta
has successfully identified crucial residues as well as a variant with improved
stability and function. Valuable insight into structure is provided to further
understand the experimental results providing clues into further re-engineering
PTE with improved stability and function. 

% --------------------------to be deleted content
% --------------------------need to remove ref
%Protein stability is one of key features of protein engineering. Different
%strategies were widely developed for enhancing stability, such as rational
%design\cite{Jackson2009a,Xie2014,Shoichet1995}, directed
%evolution\cite{Roodveldt2005}, or homologous comparison\cite{VandenBurg2002a}.
%In the recent decades, new methodologies, such as unnatural amino acid
%incorporation\cite{Tang2001,Voloshchuk2009,Panchenko2006b,Jackson2006a} and
%\emph{in silico} design of
%protein\cite{Richardson1989,Pavelka2009,Korkegian2005,Xie2014}, were also
%adapted for stabilizing protein. 
%
%For example, Xie \latin{et al.} introduced two mutations
%into Candida antarctica lipase B (CalB) in the active site, and demonstrated
%increased half-life after thousands of colonies were screened.\cite{Xie2014}
%Numerous enzymes were also developed for the improved stabilities against
%temperatures\cite{Baker2011b,Yang2014a,Eijsink1992} or pH.  To understand the
%effect of residues in T4 lysozyme, Shoichet \latin{et al.} mutated functional
%residues in the active site and further analyzed the relationship between the
%enzyme stability and function.\cite{Shoichet1995} Previously, our lab
%demonstrated the enhanced assembly of bio-material through the unnatural amino
%acid incorporation.\cite{Yuvienco2012b} These examples successfully illustrate
%different strategies for creating a stabilized protein.
%
%Several residues across the PTE sequence have been studied for enhancement of
%stability \cite{Baker2011b,Yang2014a} or alternation of
%selectivities\cite{Bigley2013b,Chen-Goodspeed2001a,Pavelka2009}. Rauchel group
%using the alanine scanning method to locate key residues for PTE hydrolysis
%efficiency\cite{Chen-Goodspeed2001a}. As Ile106, Ser306 were swapped with
%alanine, they found the mutations increased substrates hydrolysis efficiency,
%including R\textsubscript{P}-enantiomers containing a phenyl substituent. Due
%to the sterically hindering effect, they suggested that small pocket of PTE
%dictated the chiral preference for the S\textsubscript{P}-enantiomers.
%Interestingly, enlarging the large pocket site did not significantly increase
%the hydrolysis efficiency. However, they did note that H254A in the large
%pocket sites might interacted differently via Asp301 as they were binding with
%$\alpha$ metal. 
%
%Raushel group demonstrated that the loop 7 region were important for hydrolysis
%of PTE. Among hundreds of residues in PTE, F306 position was studied
%intensively, and it had been shown that the mutations, F306Y and F306H, altered
%the efficiency of hydrolysis\cite{Pavelka2009, Chen-Goodspeed2001a}. To invest
%the behaviors of F304L and F306L, we render the simulated structures through
%Rosetta. After minimization, we may compare the RMSD between wild-type and variants.
%
%While we were investigating the increase rate of hydrolysis in F216L, there was
%no significant change in RMSD. With only nine residues that were mildly
%perturbed, F216L exhibited similar REU score compared with wild-type PTE (F216:
%-1118.74 $\pm$ 5.57; wild-type: -1119.94 $\pm$ 5.42).  Interestingly, while
%F216L is buried inside PTE, the overall change is not significant. Simply
%comparing the T\textsubscript{m}1 and T\textsubscript{m}2, as well as catalytic
%efficiency at \SIlist{35;45;55}{\celsius}, F216L exhibits similar properties to
%wild-type.  The distortion of RMSD is also minor --- only nine residues that
%are affected --- compared with other variants . At the same time, the hydrogen
%bonds pattern shows minor changes while we were screening overall effect in
%PTE. (Figure \ref{fgr:hbond-plot}) The only affected residue in the active site
%is W131 in the simulated model. With the only evidence of slight increased
%catalytic efficiency at \SI{25}{\celsius}, we may consider F216L mutation
%neutral in PTE.
%
%Among residues in PTE, F306 position was studied intensively, and it had been
%shown that the mutations, F306Y and F306H, altered the efficiency of
%hydrolysis\cite{Pavelka2009, Chen-Goodspeed2001a}. 
%
%While introducing mutations into PTE, we expected the performances behaving
%similarly to wild-type due to the fact that the sizes and properties of amino
%acids of interest are similar.\cite{Richards1974,McDaniel1988} With the
%successful expressions and purification of all proteins, we performed paraoxon
%hydrolysis and compared the efficiency with simulation results. While
%replacements of phenylalanines with methionines (F150M, F335M) or leucines
%(F51L, F216L, F304L, F306L, F327L ,and F357L) were expected to maintain the
%majority of structure and property of PTE, we found that most of predictions
%follow the pattern of paraoxon hydrolysis.
%
%F51L and F150M PTE hydrolyzed paraoxon similarly to wild-type PTE
%(\SI{140000}{\per\Molar\per\second} and \SI{130000}{\per\Molar\per\second},
%respectively). With REU scores of -1118.75 and -1118.31, these two variants
%slightly changed their structures.  (Figure \ref{fgr:rosetta-box-plot} and
%Table \ref{tal:dsc-result}) In addition to these two neutral variants, F357L
%also exhibited similar hydrolysis efficiency compared with wild-type. (Table
%\ref{tal:kinetics-result}) F357L was scored at -1117.16 from Rosetta, with
%k\textsubscript{cat}/K\textsubscript{M} of \SI{191000}{\per\Molar\per\second}.
%While both scores and kinetics numbers are within the error, these mutations
%turn out to be neutral in PTE sequence despite the minor changes in REU.
%Considering the sizes and properties of methionine, leucine, and phenylalanine,
%we conclude that the replacement at F51L, F150M, and F357L can be selectively
%combined without sacrificing PTE hydrolysis efficiency on paraoxon. 
%
%While focusing on key residues that were
%discussed\cite{Bigley2013b,Chen-Goodspeed2001a,Pavelka2009}, we focus on the
%comparisons of three variant, F304L, F306L, and F335M. Among eight variants,
%F304L and F306L show dramatic differences in terms of paraoxon hydrolysis
%efficiency. (Table \ref{tal:kinetics-result}) While two residues, F304 and
%F306, are close to each other, the effects from these two mutations are
%opposite. F304L shows impaired catalytic efficiency on paraoxon
%(k\textsubscript{cat}/K\textsubscript{M} = \SI{55000}{\per\Molar\per\second})
%with REU of -1107.29 $\pm$ 5.69. On the contrary, F306L with the REU score of
%-1121.28 $\pm$ 5.60, F304L showed enhanced
%k\textsubscript{cat}/K\textsubscript{M} of \SI{243000}{\per\Molar\per\second}
%compared with wild-type. To understand the interactions between F306L, we look
%into the RMSD of F306L. 
%
% --------------------------deleted content
% With respect to the codon usages, we design primer
% sets for each of variants of PTE.  With the site-directed mutagenesis, each
% variant is mutated from the parent DNA and the sequences confirmed.  Plasmids
% for expression of PTE variants were generated from our pQE30-S5PTE construct
% using the polymerase chain reactions.  After 18 rounds of DNA amplification
% reactions, the methylated parent DNA was digested by DpnI enzyme.  The rest of
% the DNA products were then transformed into \emph{E. coli} stain BL21 for
% amplification.
% --------------------------
%
%In addition to the correlation of REU and kinetics properties, we were also
%interested in the relationship of Rosetta scores and different melting
%temperatures (T\textsubscript{m}1 and T\textsubscript{m}2). While we found each
%mutations did limited effects due to the similarity of amino acids, there was
%weak correlation between Rosetta score and protein melting temperatures. As the
%CD and nano-DSC demonstrate overviews of protein structures, the pattern of
%RMSD and hydrogen bond from Rosetta provide more details and insights of
%interactions.
% -------------------------- deleted content
%After the one hour incubation at, the residual
%activities of wild-type and PTE variants decreased. While wild-type PTE retains
%only 38\% initial activity, F51L exhibited sightly higher
%k\textsubscript{cat}/K\textsubscript{M} at 50\%. We are comparing hydrolysis of
%paraoxon among variants thermo-stabilities from \SI{25}{\celsius} to
%\SI{55}{\celsius} with \SI{10}{\celsius} increment. The wild-type PTE lost 41\%
%of its activity after \SI{45}{\celsius} incubation and only exhibited
%\SI{65000}{\per\Molar\per\second} after treated at \SI{55}{\celsius}. The
%results followed the pattern published before.\cite{Yang2014a} 
%
%While the majority of variants retain their hydrolysis
%efficiency, three variants, however, lose more than 30\% of
%k\textsubscript{cat}/K\textsubscript{M}. F304L exhibited only 32\% of wild-type
%PTE activity at \SI{25}{\celsius} (k\textsubscript{cat}/K\textsubscript{M} =
%\SI{55000}{\per\Molar\per\second}). F327L and F335M, at the same time, reduce
%the paraoxon hydrolysis efficiency to 69\% and 60\%, respectively.
%
%Focusing on the thermo-stability, we then compare the variants activities at
%\SIlist{35;45;55}{\celsius}. While F216L loses almost half (~48\%) of its
%\SI{25}{\celsius} activity, the rest of seven variants retain more than 70\% of
%their own \SI{25}{\celsius} activities. However, upon elevated temperature at
%\SI{55}{\celsius}, three variants, F304L, F327L, and F335M, were inactivated in
%comparison with other variants. 
% --------------------------
%
%Previously, our lab demonstrated enhanced
%thermo-stability and shelf-life of PTE.\cite{Yang2014a, Baker2011b} With the
%incorporation of phenylalanine analogs (\emph{p}FF,
%\emph{p}-flourophenylalanine), Baker \emph{et al.} demonstrated that
%\emph{p}FF-PTE exhibited higher residual activities against OP and non-OP
%substrates. To further stabilized PTE, we mutate phenylalanines positions to
%study the individual effect on PTE. With the comparison of paraoxon hydrolysis
%of F51L, F150M, F216L, F304L, F306L, F327L, F335M, and F357L, we report
%and evaluate the performance of computational modeling tool, Rosetta, for the
%screening and analysis of PTE. We discover that three specific Phe
%residues, F304L, F327L, and F335M, greatly impact the activity of PTE. Notably,
%we also discover that F306L is stabilized both from experimental and simulated
%results. With the cutoff value set up in Rosetta, it is of great advantage to
%screen potential PTE mutants. 
%
%F327L exhibited only \SI{69}{\percent} of wild-type kinetics at
%\SI{25}{\celsius}, we looked into the differences in its structure. Similar to
%F216L, this F327L mutation affects simply 9 residues in PTE structure. But, the
%introduction of this mutations does affect the hydrogen bonding pattern. There
%seems higher probability of hydrogen bonding between N321 and H257, and this
%changed interaction is only observed in F327L. While F335 is swapped with
%methionine, there are more than 50 residues changing their orientations. While
%F335M is introduced into PTE, the large group residues, H254, H257, and M317,
%are both affected. The changes of these key residues decrease the paraoxon
%catalytic efficiency to 60 \% at \SI{25}{\celsius}
%(k\textsubscript{cat}/K\textsubscript{M} \SI{102000}{\per\Molar\per\second}).
%
% reverse primer (5$'$-G AGA CTT CGC CCA \emph{GAC} TGT GAC TGA GTG-3$'$)
% reverse primer (5$'$-CAT CTC CTT GAG TGT GTC AAG \emph{TAC} GAC GCA CTC TA-3$'$)
% reverse primer (5$'$-CGG CGG TAA \emph{AAT} CTC AGG CTT CCG A-3$'$).
% reverse primer (5$'$-CAA AGC TTA CTG ACC GAC \emph{GAC} CCC AAA AGC TC-3$'$).
% reverse primer (5$'$-CTG ACC GAC AAG CCC \emph{GAC} AGC TCG ATA CAG-3$'$).
% reverse primer (5$'$-CCC TAC CGG \emph{AAT} TAA GGT GAC TCT C-3$'$)
% reverse primer (5$'$-C TCT CAC TAG GGT \emph{GAC} GAT GCT CTC TTC C-3$'$).
% reverse primer (5$'$-GA TTG GGC CGC GCC \emph{AAT} AAC AGT GGC TGG AAC-3$'$).
%
%Protein stabilities have been studied in decades. Shoichet \latin{et al.}
%determined the relationship between stability and functionality of T4 lysozyme
%via mutagenesis\cite{Shoichet1995}. Previously, Baker \latin{et al.}
%incorporated \emph{p}-fluorophenylalanine into PTE and demonstrated the
%enhanced residual activity among its substrates, paraoxon, chlorpyrifos, and
%2-naphthyl-acetate (2NA).\cite{Baker2011b} We use circular dichroism (CD) here
%to evaluate the secondary structures of PTE variants. By comparing the
%ellipticity at 208 nm and 222 nm, most of variants are structured as wild-type
%PTE. After \SI{10}{\micro\Molar} proteins were incubated at \SI{55}{\celsius}
%for one hour, samples were then cooled down to room temperature for CD
%measurements. Wild-type PTE result was consistent with previous studies.
%\cite{Yang2014a, Baker2011b} Upon elevated temperature, wild-type PTE lost the
%majority of its secondary structure.
%
%The polymerase chain
%reaction (PCR) parameters were set as follow for 18 cycles: initial
%denaturation in \SI{95}{\celsius} for 30 seconds, sequential denaturation in
%\SI{95}{\celsius} for 30 seconds, annealing in \SI{55}{\celsius} for 1 minute,
%and extension in \SI{68}{\celsius} for 4 minutes. The mixture was then
%incubated \SI{37}{\celsius} overnight with DpnI to digest methylated parent DNA
%strands, which lack the desired mutation.
%
%However, the phenylalanines outside the dimer interface may not be necessary
%for stability. In fact, Schwehm \emph{et al.} demonstrated the effects of
%hydrophobicity from the solvent-exposed side chains. While it was argued that
%the increased hydrophobicity at the solvent-exposed residue destabilized
%proteins,\cite{Pakula1990,Mollah2003,Herrmann1997} Schwehm chose to conduct 47
%phenylalanine substitutions on the surface of nuclease. Results demonstrated
%the content-dependency among these substitutions. 
%
%From our study, we affirm this as five (F51, F150, F216, F306, F357) of the
%eight phenylalanine substitutions did not negatively impact stability and
%function. In fact, F306L demonstrated the improved stability and activity
%relative to wild-type PTE. 
%
%  The
%significant change of enthalpy due to a single mutation was also discovered in
%Sauer group.\cite{Hecht1984a} Due to factors including hydrophobicity, packing
%interactions, and introduction or removal of hydrogen bonds, the thermal
%stability would depend on the contents of mutations.
%
%
% While F327L had also
%demonstrated change in this region, the magnitude of deviation was greater
%F306L by 70\% (Figure \ref{fgr:rmsd}). 
%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\printbibliography[heading=subbibliography]

\end{refsection}
