\chapter{Improved Stability and Half-Life of Fluorinated Phosphotriesterase
Using Rosetta} 
\label{chap:uaa}

\begin{refsection}

\section{Introduction}

\subsection{Protein Engineering}
\label{sec:protein-engineering}

Proteins (Greek word \emph{proteins}, meaning “of
primary importance") are biomolecules composed of 22 amino
acids\cite{Nelson2005}. The amino acids are linked together by amide bonds form
different secondary structures.  The amino acids within a chain are linked by
peptide bonds. Each amino acid consists of a backbone part, and a side chain of
a amino acid contributes to the unique property of a protein. The primary
structure of a protein refers to the linear form of amino acids in the
polypeptide chain\cite{Sanger1945}. Secondary structures are defined by the
pattern of hydrogen bonds of the protein, including $\alpha$-helix,
$\beta$-sheet, and turns\cite{PAULING1951}. The $\alpha$-helix is a spiral
conformation with backbone amine group donating a hydrogen bond to the backbone
carbonyl group, which leads to an overall dipole moment from the individual
micro-dipoles of the carbonyl groups\cite{Hol1978}. This type of  hydrogen bond
is formed every 4 amino acid residues, and every complete turn of the helix is
only 3.6 amino acid residues. Sheridan \latin{et al.} further quantified the
destabilized array of parallel helices of
\SI{83.6}{\joule\per\mol}\cite{Sheridan1982}. (See Figure
\ref{fig:alpha-helix-stabilization})
% -------------------------- fig-dipole-of-alpha-helix
\begin{figure}[h!] \centering \includegraphics[width=0.45\textwidth]{fig1_12}
    \caption[Schematic illustration of the recurrent 4-$\alpha$-helical
    structure motif. Four sequentially connected $\alpha$-helices are packed in
an approximately 4-fold symmetric array.]{Schematic illustration of the
    recurrent 4-$\alpha$-helical structure motif. Four sequentially connected
    $\alpha$-helices are packed in an approximately 4-fold symmetric
    array\cite{Sheridan1982}.} 
    
    \label{fig:alpha-helix-stabilization}
\end{figure}
% -------------------------- fig*

The second regular form of secondary structure in $\beta$-sheet, including
parallel and anti-parallel $beta$-sheets\cite{Astbury1932}. This structure
occurs when two segments of a peptide chain overlap one another and form a row
of hydrogen bonds with each other. 

Proteins are essential to cell functionalities and metabolism. Recently,
Proprotein convertase subtilisin/kexin type 9 (PCSK9), one of the serine
endoproteases, was found to regulate low density lipoprotein (LDL) in liver
cells\cite{Lambert2012,Horton2007}. Autoimmune diseases, like Celiac
disease\cite{Dessein2008} or Crohn's disease\cite{Sollid2005,
Meize-Grochowski}, are found associated with antibodies - a large, Y-shape
protein produced by plasma cells. Overall, proteins play a crucial role within
cells and make up ~55\% the dry weight of an \emph{Escherichia coli}
cell\cite{Neidhardt1990}. (Table \ref{tab:protein-app})
% ------------------------- table-protein-examples
\begin{table}[h!]
    \centering
    \begin{tabular}{ lll }
        \hline
        Protein & Usage & References \\
        \hline
        
        PCSK9 & serine endoprotease & \cite{Lambert2012,Horton2007} \\
        IgG & antibody & \cite{Sollid2005, Meize-Grochowski} \\
        PCAF & histone acetyltransferase & \cite{Mehta2011b} \\
        Erythropoietin & hormone & \cite{Haroon2003,Siren2001} \\
        Collagen & n.a & \cite{Bella1995} \\
        Thaumatin & sweetener & \cite{Green1999} \\
        PSA & biomarker & \cite{Crawford2014} \\
        Mannanases & detergent & \cite{Kirk2002} \\
        Polymerase & polymerase chain reaction & \cite{Berg2002} \\ 
        
        \hline
    \end{tabular}
    \caption[Examples of proteins and their applications.]{Examples of proteins
    and their applications.} \label{tab:protein-app}
\end{table}
% ------------------------- table*
 
One of well-known roles of protein is as an enzyme. Enzymes mediate chemical
reactions and serves different functions biological systems. In the  
\emph{ENZYME} database, there are roughly 5000 different enzymes information
stored in the repository.  Enzyme-catalyzed reactions were initially studied in
the 19th century\cite{AthelCornish-Bowden2012}. Compared to chemical catalysts,
enzymes are highly specific and enhance the rate of reaction by
\SI{e8}-fold over non-catalyzed reaction\cite{Stryer1995}. These
are due to the facts that (a) enzymes are specific; (b) the reduction of
activation energy. (See Figure \ref{fig:enzyme-intro}) First, binding of enzyme
and its substrate forms a low energy complex (ES). While the complex lowers the
transition state, less energy is needed to achieve the reaction compared to the
un-catalyzed reaction (ES‡). Finally the enzyme-product, EP,  dissociates and
hence releases the products. The other great example also demonstrates how
enzyme enhances the reaction by \SI{e17}-fold over the
un-catalyzed reaction in the case of decarboxylase\cite{Radzicka1995}. Without
taking 78 million years for decarboxylation, Radzicka \latin{et al.} identified
Orotidine 5'-phosphate decarboxylase (OMP) as an extremely proficient enzyme,
and further estimated the dissociation constant of less than
\SI{5e-24}{\Molar}.
% -------------------------- fig-enzyme-catalysis
\begin{figure}[h!] 
    \centering \includegraphics[width=0.8\textwidth]{fig1_14}
    \caption[The energies of the stages of a chemical reaction. Uncatalysed
    (dashed line), substrates need a lot of activation energy to reach a
transition state, which then decays into lower-energy products. When enzyme
catalysed (solid line), the enzyme binds the substrates (ES), then stabilizes
the transition state (ES‡) to reduce the activation energy required to produce
products (EP) which are finally released.]{The energies of the stages of a
    chemical reaction. Un-catalysed (dashed line), substrates need a lot of
    activation energy to reach a transition state, which then decays into
    lower-energy products. When enzyme catalysed (solid line), the enzyme binds
    the substrates (ES), then stabilizes the transition state (ES‡) to reduce
    the activation energy required to produce products (EP) which are finally
    released\cite{Berg2002}.} 
    \label{fig:enzyme-intro}
\end{figure}
% -------------------------- fig*

Protein engineering involves the design of proteins with new or
desirable functions. Ulmer \latin{et al.} defined the protein engineering back
to 80s, including the different prospects of protein engineering, like X-ray
crystallography, chemical DNA synthesis, computer modelling, and protein
folding. The combination of crystal structure and chemistry information was
emphasized as a powerful approach to design proteins with desirable
properties\cite{Ulmer1983}. Wiseman \latin{et al.} later adapted the concept
and gave examples for protein engineering, including the improved proteases for
the detergent industry\cite{Wiseman1993,Harwood1992} and imobilized lactase for
food precessing\cite{Wiseman1993}. The regular protein engineering methods are
listed in Table. \ref{tab:protein-engineering}. The catalytic promiscuity of
protein engineering are also important concepts. This involves the property
that is defined as the ability to catalyse more than one chemical reaction in
the acitive of a protein\cite{Kazlauskas2005}. While Kazlauskas \latin{et al.}
focused on fluorescent proteins based on protein domains bound to natural
intracellular cofactors (FMN, bilirubin or biliverdin), these proteins exhibits
unique properties - such as oxygen-independent maturation - for novel
applications. (Figure \ref{fig:protein-engineering-example})
% ------------------------- table-protein-engineering
\begin{table}[h!]
    \centering
    \begin{tabular}{ ll }
        \hline
        Method & Reference(s) \\
        \hline
        
        Rational design & \cite{Arnold1993}, Section\ref{sec:rational-design} \\
        Site-directed mutagenesis & \cite{Antikainen2005} \\
        Random mutagenesis & \cite{Wong2006, Jackson2006, Labrou2010} \\
        DNA shuffling & \cite{ Jackson2006, Antikainen2005} \\
        Molecular dynamics & \cite{Anthonsen1994} \\
        Peptidomimetics & \cite{Venkatesan2002} \\
        Phage display technology & \cite{Sidhu2007,Chaput2008} \\
        \emph{De novo} enzyme engineering & \cite{Golynskiy2010} \\

        \hline
    \end{tabular}
    \caption[Examples of methods used for protein engineering.]{Examples of
        methods used for protein engineering\cite{Antikainen2005}.}
        \label{tab:protein-engineering}
\end{table}
% ------------------------- table*

% -------------------------- fig-protein-engineering
\begin{figure}[h!] \centering \includegraphics[width=0.6\textwidth]{fig1_15}
    \caption[Biliverdin- binding fluorescent proteins (IFP1.4, iRFP, IFP2.0,
    etc.).]{Biliverdin- binding fluorescent proteins (IFP1.4, iRFP, IFP2.0,
        etc.)\cite{Kazlauskas2005}.} \label{fig:protein-engineering-example} 
\end{figure}
% -------------------------- fig*

\subsubsection{Rational Design}
\label{sec:rational-design}

Rational design provides an effective approach to design proteins of interested
when the structure and mechanism are well-known. Quillin \latin{et al.} adapted
the concept and modified His64 of sperm whale myoglobin. The superposition of
deoxy and carbonmonoxy structures reveals  the conformation of the Fe-C-O
complex, and further showed the insight of the mechanism of ligand
binding\cite{Quillin1993}. The most classical method used for rational design
is site-directed mutagenesis (SDM)\cite{Arnold1993}. With two common methods
for SDM, overlap extension method and whole plasmid single round PCR method, an
intentional change is introduced to a DNA sequence. The first method involves
two set of primer pairs, where one primer of each set includes the mutant codon
within the DNA sequence (Figure \ref{fig:sdm-1})\cite{Ho1989}. Upon the first
polymerase chain reaction, two double-stranded DNA products are obtained with
these four primers. With further denaturation and annealing,two hetero-duplexes
-each with desired muatagenic codon - are formed. DNA polymerase is then used
for second PCR to amplify the desired DNA\cite{Ho1989}. 
% -------------------------- fig-sdm-1
\begin{figure}[h!] \centering \includegraphics[width=0.6\textwidth]{fig1_16}
    \caption[Schematic diagram of site-directed mutagenesis by overlap
        extension. The double-stranded DNA and synthetic oligos are represented by lines
        with arrows indicating the 5’-to-3’ orientation. The site of
        mutagenesis is indicated by the small black rectangle. Oligos are
        denoted by lower-case letters and PCR products are denoted by pairs of
        upper-case letters corresponding to the oligo primers used to generate
        that product. The boxed portion of the figure represents the proposed
        intermediate steps taking place during the course of reaction (3),
        where the denatured fragments anneal at the overlap and are extended 3’
        by DNA polymerase (dotted line) to form the mutant fusion product. By
        adding additional primers ‘a’ and ‘d’ the mutant fusion product is
        further amplified by PCR.]
        {Schematic diagram of site-directed mutagenesis by overlap extension.
            The double-stranded DNA and synthetic oligos are represented by lines
            with arrows indicating the 5’-to-3’ orientation. The site of
            mutagenesis is indicated by the small black rectangle. Oligos are
            denoted by lower-case letters and PCR products are denoted by pairs
            of upper-case letters corresponding to the oligo primers used to
            generate that product. The boxed portion of the figure represents
            the proposed intermediate steps taking place during the course of
            reaction (3), where the denatured fragments anneal at the overlap
            and are extended 3’ by DNA polymerase (dotted line) to form the
            mutant fusion product. By adding additional primers ‘a’ and ‘d’ the
            mutant fusion product is further amplified by PCR\cite{Ho1989}.}
            \label{fig:sdm-1} 
\end{figure}
% -------------------------- fig*
The second method is developed as a kit, QuikChange Site-Directed Mutagenesis,
by Stratagene\cite{Arnold2003}. It requires one set of primers with the desired
mutation which are complementary to the opposite strands of a double-stranded
DNA template. While PCR takes place, both strands of the template are
replicated. After PCR products are collected. DpnI methylase is used for
digestion in order to the mutant gene.  Upon transformation into competent
cells, a circular, mutated plasmid DNA is then obtained\cite{Antikainen2005a}.
(Figure \ref{fig:sdm-2})
% -------------------------- fig-sdm-2
\begin{figure}[h!] \centering \includegraphics[width=0.6\textwidth]{fig1_17}
    \caption[Whole plasmid, single-round PCR produces the desired mutation
    directly from a plasmid template: an arrows represents a primer, X
    represents a mutagenic codon, a thick line represents the template plasmid
    DNA, and a thin line represents newly synthesized DNA.]{Whole plasmid,
        single-round PCR produces the desired mutation directly from a plasmid
        template: an arrows represents a primer, X represents a mutagenic
        codon, a thick line represents the template plasmid DNA, and a thin
        line represents newly synthesized DNA\cite{Antikainen2005a}.}
        \label{fig:sdm-2}
\end{figure}
% -------------------------- fig*

In addition to the conventional methods of rational design, computational
protein design tools are emerging. Computational protein design principles
are based on the combination of a force field and a search algorithm to
identify the promising protein mutations with a given protein structure
\cite{VanDerSloot2009a}. At the selected positions, algorithm mutate the
original amino acid to other amino acids and render new conformations. The
energy of the structure is then calculated after optimization of the side-chain
and backbone conformations with the interacting amino
acids\cite{VanDerSloot2009a}. (Figure \ref{fig:cpd})
% -------------------------- fig-CPD
\begin{figure}[h!] \centering \includegraphics[width=0.6\textwidth]{fig1_18}
    \caption[The basic principle of computational protein design algorithms;
        (A) at the positions considered for design the algorithm ‘mutates’ to
        all the other naturally occurring amino acids by replacing the original
        amino acid with discrete conformations of the substituting amino acid
        (rotamers); (B) favorable — low energy — substitutions are retained;
        (C) unfavorable - high energy — substitutions are discarded.]{The basic
            principle of computational protein design algorithms; (A) at the
            positions considered for design the algorithm ‘mutates’ to all the
            other naturally occurring amino acids by replacing the original
            amino acid with discrete conformations of the substituting amino
            acid (rotamers); (B) favorable — low energy — substitutions are
            retained; (C) unfavorable - high energy — substitutions are
            discarded\cite{VanDerSloot2009a}.} 
    \label{fig:cpd} 
\end{figure}
% -------------------------- fig*
Bolon \latin{et al.} used the Connolly algorithm\cite{Connolly1983} to
calcualte the side-chain accessible surface areas, and further identified
identify the thioredoxin rotamers that were likely to be energetically
favorable\cite{Bolon2003}. Hence, such tools speed up the developments of
rational design. Otten \latin{et al.} and Moss \latin{et al.} demonstrate that
the algorithms alter enzymes activity \cite{Ottern2010} and the growth hormone
stability\cite{Moss2009}. Different computational tools has also been
employed in the redesign of affinity of a wide variety of protein–protein
interactions\cite{Selzer2000,Reina2002}. Selzer \latin{et al.} engineer the
protein TEM1 $\beta$-lactamase by using \emph{PARE}, and the affinity between
TEM1 and protein inhibitor BLIP is enhanced 250-fold\cite{Selzer2000}. (Figure
\ref{fig:tem1})
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.6\textwidth]{fig1_19}
    \caption[Validating PARE for different proteins. Calculations of
        k\textsubscript{on} were performed using the program PARE. The proteins
        studied were barnase with barstar (crosses), hirudin with thrombin
        (filled circles), and AChE with fasciculin (open squares). The data
        points are for wild type and 32 mutant complexes.]{Validating PARE for
            different proteins.  Calculations of k\textsubscript{on} were
            performed using the program PARE. The proteins studied were barnase
            with barstar (crosses), hirudin with thrombin (filled circles), and
            AChE with fasciculin (open squares). The data points are for wild
            type and 32 mutant complexes.\cite{Selzer2000}.} 
            \label{fig:tem1} 
\end{figure}
% --------------------------
In addition to re-design, algorithms can also be employed in different
applications, including alanine scanning (Massova and Kollman, 1999; Kortemme
et al., 2004b) enzyme–substrate (Bolon and Mayo, 2001; Jiang et al., 2008;
Rothlisberger et al., 2008; Chen et al., 2009; Damborsky and Brezovsky, 2009)
and protein–nucleic acid interactions (Ashworth et al., 2006) have been
modified. Numerous computational tools have already shown promising results for
the developments of protein designs. 

\subsubsection{Rosetta Design}

% describe the advantage, esp. the ability to explore proteins w/ uaa
While computational tools are widely used for protein
engineering\cite{Rothlisberger2008,DiMaio2011a,Korkegian2005,
Leaver-Fay2013a,Leaver-Fay2011,Drew2013a,Kaufmann2010,Rohl2004}, Rosetta suite
was developed in University of Washington. Baker \latin{et al.} adapted this
suite for prediction of three-dimensional structure of
proteins\cite{Leaver-Fay2011}. This suite provides a handful of protocols for
analyzing and mutating protein structures.  The simulation replies heavily on
knowledge-based potentials. It is a suite of libraries and tools for
macromolecular ligand docking, to thermo-stabilize proteins, to design a
hydrogen-bond network, to design novel protein folds, to create novel protein
interfaces, and to design enzymes, including some containing unnatural amino
acids in the active sites (Figure \ref{fig:rosetta-intro}).

% -------------------------- fig-rosetta
\begin{figure}[h!] \centering \includegraphics[width=0.7\textwidth]{fig1_01}
    \caption[The scheme of scoring function of Rosetta. 1: Lennard-Jones
    Potential; 2: implicit solvent model; 3: hydrogen bonding; 4:
electrostatics; 5: PDB drived torsion potential.]{The scheme of scoring
    function of Rosetta. 1: Lennard-Jones Potential; 2: implicit solvent model;
3: hydrogen bonding; 4: electrostatics; 5: PDB derived torsion potential.}
\label{fig:rosetta-intro}
\end{figure}
% -------------------------- fig*

The scoring function sums the weighted terms that represent the physical or
probabilistic pairwise interactions\cite{Rohl2004}, including Lennard-Jones
potential, implicit solvent model\cite{Lazaridis1999}, orientation-dependent
hydrogen bond term\cite{Kortemme2003}, sidechain and backbone torsion
potentials derived from the PDB, short-ranged knowledge-based electrostatic
term, and reference energies for each of the 20 amino acids that model the
unfolded state.

% --------------------------
By using Rosetta for design with UAAs (See \ref{sec:uaa-intro}), we gain access to a
wide variety of kinematic and optimization based methods for exploring backbone
and side chain configurations. Conformation- al searches of backbone degrees of
freedom are typically performed using small perturbations to the backbone
dihedral angles, fragment insertions, backrub movements, or using more
sophisticated procedures like using robotic arm motion planning inspired
loop-closure algorithms [19–21].

c arm motion planning inspired loop-closure algorithms [19–21]. Conformational
search- es of side chain degrees of freedom are performed in a discrete space
of high probability side chain conformations typically encoded as
backbone-dependant rotamer libraries. Rotamer libraries are lists of commonly
seen side chain dihedral angles [22] accompanied by the probability of
observing each rotamer in naturally occurring proteins. In Rosetta, the side
chain coordinates are constructed using dihedral angles from the rotamer
library and idealized bond lengths, bond angles, and non-x dihedrals [23].Amino
acid rotamers are not observed with equal frequency in large databases of
experimental structures. The probability of seeing a given rotamer given its
local structure context can be used to compute a pseudo-energy that represents
the internal energy of the amino acid.
% --------------------------

\subsubsection{Protein Engineering Bearing Unnatural Amino Acids}
\label{sec:uaa-intro}

The genetic code of deoxyribonucleic acid (DNA), a gene, is transcribed and
translated into a sequence of polypeptide through the central
dogma\cite{CRICK1970}. The DNA information is first transcribed into messenger
ribonucleic acid (mRNA) through RNA polymerase and transcription factors. Once
mRNA is ready for translation, the ribosome facilitates the decoding of mRNA.
Aminoacyl-tRNA synthetases (aaRS) are, at the same time, carrying the paired
amino acids to their transfer RNAs (tRNAs). The aminoacylated tRNAs are then
used for synthesis of polypeptide chain. The specific codon that consists of
three nucleotides corresponds to a single amino acid. The start codon, AUG on
mRNA sequence, is recognized by initial tRNA. While synthesis of peptide ends
up with stop codon, each pair of codon is recognized by amino acid paired
tRNA\cite{Sadava2006}.

% background on proteins designed with uaa
% focus on a) activity b)specificity c) stabilities
% --------------------------from Princeton and Caltech thesis
The fluorinated protein has been studied extensively in the hope that it will
provide benefits to protein engineering, including properties of increased
hydrophobicity, thermostability, and chemical denaturant
stability\cite{Voloshchuk2007b}. The replacement of hydrophobic amino acids
like leucine and phenylalanine with their fluorinated analogs has provided
content-dependent results.

Budisa group showed that enzymes change the specificities with the
incorporation of unnatural amino acids\cite{Budisa2006}. Biocatalysis with UAAs
are selectively active to different chimerical compounds. 
% --------------------------end of new

\subsection{Incorporation of Unnatural Amino Acids} 
\label{sec:rsi-intro}

% --------------------------fig
\begin{figure}[h!] \centering \includegraphics[width=0.6\textwidth]{fig1_10}
    \caption[Princilple mechanism of NCL.]{Principle mechanism of NCL.} 
    \label{fig:ncl-intro} 
\end{figure}
% --------------------------fig

Several methods have been developed for the incorporation of unnatural amino
acids into proteins, which can be categorized under synthetic and biosynthetic
approaches. \latin{in vivo} and \latin{in vitro} site-specific
incorporation,\cite{Cellitti2008,Hassan2008} and residue-specific incorporation
(Figure \ref{fig:rsi})\cite{Johnson2010}.  In SPPS, activated amino acids are
immobilized on a solid support and synthesized step-by-step in the reactant
solution. This method is convenient for the introduction of functional groups
into peptides, but it is still restricted to the yield and the expense of
peptides. For example, if each coupling step has 99\% yield, a 26-amino acid
peptide would be synthesized in 77\% final yield.  To synthesis longer chain
peptides and proteins bearing UAAs, biosynthetic methods have been developed.
There exists three contemporary methods to biosynthetically incorporate
non-natural amino acids into proteins: site-specific incorporation (SSI)(Figure
\ref{fig:ssi-intro}), residue-specific incorporation (RSI), and multi-site
specific incorporation (MSI). Bain \latin{et al.} have developed a general
approach for the \latin{in vitro} synthesis of proteins\cite{Bain1991}.  The
approach relies on the suppression of an amber termination codon (UAG) in the
mRNA by an amber suppressor tRNA chemically charged with the amino acid analog.
(Figure \ref{fig:ssi-intro}) This method has been well studied and developed in
research of protein structures and functions\cite{Martoglio1995,Eichler1997}.

% --------------------------fig
\begin{figure}[h!] \centering \includegraphics[width=1.0\textwidth]{fig1_09}
    \caption[Schematic representation of in vitro SSI via amber
    suppression.]{Schematic representation of in vitro SSI via amber
    suppression.} 
    \label{fig:ssi-intro} 
\end{figure}
% --------------------------fig

Methods to incorporate amino acid analogues site-specifically into proteins
\latin{in vivo} greatly expand research of unnatural amino acids. We are not
only able to synthesize large amounts of protein, but capable of overcoming
potential problems including post-translational modifications. An \latin{in
vivo} site-specific method to UAA incorporation was developed by Schultz and
coworkers\cite{Wang2001,Wang2002}. A stop codon at the position of interest is
encoded in the mRNA. For \latin{in vivo} site-specific UAA incorporation, an
orthogonal aminoacyl-tRNA synthetase charges an orthogonal tRNA with particular
UAA, and the suppressor tRNA would help the incorporation of UAA by recognition
of a stop codon. As cells contain 20 aminoacyl-tRNA synthetase/tRNA pairs, a
new one is required for UAA incorporation. An orthogonal aminoacyl-tRNA
synthetase/suppressor tRNA pair based on a TyrRS/tRNA\textsuperscript{Tyr} pair
in the \emph{Methanococcus jannaschii} has been engineered for use in \emph{E.
coli} for the incorporation of tyrosine analogs\cite{Wang2001}.

% -------------------------- fig
\begin{figure}[h!] \centering \includegraphics[width=1.0\textwidth]{fig1_03} 
    \caption[Illustration  of NAA incorporation via (a) RSI using auxotrophic
    strain, (b) engineering additional copies of endogenous AARS, (c) expanding
the AARS binding pocket and (d) shrinking the AARS editing
pocket.]{Illustration  of NAA incorporation via (a) RSI using auxotrophic
strain, (b) engineering additional copies of endogenous AARS, (c) expanding the
AARS binding pocket and (d) shrinking the AARS editing pocket.} \label{fig:rsi} 
\end{figure}
% -------------------------- fig*

As an alternative to site-specific incorporation, residue-specific
incorporation has been developed in which a natural amino acid is replaced with
an UAA. Auxotrophic strains or organisms that cannot biosynthesize a particular
natural amino acid, has been used to introduce multiple UAAs throughout the
protein sequence. UAAs that are isosteric to natural amino acids are capable of
being recognized by the natural aminoacyl-tRNA synthetase (aaRS), charging the
appropriate tRNA enabling the introduction of UAA into the protein sequence
without alteration of the biosynthetic machinery. (Figure \ref{fig:rsi})
However, to introduce UAAs with gross differences from the natural amino acids,
further engineering of the aaRS is required. To incorporate refractory
methionine analogs, Tirrell and coworkers engineered additional copied of the
methionyl-tRNA synthetase (MetRS) by adding the MetRS gene under constitutive
promotor\cite{Kiick2000}.  Alternatively, Schimmel and coworkers mutated
editing pocket of valyl-tRNA synthetase (ValRS) to facilitate the incorporation
of analogs that normally would not be accepted by endogenous
aaRS\cite{Doring2001}. Finally, Kast and coworkers generated a mutated
phenylalanyl-tRNA synthetase (PheRS), ePheRS* under a constitutive promoter,
with a large binding pocket (T251G) and showed relaxed
specificity\cite{Kast1991}.

\subsubsection{Fluorinated Amino Acids In Proteins} 
\label{sec:faa-intro}

% -------------------------table
\begin{table}[h!]
\centering
\begin{tabular}{ llll }
  \hline
  Bond & Length & Van der Waals radius & Total size \\
  \hline

  C-H & 1.09 & 1.2 & 2.29 \\
  C-F & 1.35 & 1.7 & 2.82 \\

  \hline
\end{tabular}
\caption[(A) physical properties of the C-F bond. (B) comparison of C-H and C-F
bonds, van der Waals radius, and total size]{(A) physical properties of the C-F
bond. (B) comparison of C-H and C-F bonds, van der Waals radius, and total
size\cite{Tang2001,Odar2015}}
\label{tab:c-fbond}
\end{table}
% -------------------------table

% Describe the properties of faa in the proteins.
Fluorinated amino acids (FAAs), represent a unique class of UAAs. They have
different bond energies, electron distributions, and
hydrophobicity\cite{Biffinger2004} as compared to their hydrogenated
counterparts. With the electro-negativity of 4.0 on the Pauling scaling, the
C-F bond is highly dipolar while the hydrocarbon is less. The C-F bond is
roughly 0.24 {\AA} longer than C-H bond (Table
\ref{tab:c-fbond})\cite{Tang2001}. While in some cases the global replacement
of hydrophobic amino acids with fluorinated analogs has led to the
stabilization of protein structure\cite{Biffinger2004}, it has all been shown
that in some cases they can reduce the thermodynamic
stability\cite{Panchenko2006b}. The expansion of the genetic code has led to
the biosynthetic incorporation of a wide range of unnatural amino acids into
proteins\cite{Voloshchuk2010}. In particular, fluorinated amino acids (FAAs)
have been integrated into small coiled-coil
proteins\cite{Montclare2009b,Tang2001}, a range of
enzymes\cite{Voloshchuk2009,Panchenko2006b,Voloshchuk2007b,Mehta2011b,Hammill2007},
and biomaterials\cite{Yuvienco2012b}. Although incorporation of FAAs into a
target protein can lead to enhanced function or stability, in some cases loss
of activity or stability occurs, and further improvements to the artificial
protein have been made by rational mutagenesis\cite{Voloshchuk2007b} and
directed evolution strategies\cite{Montclare2006b}.
% fig1_04: add comparison of phe vs pFF

\subsection{Phosphotriesterase} 
\label{sec:pte-intro}

% This section focuses on PTE.
PTE is a homodimeric protein composed of two monomers, each of which contains a
metallo-active site. Phosphotriesterase (PTE) are enzymes, which hydrolyze
organophosphates (OPs) as well as synthetic esters (Figure
\ref{fig:pte-structure})\cite{Ghanem2005a}. The proenzyme form of PTE contains
29 amino acids signal peptide at the N-terminus. It is originally found as a
39kDa monomeric form in the solution\cite{Mulbry1989}. Later, the proenzyme of
PTE is engineered and expressed in the form of mature protein from \latin{E.
coli}. A ($\beta$/$\alpha$)\textsubscript{8} TIM-barrel structure forms the
monomeric PTE\cite{Roodveldt2005,Seibert2005}. The globular monomer is roughly
51\AA $\times$ 55\AA $\times$ 51\AA.  OPs are a synthetic class of small molecule
that irreversibly inactivate acetylcholinesterase (AChE), disrupting
neural transmission. AChE is an enzyme that degrades the neurotransmitter,
acetylcholine, at the neuromuscular junction in the cholinergic nervous system.
After the acetylcholine is hydrolyzed, the synaptic transmission would be
terminated. Inhibition of AChE lead to hyper-stimulation from toxic
accumulation of acetylcholine\cite{Soreq2001}. Army also adapted this protein
for chemical weapons neutralization \cite{Yang2014a}.

% --------------------------fig
\begin{figure}[h!] \centering \includegraphics[width=1.0\textwidth]{fig1_02}
    \caption[Structure of and active site of phosphotriesterase: (A) Crystal
    structure of PTE (PDB 1HZY). Wild-type PTE consists of two monomers. Shown
in light blue is one of them, and dark blue is the other. Yellow dots represent
zinc atoms; (B) Paraoxon hydrolysis by PTE; (C) Small pocket residues are
labeled in red: G60,I106, L303, S308; large pocket residues are labeled in
blue: H254, H257, M317, while leaving group residues are labeled in grey: W131,
F132, F306, Y309.] {Structure of and active site of phosphotriesterase: (A)
Crystal structure of PTE (PDB 1HZY). Wild-type PTE consists of two monomers.
Shown in light blue is one of them, and dark blue is the other. Yellow dots
represent zinc atoms; (B) Paraoxon hydrolysis by PTE;  (C) Small pocket
residues are labeled in red: G60,I106, L303, S308; large pocket residues are
labeled in blue: H254, H257, M317, while leaving group residues are labeled in
grey: W131, F132, F306, Y309.}
\label{fig:pte-structure} 
\end{figure} 
% --------------------------fig

% add a) metal effect c) fluorinated pte.
Omburo \latin{et al.} showed that the active PTE contained metal ions in the
active site. \ch{Mn^{2+}}, \ch{Co^{2+}}, \ch{Ni^{2+}}, \ch{Cd^{2+}},
\ch{Zn^{2+}} help the hydrolysis of organophosphates\cite{Omburo1992a}. From
the spectrum of \ch{^{113}Cd}-NMR, the peaks indicated that PTE incorporated
two distinct metals in the active site\cite{Omburo1993}. Histidines, including
His55, His57, His201, and His230, interact metal ions at the active
site\cite{Benning2001a}. 

\subsubsection{Hydrolysis Mechanism of PTE}
% SN2 reaction and discuss binding pocket.

See Section \ref{sec:side-chain}

\subsubsection{Fluorinated PTE}
% describe PJB papper quickly here

In 2011, Baker \latin{et al.} showed improved thermo-stabilized PTE with the
incorporation of \emph{para}-fluoro-phenylalanine
(\emph{p}FF)]\cite{Baker2011b}. With residue specific incorporation (see
\ref{sec:rsi-intro}), all phenylalanines were replaced with \emph{p}FFs, and
\emph{p}FF-PTE exhibited residual activities to organophosphates.

\subsection{Scope of Work}

The primary goals of this work were to adapt Rosetta for phosphotriesterase.
Overall, with incorporation of \emph{p}FF into protein, we will be able
evaluate the performance of scoring function. In advance, we would evaluate the
shelf life and thermo-stability of phosphotriesterase.

\section{Methods}

\subsection{General}

\emph{DpnI} and dNTP were purchased from Roche. All other chemicals, including
\ch{NaCl}, sodium phosphates monobasic, sodium phosphate dibasic, were
purchased from Sigma or VWR. DNA sequence was confirmed by Eurofins MWG Operon.
96-well plates were purchased from Thermo Fisher Scientific (Waltham, MA).

\subsection{Recombinant Gene Construction}

pQE30-PTE was used as described before\cite{Baker2011b}. The pQE30-F104A plasmid
was prepared with forward primers (5’-GAT GTG TCG ACT \textbf{GCC} GAT ATC GGT
CG-3’, Fisher Scientific), reverse primers (5’-CG ACC GAT ATC \textbf{GGC} AGT
CGA CAC A-3’, Fisher Scientific). The polymerase chain reaction (PCR)
parameters were set as follow for 18 cycles: initial denaturation in
\SI{95}{\celsius} for 30 seconds, sequential denaturation in \SI{95}{\celsius}
for 30 seconds, annealing in \SI{55}{\celsius} for 1 minute, and extension in
\SI{68}{\celsius} for 4 minutes. The mixture was then incubated
\SI{37}{\celsius} overnight with DpnI to digest methylated parent DNA strands,
which lack the desired mutation. DNA sequence was further confirmed by Eurofins
MWG Operon. (See appendix for plasmid map)

\subsection{Protein Expression}
\label{sec:protein-expression-method}

Mutant and wild type plasmids were transformed into electro-competent \latin{E.
coli} phenylalanine auxotrophic strains (AF-IQ cells).[5] Electroporation was
done at \SI{25}{\micro\farad}, \SI{100}{\ohm}, 2.5 kV (Biorad Gene Pulser II).
Cells were plated on agar plates containing \SI{200}{\ug\per\mL} ampicillin,
\SI{34}{\ug\per\mL} chloramphenicol. A single colony was picked and grown in
medium (M9 medium supplemented with 0.2 wt \% glucose, \SI{35}{\mg\per\L}
thiamine, \SI{1}{\milli\Molar} \ch{MgSO4}, \SI{0.1}{\milli\Molar}\ch{CaCl2},
\SI{200}{\ug\per\mL} ampicillin, and \SI{34}{\ug\per\mL} chloramphenicol) with
\SI{20}{\mg\per\L} of 20 amino acids at \SI{37}{\celsius}, 300 r.p.m for 16
hours \SI{37}{\celsius} incubation.  Afterwards, \SI{250}{\mL} of M9 medium for
large-scale expression was innoculated 1:50 with the overnight culture.  After
optical density reached 1.0 at 600 nm, media shift was carried out by washing
the cells three times with \SI{4}{\celsius} 0.9\% \ch{NaCl}. Cells were then
transferred to M9 minimal medium containing either 20 amino acids or 19 amino
acids (-Phe). Ten minutes of \SI{37}{\celsius} incubation was adapted for
depriving of extra phenylalanine in the medium. \emph{p}FF-PTE and
\emph{p}FF-104A expression media were supplemented with and
\SI{3}{\milli\Molar} of \emph{p}FF and \SI{1}{\milli\Molar}
isopropyl-$\beta$-D-thiogalactopyranoside (IPTG) to induce protein expression.
\SI{1}{\milli\Molar} of \ch{CoCl2} was added in each post-induction medium.
After three hours incubation at \SI{37}{\celsius}, 300 r.p.m., the cells were
harvested by using 4000 r.p.m centrifugation at \SI{4}{\celsius} for 15 minutes
and then resuspended with \SI{20}{\milli\Molar} Tris-HCl,
\SI{500}{\milli\Molar} \ch{NaCl}, \SI{5}{\milli\Molar} imidazole, 10\% glycerol
(pH 8.0) and \SI{1}{\micro\Molar} \ch{CoCl2}. Cell lysate was immediately
sonicated for 1.5 minutes at \SI{4}{\celsius} and then a clarification spin was
performed (20, 000 g, \SI{4}{\celsius}, 30 minutes).  Clarified supernatants
were loaded into a \SI{5}{\mL} His Trap column (G.E Healthcare, Piscataway, NJ)
using AKTA FPLC purifier (G.E.  Healthcare, Piscataway, NJ).  Protein elution
was generated using elution buffer B (\SI{20}{\milli\Molar} Tris-HCl,
\SI{500}{\milli\Molar} sodium chloride, \SI{500}{\milli\Molar} imidazole (pH
8.0)).  The purified samples were then transferred for buffer exchange using
\SI{12}{\L} \SI{20}{\milli\Molar} phosphate buffer (pH 8.0).  Dialyzed protein
was subjected to kinetic assays immediately.

\subsection{PyRosetta Design}
\label{sec:pyrosetta-method}

Rosetta\cite{Leaver-Fay2011,DiMaio2011a} was used to generate a symmetric,
\emph{p}FF-incorporated PTE structure used by all simulations. The structure
(PDB code: 1HZY) of wild type PTE was used as the input. In addition to the
phenylalanine positions being mutated to \emph{p}FF, three positions in the
wild-type PTE sequence were mutated (K185R, D208G, and R319S) based on S5PTE to
generate \emph{p}FF-PTE.\cite{Roodveldt2005} Mutations were made using the
Rosetta \emph{fixbb} application and were followed by side chain repacking and
minimization. The amino acids directly interacting with the \ch{Co^{2+}} ions
are crucial in binding the necessary divalent cation for PTE activity[3], so
they were fixed in their native rotamers during repacking and minimization.
PyRosetta, a python interface to the Rosetta libraries[4], was used to make and
characterize point mutations. Every \emph{p}FF position was individually
mutated into any natural amino acid minus phenylalanine. To simulate a
mutation, a single \emph{p}FF position was mutated and neighboring amino acid
within \SI{10}{\AA} (as measured by C$\alpha$-C$\alpha$ atom distance) was
allowed to repack and minimize to accommodate the point mutation to fill in
potential high-cost-energy voids or to supplement the hydrophobicity, polarity,
or charge in the vicinity. For each \emph{p}FF position, 500 decoys were
generated. After the mutations were made, representative structures of each
mutation were chosen based on the overall stability of the enzyme, reflected by
the total score. The binding energy is the total energy minus the energy of
both monomers separated by \SI{1000}{\AA}. Point mutations were chosen based on
the difference between relative total and predicted binding energies of the
mutant and \emph{p}FF-PTE sequence. As above, amino acids directly interacting
with the \ch{Co^{2+}} ions were fixed in their native rotamers during repacking
and minimization. All Rosetta and PyRosetta calculations were done using the
\emph{score12} score function, and possessed extra rotamer sampling, including
the native rotamers.

\subsection{Thermo-stability and Secondary Structure of Phosphotriesterase}
\label{sec:thermo}

\subsubsection{Nano-DSC}
\label{sec:dsc-method}

Differential scanning calorimetry (Nano-DSC, TA instrument, USA) was performed
by using \SI{600}{\micro\L} (\SI{0.1}{\mg\per\mL}) of protein right after
dialysis into \SI{20}{\milli\Molar} sodium phosphate buffer (pH 8.0).
Measurements were conducted at a scan rate of \SI{1}{\celsius\per\minute} from
\SI{20}{\celsius} to \SI{70}{\celsius}.  Signals was blanked with buffer under
the same condition.  The observed diagram was then analyzed by using
NanoAnalyze software (TA instrument,USA).

\subsubsection{Circular Dichroism}
\label{sec:cd-method}

Circular Dichroism spectra were recorded on a JASCO J-815 Spectropolarimeter
(Easton, MD) using Spectra Manager software. Temperature was controlled at
\SI{25}{\celsius} using a Fisher Isotemp Model 3016S water bath. Proteins
concentrations were \SI{10}{\micro\Molar} in \SI{20}{\milli\Molar} phosphate
buffer (pH 8.0).  \SI{20}{\milli\Molar} phosphate buffer was used for blanking
signals. To calculate ellipticities, the following formula was
used(Eq.~\ref{eqn:CD-chap1}): 
\begin{equation}
    θmrw = MRW(θobs) / (10 * c * l) 
    \label{eqn:CD-chap1}
\end{equation}
where \emph{MRW} is the mean residue weight of the specific phosphotriesterase,
θobs is the observed ellipticities (mdeg), \emph{l} is the path length (cm),
\emph{c} is the concentration in \SI{}{\micro\Molar}. Spectra was recorded from
\SIrange{190}{250}{\nm} with a scan speed of \SI{1}{\nano\meter\per\minute}.

\subsection{Enzyme Kinetics}
\label{sec:kinetics-method}

The protein was diluted to a final concentration of \SI{30}{\nano\Molar} in
\SI{20}{\milli\Molar} sodium phosphate (pH 8.0) by using the extinction
coefficient \SI{29575}{\per\Molar\per\cm} for all proteins\cite{Gasteiger2005,
Pace1995}.  Reactions were monitored spectrophotometrically (Synergy H1,
BioTek, Winooski VT) at \SI{405}{\nm} for paraoxon (coefficient =
\SI{17000}{\per\Molar\per\cm})\cite{Baker2011b}. Reactions for paraoxon
(\SIrange{13}{104}{\micro\Molar}) was done in 0.2\% methanol.
K\textsubscript{M} and k\textsubscript{cat} values were determined by a
Lineweaver-Burk plot.\cite{Baker2011b} The equation used is shown below
(Eq.~\ref{eqn:MM-chap1}): 
\begin{equation} 
    \frac{1}{v} =
    \frac{K\textsubscript{M}}{V\textsubscript{max}}\times\frac{1}{S} +
    \frac{1}{V\textsubscript{max}} 
    \label{eqn:MM-chap1}
\end{equation}
where S represents substrate concentration; K\textsubscript{M} represents the
substrate concentration at which the reaction rate is half of
V\textsubscript{max}. The data reported is the average of three trials and the
error represents the standard deviation of those trials.

\subsection{MALDI-TOF Mass Spectrometry}

To determine level of \emph{p}FF incorporation, \SI{20}{\micro\liter} of
purified PTE, \emph{p}FF-PTE, F104A, or \emph{p}FF-104A was incubated with
\SI{12.5}{\ng\per\uL} of trypsin solution (in \SI{50}{\milli\Molar} of ammonium
bicarbonate) at \SI{37}{\celsius} overnight. \SI{2}{\uL} of 10\%
trifluoroacetic acid (TFA) was used to quench each reaction. Reaction was then
purified with a C\textsubscript{18} packed zip-tip (Millipore, Billerica, MA).
Tips were wetted in 50\% acetonitirile (ACN), equilibrated in 0.1\% TFA, and
eluted with 0.1\% TFA in 75\% ACN. Matrix was dissolved in \SI{10}{\mg\per\mL}
$\alpha$-cyano-4-hydrocinnamic acid (CCA) in 50\% ACN, 0.05\% TFA. Theoretical
trypsin digest were calculated from Peptide Mass
(www.expasy.org/tools/peptide-mass.html). Samples were added to the matrix at a
1:1 ratio and spotted on MALDI plate. Five standards were spotted separately
for calibration: angiotensin I (MW = \SI{1295.69}{\g\per\mole}), neurotensin
(MW = \SI{1671.92}{\g\per\mole}), ACTH (1-17) (MW = \SI{2092.09}{\g\per\mole}),
ACTH (18-39) (MW = \SI{2464.20}{\g\per\mole}), and ACTH (7-38) (MW =
\SI{3656.93}{\g\per\mole}).  Compass 1.4 for flex software was then used to
analyze the MALDI spectra (www.bruker.com/).

\section{Results and Discussion}

\subsection{PyRosetta Design of Phosphotriesterase}

Using PyRosetta (a python interface to the Rosetta
libraries)\cite{Leaver-Fay2011}, we then examined every \emph{p}FF position in
the model. Each was allowed to mutate to any amino acid except phenylalanine.
We then evaluated the mutated residue relative to the “native” \emph{p}FF,
based on the total energy and predicted binding energy of the two chains
(Section \ref{sec:pyrosetta-method}). This protocol was used to predict
stabilizing site-specific mutations as well as mutations that would remove
phenylalanine positions that would not tolerate \emph{p}FF substitution. (In
our residue-specific \emph{p}FF incorporation system we cannot control which
phenylalanine positions are fluorinated, and thus removing phenylalanine
positions that cannot tolerate fluorination could improve stability if
alternative natural amino acids score well at these positions.) For each
mutation, every neighboring amino acid within \SI{10}{\AA} of the C$\alpha$
atom of \emph{p}FF was allowed to repack to accommodate the point mutation, by
filling in potential high-cost-energy voids or to supplement the
hydrophobicity, polarity, or charge in the vicinity (Figure
\ref{fig:rosetta-pte}, Section \ref{sec:pyrosetta-method}).  Replacing the
phenylalanine at position 104 with \emph{p}FF created energy-costing clashes
with neighboring amino acids at the dimerization interface (Figure
\ref{fig:rosetta-pte}). As a dimer is known to be crucial for
function\cite{Baker2011b}, we sought to restabilize the interface with
incorporated pFF. Upon mutation of F104 to alternative natural amino acids, we
identified the variant \emph{p}FF-F104A, which exhibited improved packing in
the absence of the \emph{p}FF. (Figure \ref{fig:rosetta-pte}).

% --------------------------fig
\begin{figure}[h!] \centering \includegraphics[width=1.0\textwidth]{fig1_11}
    \caption[Structure of \emph{p}FF-F104A identified from Rosetta,
        highlighting \emph{p}FFs and F104A. Comparison of \emph{p}FF-F104A and
        \emph{p}FF-PTE with energy scores shows that \emph{p}FF-PTE exhibits
        steric clash with a neighboring residue.  The original structure (PDB
        ID: 1HZY) was mutated and modified by Rosetta and rendered by UCSF
    Chimera.]{Structure of \emph{p}FF-F104A identified from Rosetta,
    highlighting \emph{p}FFs and F104A. Comparison of \emph{p}FF-F104A and
    \emph{p}FF-PTE with energy scores shows that \emph{p}FF-PTE exhibits steric
    clash with a neighboring residue. The original structure (PDB ID: 1HZY) was
    mutated and modified by Rosetta and rendered by UCSF Chimera.}
    \label{fig:rosetta-pte}
\end{figure}
% --------------------------fig

\subsection{Biosynthesis of Proteins}

The \emph{p}FF-F104A variant and the \emph{p}FF-PTE parent were biosynthesized
by residue-specific incorporation with the phenylalanine auxotrophic
\emph{Escherichia coli} strain AFIQ\cite{Yang2014a}. As controls, the
non-fluorinated counterparts, PTE and F104A, were expressed under conventional
conditions. As expected, all four proteins exhibited good expression in the
presence of phenylalanine or \emph{p}FF. SDS-gel result showed that
over-expression of proteins and the purities after FPLC purification. (Figure
\ref{fig:sds-gel}) \emph{p}FF-F104A and \emph{p}FF-PTE exhibited 80\% and 92\%
incorporation, respectively, as determined by MALDI-TOF
mass spectrometry. (Figure \ref{fig:MALDI-fig}) Notably, purified yields of
\emph{p}FF-F104A were twofold higher than for \emph{p}FF-PTE, thus indicating
more soluble protein yield. (Figure \ref{fig:sds-gel})

% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=1.0\textwidth]{fig1_04}
    \caption[0.9\% SDS-PAGE of purified proteins: (A) Expression of PTE and
    F104A, (B)purified PTE and F104A, (C) purified \emph{p}FF-PTE and
\emph{p}FF-F104A.]{0.9\% SDS-PAGE of purified proteins: (A) Expression of PTE
    and F104A, (B)purified PTE and F104A, (C) purified \emph{p}FF-PTE and
    \emph{p}FF-F104A.} 
    \label{fig:sds-gel}
\end{figure}
% --------------------------

\subsection{Secondary Structures And Stability}

Circular dichroism (CD) was performed to determine whether the mutation had an
impact on the overall secondary structure and stability. Far-UV wavelength
scans of \emph{p}FF-F104A and \emph{p}FF-PTE showed a double minimum at
\SIlist{208;222}{\nm} (\SI{25}{\celsius}), as expected for a
($\beta$/$\alpha$)\textsubscript{8}-barrel protein, thus suggesting that the
mutation did not affect the overall structure (Figure \ref{fig:CD-fig}).  

% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=1.0\textwidth]{fig1_05}
    \caption[MALDI-TOF mass spectra of tryptic peptide fragments.]{MALDI-TOF
    mass spectra of tryptic peptide fragments.} \label{fig:MALDI-fig} 
\end{figure}
% --------------------------

Surprisingly, comparison of the non-fluorinated counterparts revealed that
F104A was less structured than PTE (Figure \ref{fig:CD-fig}). 

% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.7\textwidth]{fig1_06}
    \caption[CD wavelength scans of PTE (circles), 104A (triangles),
    \emph{p}FF-PTE (diamonds) and \emph{p}FF-104A (squares).]{CD wavelength
        scans of PTE (circles), 104A (triangles), \emph{p}FF-PTE (diamonds) and
        \emph{p}FF-104A (squares).} \label{fig:CD-fig} 
\end{figure}
% --------------------------

To assess the stability, differential scanning calorimetry (DSC) was performed
(Figure \ref{fig:DSC-fig}). Upon heating the sample from \SIrange{20}{70}{\celsius},
\emph{p}FF-PTE exhibited two transitions (T\textsubscript{m}1: 42.0 $\pm$
\SI{0.1}{\celsius}; T\textsubscript{m}2 : 48.6 $\pm$ \SI{0.2}{\celsius}); this
was consistent with our previous studies\cite{Baker2011b}. This biphasic
unfolding was also observed by Grimsley \latin{et al.}  in a study of
organophosphorus hydrolase, and was attributed to the presence of a dimeric
unfolded intermediate\cite{Grimsley1997b}. In contrast, \emph{p}FF-F104A
exhibited a single transition at 49.7 $\pm$ \SI{0.2}{\celsius}, which was
higher than both \emph{p}FF-PTE values (by 7.7 and \SI{1.1}{\celsius}).
Remarkably, after heating, \emph{p}FF-F104A retained the single
T\textsubscript{m} of 49.2 $\pm$ \SI{0.1}{\celsius}, thus demonstrating
regaining of structure after undergoing thermal unfolding.  In the absence of
\emph{p}FF, F104A demonstrated two transitions similar to \emph{p}FF-PTE, thus
suggesting that fluorination was critical for stability.  These data
demonstrate the overall thermodynamic stability of \emph{p}FF-F104A.

% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.9\textwidth]{fig1_07}
    \caption[Differential scanning calorimetry thermograms of (A) F104A and (B)
    \emph{p}FF-F104A.]{Differential scanning calorimetry thermograms of A)
    F104A and B) pFF-F104A.} \label{fig:DSC-fig} 
\end{figure}
% --------------------------

The data further suggest that the unfolding model has been altered
from a 3-state to 2-state transition, and that the energy requirement to
attaining the unfolded intermediate was increased, thereby resulting in a more
cooperative transition. Mutations resulting in this particular transformation
have been observed for other proteins; for example, Fan \latin{et al.} showed
that removal of a C-terminal domain of the oligomeric \latin{E. coli} trigger
factor protein resulted in the transformation of an otherwise n-state unfolding
process to a distinct two-state unfolding process, indicative of pronounced
stabilization of the native structure by interdomain interactions[18]. We
propose that the \emph{p}FF-F104A mutation might also be stabilizing the native
structure of the overall protein (in effect the reverse of the mutation
observed by Fan \latin{et al.})\cite{Fan2008}. That is, \emph{p}FF-F104A unfolds
cooperatively in a single step, concurrent with its dissociation into monomeric
species. Although it was expected that the \emph{p}FF-F104A mutation would have an
effect on interdomain stability (such that neighboring residues would be
allowed to repack and energy would be minimized between monomers at the dimer
interface), the apparent stabilization of the native structure concluded from
the 3-state to 2-state transition transformation through these new interdomain
interactions were indeed unanticipated. Prior examples of this transformation
exist in cases involving subdomains of similar stabilities or where strong
coupling exists between subdomains\cite{Tsytlonok2013}.

\subsection{Enzymatic Kinetics of Phosphotriesterase}

% --------------------------
\begin{table}[h!]
\centering
    \begin{tabular}{llllll}
    \hline
%%
    protein                 &  & \SI{25}{\celsius} & \SI{35}{\celsius} &
    \SI{45}{\celsius} & \SI{55}{\celsius} \\ 
    \hline
%%
    \multirow{2}{*}{PTE}    & k\textsubscript{cat}/K\textsubscript{M} & 2.00$
    \pm$ 0.13 & 0.76 $\pm$ 0.11 & 0.72 $\pm$ 0.12 & 0.46 $\pm$ 0.18 \\
    
    & k\textsubscript{cat} & 2.1 $\pm$ 0.4 & 1.3 $\pm$ 0.1 & 1.4 $\pm$ 0.1 & 0.9
    $\pm$ 0.1 \\
%%
    \multirow{2}{*}{F104A}  & k\textsubscript{cat}/K\textsubscript{M} & 3.27
    $\pm$ 0.11 & 2.42 $\pm$ 0.10 & 1.84 $\pm$ 0.21 & 0.80 $\pm$ 0.09 \\ 
    
    & k\textsubscript{cat} & 6.0 $\pm$ 1.1 & 5.6 $\pm$ 0.1 & 4.0 $\pm$ 1.1 &
    2.0 $\pm$ 0.9 \\
%%
    \multirow{2}{*}{\emph{p}FF-PTE} & k\textsubscript{cat}/K\textsubscript{M} &
    0.23 $\pm$ 0.04 & 0.21 $\pm$ 0.03 & n.a & n.a \\ 
    
    & k\textsubscript{cat} & 0.01 $\pm$ 0.0 & 0.1 $\pm$ 0.0 & n.a & n.a \\
%%
    \multirow{2}{*}{\emph{p}FF-F104A} & k\textsubscript{cat}/K\textsubscript{M}
    & 2.23 $\pm$ 0.15 & 1.94 $\pm$ 0.18 & 1.49 $\pm$ 0.20 & 1.11 $\pm$ 0.09 \\
    & k\textsubscript{cat} & 3.3 $\pm$ 0.3 & 3.3 $\pm$ 0.6 & 2.6 $\pm$ 1.0 &
    2.0 $\pm$ 0.7 \\ 
    
    \hline
    \multicolumn{6}{l}{n.a = not available; 
        k\textsubscript{cat}/K\textsubscript{M}:
        $\times$10\textsuperscript{5}\SI{}{\per\Molar\per\second};
        k\textsubscript{cat}: \SI{}{\per\second}.}            
    \end{tabular}
    \caption[Paraoxon hydrolysis efficiency summary of PTE, F104A,
    \emph{p}FF-PTE, and \emph{p}FF-F104A. Residual activities were preformed
after incubation at \SIlist{35;45;55}{\celsius}.]{Paraoxon hydrolysis
    efficiency summary of PTE, F104A, \emph{p}FF-PTE, and \emph{p}FF-F104A.
    Residual activities were preformed after incubation at
    \SIlist{35;45;55}{\celsius}.} 
    \label{tab:kinetics-result}
\end{table}
% --------------------------

To assess function, we determined the Michaelis–Menten kinetics of
\emph{p}FF-F104A, \emph{p}FF-PTE, F104A, and PTE with paraoxon. At
\SI{25}{\celsius}, \emph{p}FF-PTE exhibited the highest activity
(k\textsubscript{cat}/K\textsubscript{M} = \SI{327000}{\per\Molar\per\second};
Table \ref{tab:kinetics-result}); \emph{p}FF-F104A was slightly lower
(k\textsubscript{cat}/K\textsubscript{M} = \SI{223000}{\per\Molar\per\second}).
Non-fluorinated PTE exhibited k\textsubscript{cat}/K\textsubscript{M} of
\SI{200000}{\per\Molar\per\second}, similar to those both fluorinated proteins;
however, F104A was dramatically less active
(k\textsubscript{cat}/K\textsubscript{M} = \SI{23000}{\per\Molar\per\second},
Table \ref{tab:kinetics-result}).  Thus, the fluorinated amino acids appear to
be necessary for \emph{p}FF-F104A activity.  Proteins were then incubated at
\SIlist{35;45;55}{\celsius} for one hour, and then cooled to room temperature
to determine residual activity. A decline in residual activity was observed for
all proteins as a function of elevated temperature. \emph{p}FF-F104A, which was
designed to stabilize the fluorinated protein, exhibited 50\% retention of
activity at \SI{55}{\celsius} (Figure \ref{fig:kinetics-fig}, Table
\ref{tab:kinetics-result}). In contrast, at \SI{55}{\celsius}, \emph{p}FF-PTE
and PTE exhibited 24\% and 23\% initial activity, respectively; F104A exhibited
a significant loss in activity at or above \SI{45}{\celsius} (Figure
\ref{fig:kinetics-fig}, Table \ref{tab:kinetics-result}). As \emph{p}FF-F104A
retained substantial activity after elevated temperatures, we then investigated
the half-life of the activity. The parent \emph{p}FF-PTE exhibited more than
50\% loss of activity after three days, whereas the non-fluorinated PTE showed
more than 50\% activity reduction after seven days (Figure
\ref{fig:kinetics-fig}, Table \ref{tab:kinetics-day-result}).  Remarkably,
\emph{p}FF-F104A retained 66\% activity after seven days. The non-fluorinated
counterpart F104A failed to exhibit activity after one day.  Together these
data confirm that \emph{p}FF-F104A is able to delay heat inactivation while
maintaining function after one week.

% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=1.0\textwidth]{fig1_08}
    \caption[(A) Residual activity profile of all proteins, (B) Activity
    measured as a function of days.]{(A) Residual activity profile of all
    proteins, (B) Activity measured as a function of days.} 
    \label{fig:kinetics-fig} 
\end{figure}
% --------------------------

% --------------------------
\begin{table}[h!]
\centering
    \begin{tabular}{llllll}
    \hline
%%
    protein                 &  & Day 1 & Day 2 & Day 3 & Day 7\\ 
    \hline
%%
    \multirow{2}{*}{PTE}    & k\textsubscript{cat}/K\textsubscript{M} & 2.06$
    \pm$ 0.23 & 1.89 $\pm$ 0.23 & 1.53 $\pm$ 0.18 & 0.89 $\pm$ 0.10 \\
    
    & k\textsubscript{cat} & 2.5 $\pm$ 0.5 & 0.97 $\pm$ 0.03 & 0.23 $\pm$ 0.01 & 0.10
    $\pm$ 0.01 \\
%%
    \multirow{2}{*}{F104A}  & k\textsubscript{cat}/K\textsubscript{M} & 3.53
    $\pm$ 0.12 & 2.01 $\pm$ 0.28 & 1.62 $\pm$ 0.21 & 1.13 $\pm$ 0.09 \\ 
    
    & k\textsubscript{cat} & 6.23 $\pm$ 0.97 & 3.23 $\pm$ 0.88 & 2.13 $\pm$ 0.94 &
    1.46 $\pm$ 0.97 \\
%%
    \multirow{2}{*}{\emph{p}FF-PTE} & k\textsubscript{cat}/K\textsubscript{M} &
    0.18 $\pm$ 0.02 & n.a & n.a & n.a \\ 
    
    & k\textsubscript{cat} & 0.11 $\pm$ 0.02 & n.a & n.a & n.a \\
%%
    \multirow{2}{*}{\emph{p}FF-F104A} & k\textsubscript{cat}/K\textsubscript{M}
    & 1.83 $\pm$ 0.21 & 1.72 $\pm$ 0.20 & 1.43 $\pm$ 0.13 & 1.20 $\pm$ 0.10 \\

    & k\textsubscript{cat} & 2.25 $\pm$ 0.32 & 2.03 $\pm$ 0.31 & 1.61 $\pm$ 0.25 &
    1.43 $\pm$ 0.25 \\ 

    \hline
    \multicolumn{6}{l}{n.a = not available; 
        k\textsubscript{cat}/K\textsubscript{M}:
        $\times$10\textsuperscript{5}\SI{}{\per\Molar\per\second};
        k\textsubscript{cat}: \SI{}{\per\second}.}            
    \end{tabular}
    \caption[Kinetics of paraoxon hydrolysis as a function of time.]{Kinetics
    of paraoxon hydrolysis as a function of time.} \label{tab:kinetics-day-result} 
\end{table}
% --------------------------

Although methods enabling the biosynthesis of artificial proteins bearing UAAs
are abundant,\cite{Voloshchuk2009} tools to help further improve the overall
activity and stability are needed. Mutagenesis and evolutionary approaches have
been employed successfully to identify variants with enhanced function;
however, these rely heavily on testing or screening several to millions of
constructs\cite{Voloshchuk2007b,Montclare2006b,Yoo2007}. We demonstrate the use
of computational methods to identify a fluorinated protein variant that
exhibits superior heat stability and half-life. Notably, the \emph{p}FF-F104A
variant is only functional in the fluorinated form, thus validating
Rosetta-based design with \emph{p}FF. This provides another useful tool for
protein design and could be employed in conjunction with the above mentioned
approaches.

\printbibliography[heading=subbibliography]

\end{refsection}
