\chapter{Improved Stability and Half-Life of Fluorinated Phosphotriesterase
Using Rosetta} 
\label{chap:uaa}

\begin{refsection}

\section{Introduction}

\subsection{Protein Engineering}
\label{sec:protein-engineering}

Proteins (in Greek, it means “of primary importance") are biomolecules composed
of 22 amino acids \cite{Nelson2005}. The amino acids are linked together by
amide bonds to form a primary sequence which can assemble into different
secondary, tertiary, quaternary structures \cite{Berg2002a}. Each amino acid
consists of amino carboxylic group and a side chain that contributes to the
unique property of a protein. The primary structure of a protein refers to the
linear form of amino acids in the polypeptide chain \cite{Sanger1945}.
Secondary structures are defined by the pattern of hydrogen bonds of the
protein, including $\alpha$-helix, $\beta$-sheet and turns \cite{PAULING1951}
(Figure \ref{fig:alpha-helix-stabilization}, Figure \ref{fig:beta-turn}).  The
$\alpha$-helix is composed of a spiral conformation with backbone amine group
donating a hydrogen bond to the backbone carbonyl group every i, i+3 residues
\cite{Kabsch1983}, and every complete turn of the helix is only 3.6 amino acid
residues. This leads to an overall dipole moment from the individual
micro-dipoles of the carbonyl groups \cite{Hol1978} (Figure
\ref{fig:alpha-helix-stabilization}).
% -------------------------- fig-dipole-of-alpha-helix
\begin{figure}[htbp] 
    \centering \includegraphics[width=1.0\textwidth]{fig1_33}
    \caption[Schematic illustration of (A) an $\alpha$-helix; (B) parallel and
    anti-parallel $\beta$ sheet.]{Schematic illustration of (A) an
        $\alpha$-helix; (B) parallel and anti-parallel $\beta$ sheet
        \cite{Bevivino2001,Berg2002a}.}
    \label{fig:alpha-helix-stabilization}
\end{figure}
% -------------------------- fig*

The second regular form of secondary structure is $\beta$-sheet, and two types
are parallel and anti-parallel $\beta$-sheets \cite{Astbury1932}. The parallel
$\beta$-sheet consists of two peptide strands pointing in the same
direction, and held together by hydrogen bonding between the strands (Figure
\ref{fig:alpha-helix-stabilization}). The other is characterized by two strands
pointing in the opposite directions. These patterns of hydrogen bonds stabilize
the protein structures. The $\beta$-turn is also one of motifs of protein, and
identified through the i, i+1, i+2, and i+3 residues \cite{Nemethy1972}. Figure
\ref{fig:beta-turn} demonstrates differences of the bond linking at i+2 and i+3
positions among type I and type II turns. In type I turn, backbone dihedral
angles of residue ($\phi$,$\psi$) are (-60,-30) and (-90,0) of residues i+1 and
i+2, respectively, while type II exhibits (-60, 120) and (80, 0), respectively
\cite{Nemethy1972}. 
% -------------------------- beta-turn
\begin{figure}[htbp] 
    \centering \includegraphics[width=0.55\textwidth]{fig1_29}
    \caption[Schematic illustration of the type I and type II $\beta$-turn
    structure of a protein.]{Schematic illustration of the type I and type II
        $\beta$-turn structure of a protein \cite{Nemethy1972}.} 
    \label{fig:beta-turn}
\end{figure}
% -------------------------- fig*

With these secondary structures, the tertiary structures are built and
determined through several interactions, such as salt bridges
\cite{Privalov2009}, hydrophobic interactions \cite{Beadle2002a} and hydrogen
bonds \cite{Sheridan1982}.  Last, the quaternary structure consists of subunits
and these units coordinate together to function as a protein complex
\cite{Privalov2009}.

Proteins are essential to cell function and metabolism
\cite{Lambert2012,Horton2007,Dessein2008} (Table \ref{tab:protein-app}). For
example, the proprotein convertase subtilisin/kexin type 9 (PCSK9), a serine
endoproteases, regulates low density lipoprotein (LDL) inside liver
cells \cite{Lambert2012,Horton2007}. This PCSK9 protein blocks the structural
transition of LDL receptor and further affects the metabolism in the endosome
\cite{Lambert2012}. Another example of how protein is essential is as the
antibodies. Autoimmune diseases, like Celiac \cite{Dessein2008} or Crohn's
disease \cite{Sollid2005, Meize-Grochowski}, are found associated with
antibodies - a large, Y-shape protein produced by plasma cells. These
antibodies attack cells in human bodies, which result in irregular cell deaths.
To solve the issues above, proteins are targeted and engineered. For example,
an monoclonal antibody that recognizes epitopes on PCSK9 has been developed to
maintain the LDL level \cite{Lambert2012}. These examples demonstrated well
essential roles in cells, and potentials for applications in biotechnology.   
% ------------------------- table-protein-examples
\begin{table}[htbp]
    \centering
    \begin{tabular}{ lll }
        \hline
        Protein & Usage & References \\
        \hline
        
        PCSK9 & serine endoprotease & \cite{Lambert2012,Horton2007} \\
        IgG & antibody & \cite{Sollid2005, Meize-Grochowski} \\
        PCAF & histone acetyltransferase & \cite{Mehta2011a} \\
        Erythropoietin & hormone & \cite{Haroon2003,Siren2001} \\
        Collagen & n.a & \cite{Bella1995} \\
        Thaumatin & sweetener & \cite{Green1999} \\
        PSA & biomarker & \cite{Crawford2014} \\
        Protease & detergent & \cite{Kirk2002} \\
        Polymerase & polymerase chain reaction & \cite{Berg2002} \\
        Lactase & food processing & \cite{Wiseman1993} \\

        \hline
    \end{tabular}
    \caption[Examples of proteins and their funtions and
    applications.]{Examples of proteins and their functions and applications.}
    \label{tab:protein-app}
\end{table}
% ------------------------- table*
 
One well-known protein role serves as an enzyme. Enzymes mediate chemical
reactions and serves different functions biological
systems \cite{AthelCornish-Bowden2012,Stryer1995,Radzicka1995a}. In the
\emph{ENZYME} database \cite{Schomburg2004}, there are roughly 5000 different
enzymes information stored in the repository.  Enzyme-catalyzed reactions have
been initially studied in the 19th century \cite{AthelCornish-Bowden2012}.
Compared to chemical catalysts, enzymes are highly specific and enhance the
rate of reaction by approximately \SI{e8}-fold over non-catalyzed reactions
\cite{Stryer1995}. These are due to the facts that enzymes are specific and the
activation energy is needed (Figure \ref{fig:enzyme-intro}). First, binding
of enzyme and its substrate forms a low energy complex (ES).  While the complex
lowers the transition state, less energy is needed to achieve the reaction
compared to the uncatalyzed reaction (ES$\ddag$).  Finally the enzyme-product, EP,
dissociates and hence releases the products. For example, Radzicka \latin{et al.}
identified \emph{O}-rotidine 5\rq-phosphate decarboxylase (OMP) as an extremely
proficient enzyme, and further estimated the dissociation constant of less than
\SI{5e-24}{\Molar} \cite{Radzicka1995a}. In particular, this decarboxylase
enhances the reaction rate by \SI{e7}-fold over the uncatalyzed
reaction \cite{Radzicka1995a}. 
% -------------------------- fig-enzyme-catalysis
\begin{figure}[htbp] 
    \centering \includegraphics[width=0.8\textwidth]{fig1_14}
    \caption[The energies of the stages of a chemical reaction. Uncatalysed
    (dashed line), substrates need a lot of activation energy to reach a
transition state, which then decays into lower-energy products. When enzyme
catalysed (solid line), the enzyme binds the substrates (ES), then stabilizes
the transition state (ES$\ddag$) to reduce the activation energy required to produce
products (EP) which are finally released.]{The energies of the stages of a
    chemical reaction. Un-catalysed (dashed line), substrates need a lot of
    activation energy to reach a transition state, which then decays into
    lower-energy products. When enzyme catalysed (solid line), the enzyme binds
    the substrates (ES), then stabilizes the transition state (ES$\ddag$) to reduce
    the activation energy required to produce products (EP) which are finally
    released. Modified from Berg \latin{et al.} \cite{Berg2002}.} 
    \label{fig:enzyme-intro}
\end{figure}
% -------------------------- fig*

Protein engineering involves the design of proteins with new or desirable
functions. Ulmer \latin{et al.} defined protein engineering back
in the 80s \cite{Ulmer1983}, including the different prospects of
protein engineering, such as X-ray crystallography \cite{Takeda2006}, chemical
DNA synthesis \cite{Pannekoek1979}, computational modeling, and protein folding
\cite{Ulmer1983}. The combination of crystal structure and chemistry
information was emphasized as a powerful approach to design proteins with
desirable properties \cite{Ulmer1983}. Wiseman \latin{et al.} later adapted the
concept with specific applications for protein engineering, including improved
proteases for the detergent industry \cite{Wiseman1993,Harwood1992} and
imobilized lactase for food precessing \cite{Wiseman1993} (Table
\ref{tab:protein-app}). 

Catalytic promiscuity of engineered proteins is an important concept. This
involves the property that is defined as the ability to catalyze more than one
chemical reaction in the active site of a protein \cite{Kazlauskas2005a}. While
Morley \latin{et al.} focused a selection of single mutations that enhanced
catalytic activity, substrate specificity was altered \cite{Morley2005a}.
Comparison of reaction to different substrates was carried out and the
$\Delta$$\Delta$G was calculated. For example, while the L-Ala–D/L-Glu
epimerase (AEE) originally catalyzed the o-succinylbenzoate synthase (OSBS)
$\beta$-elimination reaction slowly, mutations N297G led to enhanced catalytic
activity ($\Delta$$\Delta$G = -4.6 kcal/mol) (Figure
\ref{fig:protein-engineering-example}). These proteins samples were analyzed
with a collections of reactions, and further demonstrated the alternate
catalytic activity (catalytic promiscuity) \cite{Kazlauskas2005a,Schmidt2003}
(Figure \ref{fig:protein-engineering-example}).
% -------------------------- fig-protein-engineering
\begin{figure}[htbp] \centering \includegraphics[width=0.70\textwidth]{fig1_34}
    \caption[Reaction coordinates for the reactions catalyzed by the OSBS (left
    panel) and the D297G mutant of AEE (right panel). The activation free
energy associated with the uncatalyzed OSBS reaction is shown in red; the
activation free energies associated with the catalyzed OSBS reactions are shown
in blue, and the free energies associated with the proficiencies are shown in
green.]{Reaction coordinates for the reactions catalyzed by the OSBS (left
    panel) and the D297G mutant of AEE (right panel). The activation free
    energy associated with the uncatalyzed OSBS reaction is shown in red; the
    activation free energies associated with the catalyzed OSBS reactions are
    shown in blue, and the free energies associated with the proficiencies are
    shown in green \cite{Schmidt2003}.}
    \label{fig:protein-engineering-example}
\end{figure}
% -------------------------- fig*

\subsubsection{Rational Design}
\label{sec:rational-design}

Rational design provides an effective approach to design proteins of interest
when the structure and mechanism are well-known. There were studies focusing on
the relationship between protein structures and their activities
\cite{Beadle2002a,Quillin1993}. Quillin \latin{et al.} has adapted the concept
and modified histidine of sperm whale myoglobin \cite{Quillin1993}. The
superposition of deoxy and carbonmonoxy structures reveals the conformation of
the Fe-C-O complex, further providing the evidences into the mechanism of
ligand binding \cite{Quillin1993} The inhibition of reaction is resulted from
the reduction in the equilibrium constant, which is due to the water
displacement and ligand entry. 

The most classical method used for rational design
is site-directed mutagenesis (SDM) \cite{Arnold1993}. With two common methods
for SDM, overlap extension method \cite{Ho1989} and whole plasmid single round
polymerase chain reaction \cite{Antikainen2005a}, an intentional change is
introduced to a DNA sequence. The first method involves two set of primer
pairs, where one primer of each set includes the mutant codon within the DNA
sequence (Figure \ref{fig:sdm-1}) \cite{Ho1989}. Upon the first PCR, two
double-stranded DNA products are obtained with these four primers. With further
denaturation and annealing,two hetero-duplexes -each with desired muatagenic
codon - are formed.  DNA polymerase is then used for a second PCR to amplify
the desired DNA \cite{Ho1989}. 
% -------------------------- fig-sdm-1
\begin{figure}[htbp] \centering \includegraphics[width=0.6\textwidth]{fig1_16}
    \caption[Schematic diagram of site-directed mutagenesis by overlap
        extension. The double-stranded DNA and synthetic oligos are represented
        by lines with arrows indicating the 5$\rq$-to-3$\rq$ orientation. The site of
        mutagenesis is indicated by the small black rectangle. Oligos are
        denoted by lower-case letters and PCR products are denoted by pairs of
        upper-case letters corresponding to the oligo primers used to generate
        that product. The boxed portion of the figure represents the proposed
        intermediate steps taking place during the course of reaction (3),
        where the denatured fragments anneal at the overlap and are extended 3’
        by DNA polymerase (dotted line) to form the mutant fusion product. By
        adding additional primers ‘a’ and ‘d’ the mutant fusion product is
        further amplified by PCR.]
        {Schematic diagram of site-directed mutagenesis by overlap extension.
            The double-stranded DNA and synthetic oligos are represented by lines
            with arrows indicating the 5$\rq$-to-3$\rq$ orientation. The site of
            mutagenesis is indicated by the small black rectangle. Oligos are
            denoted by lower-case letters and PCR products are denoted by pairs
            of upper-case letters corresponding to the oligo primers used to
            generate that product. The boxed portion of the figure represents
            the proposed intermediate steps taking place during the course of
            reaction (3), where the denatured fragments anneal at the overlap
            and are extended 3$\rq$ by DNA polymerase (dotted line) to form the
            mutant fusion product. By adding additional primers ‘a’ and ‘d’ the
            mutant fusion product is further amplified by PCR \cite{Ho1989}.}
            \label{fig:sdm-1} 
\end{figure}
% -------------------------- fig*


The second method involves designing a single set of primers bearing the
designated mutation flanked by complementary nucleotides to be gene of interest
\cite{Arnold2003}. While PCR takes place, both strands of the template are
replicated. After PCR products are collected, DpnI methylase is used to
digest the template.  Upon transformation into competent cells, a circular,
mutated plasmid DNA is then obtained \cite{Antikainen2005a} (Figure
\ref{fig:sdm-2}).
% -------------------------- fig-sdm-2
\begin{figure}[htbp] \centering \includegraphics[width=0.6\textwidth]{fig1_17}
    \caption[Whole plasmid, single-round PCR produces the desired mutation
    directly from a plasmid template: an arrows represents a primer, X
    represents a mutagenic codon, a thick line represents the template plasmid
    DNA, and a thin line represents newly synthesized DNA.]{Whole plasmid,
        single-round PCR produces the desired mutation directly from a plasmid
        template: an arrows represents a primer, X represents a mutagenic
        codon, a thick line represents the template plasmid DNA, and a thin
        line represents newly synthesized DNA \cite{Antikainen2005a}.}
        \label{fig:sdm-2}
\end{figure}
% -------------------------- fig*

In addition to conventional methods of rational design, computational
protein design tools have emerged. Computational protein design principles
are based on the combination of force fields and search algorithms to
identify the promising protein mutations with a given protein structure
\cite{VanDerSloot2009a}. At selected positions, the algorithm mutate the
original amino acid to other possible amino acids and render new conformations.
Within the library of rotamers, the interactions from different simulated
results would be evaluated. The energy of the structure is then calculated
after optimization of the side-chain and backbone conformations with the
neighboring amino acids \cite{VanDerSloot2009a}. (Figure \ref{fig:cpd})
% -------------------------- fig-CPD
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig1_18}
    \caption[The basic principle of computational protein design algorithms;
        (A) at the positions considered for design the algorithm ‘mutates’ to
        all the other naturally occurring amino acids by replacing the original
        amino acid with discrete conformations of the substituting amino acid
        (rotamers); (B) favorable — low energy — substitutions are retained;
        (C) unfavorable - high energy — substitutions are discarded.]{The basic
            principle of computational protein design algorithms; (A) at the
            positions considered for design the algorithm ‘mutates’ to all the
            other naturally occurring amino acids by replacing the original
            amino acid with discrete conformations of the substituting amino
            acid (rotamers); (B) favorable — low energy — substitutions are
            retained; (C) unfavorable - high energy — substitutions are
            discarded \cite{VanDerSloot2009a}.} 
    \label{fig:cpd} 
\end{figure}
% -------------------------- fig*

Bolon \latin{et al.} calculated the side-chain accessible surface areas
\cite{Connolly1983}, and further identified the thioredoxin rotamers
that were likely to be energetically favorable \cite{Bolon2003}. Hence, such
tools assisted the development of rational design. Otten \latin{et al.} and
Moss \latin{et al.} demonstrated that the algorithms altered enzymes activity
\cite{Otten2010a} and growth hormone stability \cite{Moss2009,Lippow2007a}. To
study the growth hormone, Moss \latin{et al.} used  the feasibility of
exploring receptor activation by clustering the synthetic ligands for growth
factor receptors \cite{Moss2009}. The results demonstrated more than two orders
of magnitude increased affinity compared to the wild-type ligand. Different
computational tools were employed in the redesign of affinity of a
wide variety of protein to protein interactions \cite{Selzer2000,Reina2002}.
Selzer \latin{et al.} engineered the protein TEM1 $\beta$-lactamase by using
Protein Association Rate Enhancement (\emph{PARE}), and the affinity between
TEM1 and protein inhibitor BLIP was enhanced 250-fold \cite{Selzer2000}
(Figure \ref{fig:tem1}). These examples demonstrated the feasibility of using
computational tools to engineer and design proteins of interest. 
% -------------------------- fig-tem1
\begin{figure}[htbp] \centering \includegraphics[width=0.5\textwidth]{fig1_19}
    \caption[Surface representations of TEM1 and BLIP. (a) The residues on BLIP
        probed for faster association.(b, c, d) The electrostatic potentials on
        the binding surfaces of BLIP, TEM1 and the fastest BLIP mutant (+6).
        The calculations were performed using formal charges at 0.1 M salt,
        with the contours drawn at 0.7 kT / e (blue for positive and red for
        negative charge). The structures of TEM1 and BLIP were from the crystal
        structure of the complex. The green patch denotes the binding
    interface.]{Surface representations of TEM1 and BLIP. (a) The residues on
        BLIP probed for faster association.(b, c, d) The electrostatic potentials
        on the binding surfaces of BLIP, TEM1 and the fastest BLIP mutant (+6).
        The calculations were performed using formal charges at 0.1 M salt,
        with the contours drawn at 0.7 kT / e (blue for positive and red for
        negative charge). The structures of TEM1 and BLIP were from the crystal
        structure of the complex. The green patch denotes the binding
        interface \cite{Selzer2000}.} \label{fig:tem1} 
\end{figure}
% -------------------------- fig*

\subsubsection{Rosetta Design}

While computational tools are widely used for protein
engineering \cite{Rothlisberger2008,DiMaio2011a,Korkegian2005,
Leaver-Fay2013a,Leaver-Fay2011,Drew2013a,Kaufmann2010,Rohl2004}, Rosetta suite
developed at the University of Washington is a versatile tool for engineering
proteins \cite{Leaver-Fay2011}. This suite provides a handful of protocols for
analyzing and mutating protein structures.  The simulation relies heavily on
knowledge-based potentials. In Rosetta, the scoring function sums the weighted
terms that represent the physical or probabilistic pairwise interactions
\cite{Rohl2004} (Figure \ref{fig:rosetta-intro}), including Lennard-Jones
potential \cite{Clementi1999}, implicit solvent model \cite{Lazaridis1999},
orientation-dependent hydrogen bond term \cite{Kortemme2003}, side-chain and
backbone torsion potentials derived from the PDB \cite{Rohl2002}, short-ranged
knowledge-based electrostatic term \cite{Leaver-Fay2013a}, and reference
energies for each of the 20 amino acids that model the unfolded state
\cite{Leaver-Fay2013a}.
% -------------------------- fig-rosetta
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig1_01}
    \caption[The scheme of scoring function of Rosetta. 1: Lennard-Jones
    Potential; 2: implicit solvent model; 3: hydrogen bonding; 4:
electrostatics; 5: PDB drived torsion potential.]{The scheme of scoring
    function of Rosetta. 1: Lennard-Jones Potential; 2: implicit solvent model;
3: hydrogen bonding; 4: electrostatics; 5: PDB derived torsion potential.}
\label{fig:rosetta-intro}
\end{figure}
% -------------------------- fig*

Rosetta suite has been employed for multiple applications, including:
prediction of protein structure \cite{Rohl2004} (Figure
\ref{fig:rosetta-overview} A); loop modeling \cite{Das2007a} (Figure
\ref{fig:rosetta-overview} B); macromolecular ligand docking under fully
\cite{Das2008} (Figure \ref{fig:rosetta-overview} C) or partial  modifications
\cite{Willcox2003} (Figure \ref{fig:rosetta-overview} D); and enzyme design
\cite{Ashworth2006a}.
% -------------------------- fig-protein-engineering
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig1_20}
    \caption[A unified framework for tackling multiple molecular modeling
        problems. Each panel depicts a problem in biomolecule structure
        prediction or design (left) and the representation used by Rosetta
        (right); (A) \emph{de novo} structure prediction for the phage 434
        repressor protein (PDB code: 1R69); (B) loop modeling carried out
        during comparative modeling of the CASP7 target T0331, pyridoxamine
        5′-phosphate oxidase-related protein (2HHZ); (C) Protein-protein
        docking for a host and viral major histocompatibility complex receptor
        (1P7Q); (D) Protein-protein docking with backbone flexibility limited
        to a hinge region (red) and to a loop (blue).] {A unified framework for
            tackling multiple molecular modeling problems. Each panel depicts a
            problem in biomolecule structure prediction or design (left) and
            representation used by Rosetta (right); (A) \emph{de novo}
            structure prediction for the phage 434 repressor protein (PDB code:
            1R69) \cite{Mondragon1989}; (B) loop modeling carried out during
            comparative modeling of the CASP7 target T0331, pyridoxamine
            5′-phosphate oxidase-related protein (2HHZ) \cite{Das2007a}; (C)
            Protein-protein docking for a host and viral major
            histocompatibility complex receptor (1P7Q) \cite{Willcox2003}; (D)
            Protein-protein docking with backbone flexibility limited to a
            hinge region (red) and to a loop (blue) \cite{Das2008}.}
        \label{fig:rosetta-overview} 
\end{figure}
% -------------------------- fig*

\paragraph{Prediction of Protein Structure}
The challenge of \latin{de novo} protein structure prediction is to search for the
lowest free-energy of a specified amino acid sequence within a large size of
conformational space \cite{Bradley2005}. Using Rosetta, Bradley \latin{et al.}
have attempted to simulate high-resolution of \latin{de novo} structural prediction
by generating low-resolution models. To overcome the challenge, a large amounts
of sequence homologs were generated for a characteristic set of energy
models. Among 16 predicted structures, Rosetta successfully generated five
proteins that exhibited high accuracy \cite{Bradley2005}. The 434 repressor
demonstrates the lowest energy model with a C$\alpha$-RMSD of
\SI{1.3}{\angstrom} in comparison with the crystal structure. (Figure
\ref{fig:434}, Figure \ref{fig:rosetta-overview} A)
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.40\textwidth]{fig1_36}
    \caption[High-resolution de novo structure prediction of 434 repressor.
    Super-position of low-energy models (blue) with experimental structures
(red) showing core side chains.]{High-resolution de novo structure prediction
    of 434 repressor. Super-position of low-energy models (blue) with
    experimental structures (red) showing core side chains \cite{Bradley2005}.}
    \label{fig:434}
\end{figure}
% --------------------------

\paragraph{Loop Modeling}
The loop modeling adapted the similar move set to \latin{de novo} structural
prediction except for additional loop closure step \cite{Das2008}. Among more
than 100 domains, the collection of the Seventh Critical Assessment of
Techniques for Protein Structure Prediction (CASP7) provides a library for the
Rosetta comparative modeling and de novo structure prediction methods
\cite{Das2008} (Figure \ref{fig:rosetta-overview} B). Das \latin{et al.}
demonstrated the template-based predictions on T0331, T0380, T0368, and T0367
with comparison of crystal structures \cite{Das2007a}. The loop modeling was
carried out with backbone movements without perturbing segments outside the
rebuilt segments. The comparison of T0331 successfully demonstrates refinement
at the outer helix and long hairpin loops \cite{Das2007a} (Figure
\ref{fig:t0331}).
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.5\textwidth]{fig1_35}
    \caption[The template-based prediction of T0331. The crystal structure is
    shown in blue, the best of submitted models in green, and the best template
in pink.]{The template-based prediction of T0331. The crystal structure is
    shown in blue, the best of submitted models in green, and the best template
    in pink \cite{Das2007a}.} \label{fig:t0331}
\end{figure}
% --------------------------

\paragraph{Macromolecular Ligand Docking}

Upon protein binding, backbone conformational changes frequently occur, and the
fixed-backbone approximation used in Rosetta docking usually preclude
high-resolution predictions \cite{Das2008}. With a vast conformational space to
screen, the fully flexible model may be costly. (Figure
\ref{fig:rosetta-overview} C) \cite{Wang2007a}. Previously, Willcox \latin{et
al.} have revealed the binding between HLA-A2 and LIR-1, a host and viral major
histocompatibility complex receptor \cite{Willcox2003} (Figure
\ref{fig:rosetta-overview} C,D). Using reformulated RosettaDock suite
\cite{Wang2007a}, Baker group demonstrate the efficiency of sampling the
binding between HLA-A2 and LIR-1 given types of backbone movement
\cite{Wang2007a} (Figure \ref{fig:rosetta-overview} D). 
% -------------------------- fig-macromol-with-ligand
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig1_21}
    \caption[Plots of score vs. RMSD for local docking of the unbound
        structures in target 1EWY without (A) and with (B) the small molecule
        FAD bound to FNR (A), with high, medium, and acceptable accuracy decoys
        colored in brown, orange, and tan, respectively; (C) The second-lowest
        energy structure from docking using FAD with FNR (green), Fd (cyan),
        and the FAD molecule (magenta) superimposed on the crystal structure of
    the complex (gray)] {Plots of score vs. RMSD for local docking of the
        unbound structures in target 1EWY without (A) and with (B) the small
        molecule FAD bound to FNR (A), with high, medium, and acceptable
        accuracy decoys colored in brown, orange, and tan, respectively; (C)
        The second-lowest energy structure from docking using FAD with FNR
        (green), Fd (cyan), and the FAD molecule (magenta) superimposed on the
        crystal structure of the complex (gray)\cite{Chaudhury2011}.} 
        \label{fig:macro-ligand} 
\end{figure}
% -------------------------- fig*

As another example of how Rosetta could be a tool for ligand binding study, 
Chaudhury \latin{et al.} demonstrated the use of Rosetta on the
Ferredoxin-NADP\textsuperscript{+} reductase (FNR) with its ligand group flavin
adenine dinucleotide (FAD), in complex with ferredoxin (Fd) (Target
1EWY) \cite{Chaudhury2011}. Figure \ref{fig:macro-ligand} shows the score plot
for docking model with the FAD molecule, and the decoy was illustrated in Figure
\ref{fig:macro-ligand}. The results demonstrated the FAD molecule was critical for
electron transfer from the Fd to the NADP\textsuperscript{+} through the
formation of a ternary complex \cite{Hermoso2002}.  Upon iterations, the
Rosetta algorithm provided insight into ligand binding and led to the success
of finding good ligand candidates.

\paragraph{Enzyme Design}

Using Rosetta, Jiang \latin{et al.} designed retro-aldolases with four
different catalytic motifs to catalyze the breaking of a carbon-carbon bond in
a nonnatural substrate, 4-hydroxy-4-(6-methoxy-2-naphthyl)-2-butanone
\cite{Jiang2008} (Figure \ref{fig:rosetta-enzyme}). By accommodating mechanisms
involving enamine catalysis at the active site, ensembles of models of each of
the key intermediates --- RA60, RA46, RA45 --- in the reaction were generated
(Figure \ref{fig:rosetta-enzyme} A -- C). Among these ensembles, the active
designs occur on scaffolds belonging to the triose phosphate isomerase
(TIM)–barrel and jelly-roll folds \cite{Jiang2008}. After these models were
superimposed, they further created an initial composite active-site
description. For each active-site description, catalytic sites were generated
by Rosetta \cite{Jiang2008}, and ranked based on the composite active site and
the specified catalytic geometry. RA60 successfully catalyzed the
retro-aldolase reaction in the presence of
4-hydroxy-4-(6-methoxy-2-naphthyl)-2-butanone
(k\textsubscript{cat}/K\textsubscript{M} = \SI{0.30}{\per\Molar\per\second}).
The results have demonstrated that novel enzyme activities can be designed from
scratch.
% -------------------------- fig-rosetta-enzyme
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig1_22}
    \caption[Structures of designed retro-aldolases. The nucleophilic
        imine-forming lysine is in orange, the transition state model is in
        yellow, the hydrogen-bonding groups are in light green, and the
        catalytic water is shown explicitly.  The designed hydrophobic binding
        site for the aromatic portion of the transition state model is
        indicated by the gray mesh; (A) RA60 (catalytic motif IV, jelly-roll
        scaffold). A designed hydrophobic pocket encloses the aromatic portion
        of the substrate and packs the aliphatic portion of the imine-forming
        K48. A designed hydrogen-bonding network positions the `bridging water
        molecule and the composite TS. (B) RA46 (catalytic motif IV, TIM-barrel
        scaffold). Y83 and S210 position the bridging water molecule, which
        facilitates the proton shuffling required in active site. (C) RA45
        (catalytic motif IV, TIM-barrel scaffold). The bridging water is
        hydrogen-bonded by S211 and E233; replacing the E233 with Thr decreases
        catalytic activity threefold.] {Structures of designed retro-aldolases.
            The nucleophilic imine-forming lysine is in orange, the transition
            state model is in yellow, the hydrogen-bonding groups are in light
            green, and the catalytic water is shown explicitly.  The designed
            hydrophobic binding site for the aromatic portion of the transition
            state model is indicated by the gray mesh; (A) RA60 (catalytic
            motif IV, jelly-roll scaffold). A designed hydrophobic pocket
            encloses the aromatic portion of the substrate and packs the
            aliphatic portion of the imine-forming K48. A designed
            hydrogen-bonding network positions the bridging water molecule and
            the composite TS. (B) RA46 (catalytic motif IV, TIM-barrel
            scaffold). Y83 and S210 position the bridging water molecule, which
            facilitates the proton shuffling required in active site. (C) RA45
            (catalytic motif IV, TIM-barrel scaffold). The bridging water is
            hydrogen-bonded by S211 and E233; replacing the E233 with Thr
            decreases catalytic activity threefold \cite{Jiang2008}.}
        \label{fig:rosetta-enzyme} 
\end{figure}
% -------------------------- fig*

\subsubsection{Unnatural Amino Acids Incorporation Using Rosetta}

In addition to the above applications, Rosetta also provides an
advantage of easily incorporating unnatural amino acid (UAA) (See
\ref{sec:uaa-intro}) into protein sequences \cite{Renfrew2012b}. The
combination of UAAs with Rosetta algorithm allows control over the mutated
position of the UAA, and its interactions with the surrounding
protein environment \cite{Renfrew2012b}. Mills \latin{et al.} have used
\emph{RosettaMatch} package from Rosetta packages to design a metalloprotein
with incorporation of (2,2’-bipyridin-5yl)alanine (Bpy-Ala)\cite{Mills2013}.
Four enzyme active sites have been constructed through a crystal structure of
a small molecule complex of bipyridine (Bpy), iron, and
3,6-di-tert-butylcatechol. Rosetta is used to identify backbone positions
in protein scaffolds. The resultant designs are analyzed on how the geometric
constraints are satisfied. The complexes of bipyridine exhibits metal-ligand
charge-transfer (MLCT), which gives spectroscopic signatures in the UV or
visible ranges. Five of the designed proteins have yielded soluble protein in
the presence of Bpy-Ala, and mutants CB-02 and CB-12 have exhibited the
signature of complex with iron. (Figure \ref{fig:rosetta-uaa}) With the
simulation, they have designed a protein with the ability to bind metals -
\ce{Zn^{2+}}, \ce{Co^{2+}}, \ce{Ni^{2+}}, and \ce{Fe^{2+}} - with high
affinity. 
% -------------------------- fig-rosetta-uaa
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig1_23}
    \caption[Spectral and structural analysis of round 1 designs; (a)
        absorbance spectra of designs CB-02 (solid line) and CB-12 (dashed
        line); (b) superposition of CB-02 design with the structure solved. The
        design is shown in gray, the structure in yellow, and the dopamine
        ligand in green sticks. The Bpy-Ala containing loop in the CB-02
        structure is colored red. No density corresponding to the Bpy-Ala side
        chain was observed in the structure; the position of incorporation in
    the flipped out loop is colored blue.] {Spectral and structural analysis of
        round 1 designs; (a) absorbance spectra of designs CB-02 (solid line)
        and CB-12 (dashed line); (b) superposition of CB-02 design with the
        structure solved. The design is shown in gray, the structure in yellow,
        and the dopamine ligand in green sticks. The Bpy-Ala containing loop in
        the CB-02 structure is colored red. No density corresponding to the
        Bpy-Ala side chain was observed in the structure; the position of
        incorporation in the flipped out loop is colored blue \cite{Mills2013}.}
        \label{fig:rosetta-uaa} 
\end{figure}
% -------------------------- fig*

\subsubsection{Protein Engineering Bearing Unnatural Amino Acids (UAAs)}
\label{sec:uaa-intro}

While proteins have been engineered with natural amino acids, researchers
have also introduced unnatural amino acids (UAAs) into proteins
\cite{Odar2015,Hassan2008,Kiick2000,Hammill2007,Meinnel1990,Johnson2010}.
Proteins bearing UAAs have been designed, and activity and stability have been
compared. The Budisa group have reported that enzymes change the specificities
upon the incorporation of UAAs \cite{Budisa2006}. Biocatalyst bearing UAAs are
selectively active to different chemical compounds \cite{Jackson2006a}.
Jackson \latin{et al.} have incorporated several Phe analogs -
\emph{p}-aminophenylalanine (\emph{p}AF), naphthylalanine (Nap),
\emph{p}-benzoylphenylalanine (\emph{p}Bpa), and \emph{p}-methyoxyphenylalanine
(\emph{p}MOF) - into the prodrug activator nitroreductase (NTR)
\cite{Jackson2006a} (Figure \ref{fig:selectivity-example}). Current pro-drugs,
CB1954 and LH7, have been activated by NTR using antibody-directed methods
\cite{Grove2003}. To improve the interaction, they have altered the relevant
active site residues with UAAs. The incorporation of UAAs at site in NTR is
accomplished by using MjTyrRS/tRNA\textsuperscript{Tyr}\textsubscript{CUA}. A
30-fold improvement of prodrug activator nitroreductase (NTR) efficiency with
\emph{p}NF is reported over that of the native active site \cite{Jackson2006a}
(Figure \ref{fig:selectivity-example}).
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.6\textwidth]{fig1_24}
    \caption[Catalytic efficiency of modified NTR enzymes: (a)
        k\textsubscript{cat}/K\textsubscript{M} values for natural and
        unnatural NTRs with CB1954 and (b)
        k\textsubscript{cat}/K\textsubscript{M} values for natural and
    unnatural NTRs with LH7.]{Catalytic efficiency of modified NTR enzymes: (a)
        k\textsubscript{cat}/K\textsubscript{M} values for natural and
        unnatural NTRs with CB1954 and (b)
        k\textsubscript{cat}/K\textsubscript{M} values for natural and
        unnatural NTRs with LH7 \cite{Jackson2006a}.} 
        \label{fig:selectivity-example}
\end{figure}
% --------------------------

Another example of UAA incorporation demonstrates how it can stabilize
proteins. Steiner \latin{et al.} have incorporated fluorinated Pro residue,
(4R)-FPro and (4S)-FPro, in enhanced green fluorescent protein (EGFP) via the
Pro-auxotrophic \emph{E. coli} K-12 strain JM83 and assessed from the refolding
rates \cite{Steiner2008}. The proteins have been treated with boiling 8 M urea,
and then refolded at room temperature.The fluorescence recovery demonstrate
that folding stability of EGFP are affected by (4S)-FPro, and the refolding
process is 2 times faster than EGFP (Figure \ref{fig:budisa}).
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig1_32} 
    \caption[Fluorescence recovery of EGFP and (4S)-FPro-EGFP. The refolding
    kinetics of both proteins starts with an initial fast phase that is
followed by a slowrefolding phase. (4S)-FPro-EGFP refolds approximately 2 times
faster than EGFP. The percentage of refolding was calculated on the basis of
the final constant amount of fluorescence, corresponding to 100\% of refolding.
Normalized fluorescence in arbitrary units (au) was plotted against
time.]{Fluorescence recovery of EGFP and (4S)-FPro-EGFP. The refolding kinetics
    of both proteins starts with an initial fast phase that is followed by a
    slowrefolding phase. (4S)-FPro-EGFP refolds approximately 2 times faster
    than EGFP. The percentage of refolding was calculated on the basis of the
    final constant amount of fluorescence, corresponding to 100\% of refolding.
    Normalized fluorescence in arbitrary units (au) was plotted against time
    \cite{Steiner2008}.}
    \label{fig:budisa}
\end{figure}
% --------------------------

\subsection{Incorporation of Unnatural Amino Acids (UAAs)} 
\label{sec:rsi-intro}

Several methods have been developed for the incorporation of UAAs into
proteins, which can be categorized under synthetic and biosynthetic approaches.
Common synthetic approaches include solid phase peptide synthesis (SPPS)
(Figure \ref{fig:spps-intro}) and native chemical ligation (NCL) (Figure
\ref{fig:ncl-intro}) while biosynthetic methods include \latin{in vivo} and
\latin{in vitro} site-specific incorporation \cite{Cellitti2008,Hassan2008}
(Figure \ref{fig:ssi-intro}) and residue-specific incorporation
\cite{Johnson2010} (Figure \ref{fig:rsi}). An approach in between synthetic and
biosynthesis is expressed protein ligation (EPL) \cite{Muir1998} (Figure
\ref{fig:epl-intro}). Below is a description in each approach.

\subsubsection{Solid Phase Peptide Synthesis}

In SPPS  (Figure \ref{fig:spps-intro}), activated amino acids are
immobilized on a solid support and synthesized step-by-step in the reactant
solution \cite{Merrifield1963a}. This method is convenient for the introduction
of functional groups, including natural and UAAs, into
peptides, but is limited in terms of overall yield. For example, if each
coupling step has 99\% yield, a 26-amino acid peptide would be synthesized in
77\% final yield \cite{Chan2000}.  
% --------------------------fig-spps
\begin{figure}[htbp] \centering \includegraphics[width=0.5\textwidth]{fig1_26}
    \caption[Principle mechanism of solid phase peptide synthesis (SPPS). The
    first step involves an assembly with protected amino acid derivatives on a
resin support. The second step cleaves the peptide from the resin support with
the cleavage of side chain protecting groups to give a free peptide.]{Principle
    mechanism of solid phase peptide synthesis (SPPS). The first step involves
    an assembly with protected amino acid derivatives on a resin support. The
    second step cleaves the peptide from the resin support with the cleavage of
    side chain protecting groups to give a free peptide
    \cite{Merrifield1963a,Mahto2011}.} 
    \label{fig:spps-intro} 
\end{figure}
% --------------------------fig*

\subsubsection{Native Chemical Ligation}

Native chemical ligation (NCL) is initiated from transthioesterification
occurred at a side chain bearing a thiol group at the N-terminus
\cite{Dawson1994}. A Cys residue is regularly employed, while alternative
method, homocysteine (Hcy) is developed due to rare presence of Cys in protein.
The intermediate from transthioesterification rearranges via by an
intramolecular S to N acyl shift, which results in the formation of a new
fragment (Figure \ref{fig:ncl-intro}).
% --------------------------fig-ncl
\begin{figure}[htbp] \centering \includegraphics[width=0.5\textwidth]{fig1_10}
    \caption[Princilple mechanism of native chemical ligation. The reaction
    involves the chemoselective reaction between an N-terminal Cys residue of
one peptide fragment and another peptide with C-terminal $\alpha$-thioester
(SR) group.]{Principle mechanism of native chemical ligation \cite{Theato2013}.
The reaction involves the chemoselective reaction between an N-terminal Cys
residue of one peptide fragment and another peptide with C-terminal
$\alpha$-thioester (SR) group.} 
\label{fig:ncl-intro} 
\end{figure}
% --------------------------fig*

\subsubsection{Biosynthetic Approach}

To synthesis longer chain peptides and proteins bearing UAAs, biosynthetic
methods have been developed
\cite{Voloshchuk2007b,Yoo2007,Johnson2010,Link2003,Voloshchuk2010,Montclare2006b}.
The genetic code of deoxyribonucleic acid (DNA) is transcribed and translated
into a sequence of polypeptide through the central dogma \cite{CRICK1970}. The
DNA information is first transcribed into messenger ribonucleic acid (mRNA)
through RNA polymerase and transcription factors \cite{Pukkila2001}. Once mRNA
is ready for translation, the ribosome facilitates the decoding of mRNA.
Aminoacyl-tRNA synthetases (aaRS) are, at the same time, carrying the paired
amino acids to their transfer RNAs (tRNAs) \cite{Pukkila2001}. The
aminoacylated tRNAs are then used for synthesis of polypeptide chain. The
specific codon that consists of three nucleotides corresponds to a single amino
acid. The start codon, AUG on mRNA sequence, is recognized by initial tRNA.
Each pair of codon is recognized by amino acid paired tRNA \cite{Sadava2006}
until the stop condon, which is recognized by release factor
\cite{Pukkila2001}. There exists three contemporary methods to biosynthetically
incorporate non-natural amino acids into proteins: site-specific incorporation
(SSI) (Figure \ref{fig:ssi-intro}), residue-specific incorporation (RSI)
(Figure \ref{fig:rsi}), and multi-site specific incorporation (MSI). 

\paragraph{\latin{in vitro} Site-specific Incorporation}
Bain \latin{et al.} have developed a general
approach for the \latin{in vitro} synthesis of proteins \cite{Bain1991}.  The
approach relies on the suppression of an amber termination codon (UAG) in the
mRNA by an amber suppressor tRNA chemically charged with the amino acid analog
\cite{Theato2013} (Figure \ref{fig:ssi-intro}).  This method has been well
studied and developed in research of protein structures and functions
\cite{Martoglio1995,Eichler1997}.

\paragraph{\latin{in vivo} Site-specific Incorporation}
An \latin{in vivo} site-specific method to UAA incorporation was developed by
Schultz and coworkers \cite{Wang2001,Wang2002}. A stop codon at the position of
interest is encoded in the mRNA. For \latin{in vivo} site-specific UAA
incorporation, an orthogonal aminoacyl-tRNA synthetase charges an orthogonal
tRNA with particular UAA, and the suppressor tRNA would help the incorporation
of UAA by recognition of a stop codon. As cells contain 20 aminoacyl-tRNA
synthetase/tRNA pairs, a new one is required for UAA incorporation. An
orthogonal aminoacyl-tRNA synthetase/suppressor tRNA pair based on a
TyrRS/tRNA\textsuperscript{Tyr} pair in the \emph{Methanococcus jannaschii} has
been engineered for use in \emph{E.  coli} for the incorporation of tyrosine
analogs \cite{Wang2001}.
% --------------------------fig-ssi
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig1_09}
    \caption[Schematic representation of \emph{in vitro} SSI via amber
        suppression. A hybrid dinucleotide pdCpA was synthesized with the UAA,
        and the aminoacylated pdCpA was ligated to a truncated amber suppressor
        tRNA\textsubscript{CUA}\textsuperscript{Phe} using T4 RNA ligase, which
        was used for SSI.]{Schematic representation of \emph{in vitro} SSI via
        amber suppression. A hybrid dinucleotide pdCpA was synthesized with the
        UAA, and the aminoacylated pdCpA was ligated to a truncated amber
        suppressor tRNA\textsubscript{CUA}\textsuperscript{Phe} using T4 RNA
        ligase, which was used for SSI \cite{Theato2013,Wang2001}.} 
    \label{fig:ssi-intro} 
\end{figure}
% -------------------------- fig*

\paragraph{\latin{in vivo} Residue-specific Incorporation}
As an alternative to site-specific incorporation, residue-specific
incorporation has been developed in which a natural amino acid is replaced with
an UAA \cite{Wang2001}. Auxotrophic strains or organisms that cannot
biosynthesize a particular natural amino acid, has been used to introduce
multiple UAAs throughout the protein sequence \cite{Wang2001,Johnson2010}. UAAs
that are isosteric to natural amino acids are capable of being recognized by
the natural aminoacyl-tRNA synthetase (aaRS), charging the appropriate tRNA
enabling the introduction of UAA into the protein sequence without alteration
of the biosynthetic machinery (Figure \ref{fig:rsi} a).
% -------------------------- fig-rsi
\begin{figure}[htbp] \centering \includegraphics[width=0.8\textwidth]{fig1_03} 
    \caption[Illustration  of UAA incorporation via RSI (a) using auxotrophic
    strain, (b) engineering additional copies of endogenous AARS, (c) expanding
the AARS binding pocket and (d) shrinking the AARS editing
pocket.]{Illustration  of UAA incorporation via RSI (a) using auxotrophic
strain, (b) engineering additional copies of endogenous AARS, (c) expanding the
AARS binding pocket and (d) shrinking the AARS editing pocket.} 
\label{fig:rsi} 
\end{figure}
% -------------------------- fig*

However, to introduce UAAs with gross differences from the natural amino acids,
further engineering of the aaRS is required. To incorporate refractory
methionine analogs, Tirrell and coworkers engineered additional copied of the
methionyl-tRNA synthetase (MetRS) by adding the MetRS gene under constitutive
promotor (Figure \ref{fig:rsi} b) \cite{Kiick2000}.  Alternatively, Schimmel
and coworkers mutated editing pocket of valyl-tRNA synthetase (ValRS) to
facilitate the incorporation of analogs that normally would not be accepted by
endogenous aaRS (Figure \ref{fig:rsi} c) \cite{Doring2001}.  Finally, Kast and
coworkers generated a mutated phenylalanyl-tRNA synthetase (PheRS), ePheRS*
under a constitutive promoter, with a large binding pocket (T251G) and showed
relaxed specificity (Figure \ref{fig:rsi} d) \cite{Kast1991}.

\subsubsection{Expressed Protein Ligation}

Expressed protein ligation (EPL) is developed to combine the intein
thiolysis with native chemical ligation steps \cite{Muir2003}. The reaction
involves the purified recombinant protein with a C-terminal intein fusion
reacted with thiophenol or 2-mercaptoethanesulfonic acid, which promotes intein
mediated transthioesterification to produce the C$\alpha$-thioester. The
N-terminal Cys can be synthesized through standard SPPS or recombinantly
generated in which the N-terminal of Cys is fused to an intein that can be
cleaved by thiols, pH change or temperature \cite{Muir1998,Theato2013} (Figure
\ref{fig:epl-intro}). 
% --------------------------fig-epl
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig1_25}
    \caption[The principle of expressed protein ligation. The first step
        involves the purified recombinant protein with a C-terminal intein
        fusion reacted with thiophenol or 2-mercaptoethanesulfonic acid, which
        promotes intein mediated transthioesterification to produce the
        C$\alpha$-thioester. The N-terminal Cys can be synthesized through
        standard SPPS or recombinantly generated in which the N-terminal of Cys
        is fused to an intein that can be cleaved by thiols, pH change or
    temperature.]{The principle of expressed protein ligation
        \cite{Muir1998,Theato2013}.The principle of expressed protein ligation. The first
        step involves the purified recombinant protein with a C-terminal intein
        fusion reacted with thiophenol or 2-mercaptoethanesulfonic acid, which
        promotes intein mediated transthioesterification to produce the
        C$\alpha$-thioester. The N-terminal Cys can be synthesized through
        standard SPPS or recombinantly generated in which the N-terminal of Cys
        is fused to an intein that can be cleaved by thiols, pH change or
    temperature.} 
    \label{fig:epl-intro} 
\end{figure}
% -------------------------- fig*

\subsubsection{Fluorinated Amino Acids in Proteins} 
\label{sec:faa-intro}

Fluorinated amino acids (FAAs), represent a unique class of UAAs. They have
different bond energies, electron distributions, and
hydrophobicities \cite{Biffinger2004} as compared to their hydrogenated
counterparts. With the electro-negativity of 4.0 on the Pauling scaling, the
C-F bond is highly dipolar while the hydrocarbon is less. The C-F bond is
roughly \SI{0.24}{\angstrom} longer than C-H bond (Table
\ref{tab:c-fbond})\cite{Tang2001}. 
% ------------------------- table
\begin{table}[htbp]
\centering
\begin{tabular}{ llll }
  \hline
  Bond & Length & Van der Waals radius & Total size \\
  \hline

  C-H & 1.09 & 1.2 & 2.29 \\
  C-F & 1.35 & 1.7 & 2.82 \\

  \hline
\end{tabular}
\caption[(A) physical properties of the C-F bond. (B) comparison of C-H and C-F
bonds, van der Waals radius, and total size]{(A) physical properties of the C-F
bond. (B) comparison of C-H and C-F bonds, van der Waals radius, and total
size \cite{Tang2001,Odar2015}.}
\label{tab:c-fbond}
\end{table}
% ------------------------- table*

While the global replacement of hydrophobic amino acids with fluorinated
analogs has led to the stabilization of protein structure \cite{Biffinger2004},
it has also been shown that they can reduce the thermodynamic stability
\cite{Panchenko2006b}. The expansion of the genetic code has led to the
biosynthetic incorporation of a wide range of UAAs into proteins
\cite{Voloshchuk2010}. In particular, FAAs have been integrated into a range of
enzymes
\cite{Voloshchuk2009,Panchenko2006b,Voloshchuk2007b,Mehta2011a,Hammill2007} and
biomaterials \cite{Yuvienco2012b}. 

To modulate the substrate specificity, a histone acetyltransferase protein,
PCAF bearing \emph{para}-fluoro-phenylalanine (\emph{p}FF),
\emph{ortho}-fluoro-phenylalanine (\emph{m}FF), and
\emph{meta}-fluoro-phenylalanine (\emph{p}FF) were assayed for substrates
histone H3 and non-histone p53 \cite{Mehta2011a}. Mehta \latin{et al.} have
demonstrated that upon incorporation of pFF, PCAF exhibited improved activity
for p53 and concomitant loss in activity for histone H3. By contrast,
\emph{m}FF-PCAF abolishes the activity for p53, while maintaining activity for
histone H3. 
% ------------------------- table
\begin{table}[htbp]
\centering
\begin{tabular}{ lll }
  \hline
  Protein & k\textsubscript{cat}/K\textsubscript{M}, H3 &
  k\textsubscript{cat}/K\textsubscript{M}, p53 \\ \hline

  PCAF & 9.73 $\pm$ 1.75 & 0.036 $\pm$ 0.012  \\
  \emph{p}FF-PCAF & 2.73 $\pm$ 0.30 & 0.068 $\pm$ 0.002  \\
  \emph{m}FF-PCAF & 0.43 $\pm$ 0.03 & n.a. \\

  \hline
  \multicolumn{3}{l}{n.a. = not available; 
        k\textsubscript{cat}/K\textsubscript{M}:
        $\times$10\textsuperscript{3}\SI{}{\per\Molar\per\second};}
  \end{tabular}
  \caption[Summary of kinetic characterization of PCAF, \emph{p}FF-PCAF, and
  \emph{m}FF-PCAF.]{Summary of kinetic characterization of PCAF,
      \emph{p}FF-PCAF, and \emph{m}FF-PCAF \cite{Mehta2011a}.} \label{tab:PCAF}
\end{table}
% ------------------------- table*

As another example of how fluorination resulted in downstream effects on the
protein structure, Yuvienco \latin{et al.} have demonstrated that supramolecular
assemblies were altered upon fluorination from \emph{p}FF
\cite{Yuvienco2012b}. With the combination of two self-assembling domains:
elastin (E) and the coiled-coil region of cartilage oligomeric matrix proteins
(C), \emph{p}FF-EC, \emph{p}FF-CE, and \emph{p}FF-ECE were generated to
modulate the transition temperature (${T_t}$) (Figure \ref{fig:carlo}). To
assess \emph{p}FF impact, CD spectroscopy was employed for secondary structure
analysis. As illustrated in Figure \ref{fig:carlo} A, \emph{p}FF-EC is
randomly structured at lower temperatures, while \emph{p}FF-CE exhibits
structured conformation at temperatures less than \SI{30}{\celsius}. Notably,
in comparison to the wt proteins, fluorination appeared to decrease the ${T_t}$
of CE protein from 36 to \SI{28}{\celsius} \cite{Yuvienco2012b} (Figure
\ref{fig:carlo}). 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig1_37}
    \caption[CD wavelength spectra collected as a function of temperature for
    (A) \emph{p}FF-EC, (B) \emph{p}FF-CE, and (C) \emph{p}FF-ECE, indicating
the secondary structural changes that accompany the thermoresponsiveness of the
proteins; insets show OD\textsubscript{350} as temperature increases,
indicating the ${T_t}$ for pFF proteins (solid lines) and wt counterparts
(dashed lines), from which are interpreted the bulk meso-/macroscale assembly
that occurs with the addition of heat. For both experiments, samples were
prepared to \SI{4}{\micro\Molar} in phosphate buffer initially at
\SI{4}{\celsius}, establishing congruent preparation protocols.]{CD wavelength
    spectra collected as a function of temperature for (A) \emph{p}FF-EC, (B)
    \emph{p}FF-CE, and (C) \emph{p}FF-ECE, indicating the secondary structural
    changes that accompany the thermoresponsiveness of the proteins; insets
    show OD\textsubscript{350} as temperature increases, indicating the ${T_t}$
    for pFF proteins (solid lines) and wt counterparts (dashed lines), from
    which are interpreted the bulk meso-/macroscale assembly that occurs with
    the addition of heat. For both experiments, samples were prepared to
    \SI{4}{\micro\Molar} in phosphate buffer initially at \SI{4}{\celsius},
    establishing congruent preparation protocols \cite{Yuvienco2012b}.}
    \label{fig:carlo}
\end{figure}
% --------------------------

Although incorporation of FAAs into a target
protein can lead to enhanced function or stability, in some cases loss of
activity or stability occurs, and further improvements to the artificial
protein have been made by rational mutagenesis \cite{Voloshchuk2007b} and
directed evolution strategies \cite{Montclare2006b}.

\subsection{Phosphotriesterase} 
\label{sec:pte-intro}

Phosphotriesterase (PTE) is a homodimeric protein composed of two monomers,
each of which contains a metallo-active site
\cite{Aubert2004b,Benning2001a,Benning1995}. PTE acts as an enzyme, which
hydrolyze organophosphates (OPs) \cite{Ghanem2005a} (Figure
\ref{fig:pte-structure}). The proenzyme form of PTE contains a 29 amino acids
signal peptide at the N-terminus. It is originally found as a 39 kDa monomeric
form in the solution \cite{Mulbry1989}. Later, the proenzyme of PTE is
engineered and expressed in the form of mature protein from \latin{E. coli}. A
($\beta$/$\alpha$)\textsubscript{8} TIM-barrel structure forms the monomeric
PTE \cite{Roodveldt2005,Seibert2005}. The globular monomer is roughly
\SI{51}{\angstrom} $\times$ \SI{55}{\angstrom} $\times$ \SI{51}{\angstrom}
\cite{Hanusa2011}.
% --------------------------fig
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig1_02}
    \caption[Structure and function of phosphotriesterase: (A) Crystal
    structure of PTE (PDB 1HZY). Wild-type PTE consists of two monomers. Shown
    in light blue is one of them, and dark blue is the other. Yellow dots
    represent zinc atoms; (B) Paraoxon hydrolysis by PTE; (C) Active site of
    PTE where small pocket residues are labeled in red: G60,I106, L303, S308;
    large pocket residues are labeled in blue: H254, H257, M317, and leaving
    group residues are labeled in grey: W131, F132, F306, Y309.] {Structure and
        function of phosphotriesterase: (A) Crystal structure of PTE (PDB
        1HZY). Wild-type PTE consists of two monomers.  Shown in light blue is
        one of them, and dark blue is the other. Yellow dots represent zinc
        atoms; (B) Paraoxon hydrolysis by PTE;  (C)  Active site of PTE where
        small pocket residues are labeled in red: G60,I106, L303, S308; large
        pocket residues are labeled in blue: H254, H257, M317, and leaving
        group residues are labeled in grey: W131, F132, F306, Y309.}
\label{fig:pte-structure} 
\end{figure} 
% --------------------------fig

\subsubsection{Hydrolysis Mechanism of PTE}

PTE is able to react with OPs through a nucleophilic attack by \ch{H2O}
associated by Asp301 on the phosphorus center in an SN\textsubscript{2}
mechanism \cite{Lewis1988} (Figure \ref{fig:pte-mechanism}).  Evidence shows the
nucleophilic attack on the phosphorus center from the analysis of reaction
conducted in O\textsuperscript{18}-labeled water \cite{Lewis1988}.  Aubert
\latin{et al.} proposes catalytic mechanism of PTE in which two metal sites,
designated $\alpha$ and $\beta$ are involved in substrate hydrolysis
cite{Aubert2004} (Figure \ref{fig:pte-mechanism}). First step involves a
nucleophilic attack on the phosphorus center and proton transfer to Asp301.
Then, The P-O bond is broken and an complex forms as a phosphate anion that
bridges the two \ch{Zn^{2+}}.The final step involves both the regeneration of
the bridging hydroxyl and the binding of water in the open coordination site on
the $\beta$-metal \cite{Aubert2004}.
% --------------------------fig-pte-mechanism
\begin{figure}[htbp] \centering \includegraphics[width=0.6\textwidth]{fig1_27}
    \caption[Proposed catalytic mechanism of PTE. First step involves a
    nucleophilic attack on the phosphorus center and proton transfer to Asp301.
Then, The P-O bond is broken and an complex forms as a phosphate anion that
bridges the two \ch{Zn^{2+}}. Last, the product is released, and the
active-site hydroxide is regenerated.]{Proposed catalytic mechanism of PTE.
    First step involves a nucleophilic attack on the phosphorus center and
    proton transfer to Asp301. Then, The P-O bond is broken and an complex
    forms as a phosphate anion that bridges the two \ch{Zn^{2+}}. Last, the
    product is released, and the active-site hydroxide is regenerated
    \cite{Aubert2004}.} \label{fig:pte-mechanism} \end{figure}
% -------------------------- fig*

\subsubsection{Significance of Metal On PTE Stability}

Metal ions, \ch{Mn^{2+}}, \ch{Co^{2+}}, \ch{Ni^{2+}}, \ch{Cd^{2+}}, or
\ch{Zn^{2+}} help the hydrolysis of OPs
\cite{Rochu2002b,Carletti2009,Hill2003,Bigley2013,Samples2005,Kim2008}. Omburo
\latin{et al.} have showed that the most active PTE contained \ch{Co^{+}} metal
ions in the active site \cite{Omburo1992a}. From the spectrum of
\ch{^{113}Cd}-NMR, the peaks indicate that PTE incorporates two distinct metals
in the active site \cite{Omburo1993}.  Histidines, including His55, His57,
His201, and His230, interact metal ions at the active site \cite{Benning2001a}
(Figure \ref{fig:pte-structure}). 

Choices of metal ion significantly affects PTE kinetic parameters on its
substrates \cite{Hanusa2011,Perezgasga2012,Carletti2009}. The replacement of
\ch{Zn^{2+}} with \ch{Co^{2+}} has been reported to enhance activity of PTE,
but in coupled with decrease of stability \cite{Rochu2004}. Notably, Rochu
\latin{et al.} have demonstrated the structural stability of PTE with metal
cations upon heat or pH inactivation \cite{Rochu2004} (Figure
\ref{fig:metal-effect}). The half-life of purified \ch{Zn^{2+}}
-PTE was identified to be 16 min at \SI{55}{\celsius} and pH 9. An inactivation
phase was reported for the \ch{Co^{2+}}-PTE at pH 9.4 \cite{Rochu2004}. 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.9\textwidth]{fig1_38}
    \caption[Temperature-dependence profiles are shown for (A)
    \ch{Zn^{2+}}-PTE, (B)\ch{Co^{2+}}-PTE, and (C) \ch{Cd^{2+}}-PTE, at pH 8.0,
8.5, 9.0 and 9.5. Activities are normalized (maximum
activity=100\%).]{Temperature-dependence profiles are shown for (A)
    \ch{Zn^{2+}}-PTE, (B)\ch{Co^{2+}}-PTE, and (C) \ch{Cd^{2+}}-PTE, at pH 8.0,
    8.5, 9.0 and 9.5. Activities are normalized (maximum activity=100\%)
    \cite{Rochu2004}.}
    \label{fig:metal-effect}
\end{figure}
% --------------------------

\subsubsection{Fluorinated PTE}

With the thermo-stabilization property of fluorinated amino acid (See
\ref{sec:faa-intro}) and concept of protein engineering (See \ref{sec:uaa-intro}),
Baker \latin{et al.} designed an improved thermo-stabilized PTE though the
incorporation of \emph{para}-fluoro-phenylalanine (\emph{p}FF)
\cite{Baker2011b} (Figure \ref{fig:PJB} A). With residue specific incorporation
(see \ref{sec:rsi-intro}), all 15 phenylalanines were replaced with \emph{p}FFs,
and to produce \emph{p}FF-PTE. (Figure \ref{fig:PJB})
% --------------------------fig-PJB
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig1_28}
    \caption[(A) Structure of PTE with Phe highlighted. (B) Residual activity
        assay on PTE (black) and \emph{p}FF-PTE (gray) for substrates,
        paraoxon. \emph{p}FF-PTE retained more than 40\% of the initial
        activity for paraoxon upon being heated to \SI{55}{\celsius}. This
        revealed that \emph{p}FF-PTE has enhanced activity for paraoxon.]{(A)
            Structure of PTE with Phe highlighted. (B) Residual activity assay
            on PTE (black) and \emph{p}FF--PTE (gray) for substrates, paraoxon.
            \emph{p}FF-PTE retained more than 40\% of the initial activity for
            paraoxon upon being heated to \SI{55}{\celsius}. This revealed that
            \emph{p}FF-PTE has enhanced activity for paraoxon
            \cite{Baker2011b}.} 
    \label{fig:PJB} 
\end{figure}
% -------------------------- fig*

To assess the \emph{p}FF impact, differential scanning calorimetry (DSC) was
employed to determine the melting temperature (T\textsubscript{m}) of PTE and
\emph{p}FF-PTE. Upon heating the sample to \SI{70}{\celsius}, the fluorination
improved the thermostability of \emph{p}FF-PTE as T\textsubscript{m} were
increased in comparison with wild-type PTE \cite{Baker2011b} (Figure
\ref{fig:PJB-DSC}).  Notably, \emph{p}FF-PTE retained higher residual activity
on its substrate in comparison with wild-type PTE. Figure \ref{fig:PJB}
illustrate that PTE and \emph{p}FF-PTE exhibit 13\% and 60\%, respectively of
initial activity upon elevated temperature at \SI{55}{\celsius}. Results
demonstrate that fluorination provides stabilization and prevents heat
inactivation of PTE \cite{Baker2011b}.
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig1_39}
    \caption[Differential scanning calorimetry of PTE and \emph{p}FF-PTE; (A)
    initial scan of PTE at \SIrange{0}{70}{\celsius}; (B) initial scan of
\emph{p}FF-PTE at \SIrange{0}{70}{\celsius}.]{Differential scanning calorimetry
    of PTE and \emph{p}FF-PTE; (A) initial scan of PTE from
    \SIrange{0}{70}{\celsius}; (B) initial scan of \emph{p}FF-PTE from
    \SIrange{0}{70}{\celsius} \cite{Baker2011b}.} \label{fig:PJB-DSC}
\end{figure}
% --------------------------

\subsection{Scope of Work}

While the \emph{p}FF-PTE demonstrated improved stability, the amount of soluble
protein levels were significantly reduced compared to wild-type PTE. This could
be attributed to the incorporation of \emph{p}FF hydrophobicity
\cite{Baker2011b}. To design more soluble and stable regions of \emph{p}FF-PTE,
we employ Rosetta. Using Rosetta, a computational suite, we identify a
\emph{p}FF-PTE variant that exhibits extended shelf life and improved
stability. 

\section{Methods}

\subsection{General}

\emph{DpnI}, and dNTP were purchased from Roche.  Pfu DNA polymerase was
purchased from Thermo Scientific (Waltham, MA). All other chemicals, including
\ch{NaCl}, \ch{CoCl2}, Tris-HCl, tryptone, yeast extract, paraoxon, ampicillin,
chloramphenicol, sodium phosphates monobasic, sodium phosphate dibasic, were
purchased from Sigma (St. Louis, MO) or VWR (Radnor, PA). DNA sequence was
confirmed by Eurofins MWG Operon.  96-well plates were purchased from Thermo
Fisher Scientific (Waltham, MA). FPLC column was purchased from G.E Healthcare
(Piscataway, NJ). 

\subsection{Recombinant Gene Construction}

pQE30-PTE was used as described before \cite{Baker2011b}. The pQE30-F104A plasmid
was prepared with forward primers (5\rq-GAT GTG TCG ACT \textbf{GCC} GAT ATC GGT
CG-3\rq, Fisher Scientific), reverse primers (5\rq-CG ACC GAT ATC \textbf{GGC} AGT
CGA CAC A-3\rq, Fisher Scientific). The polymerase chain reaction (PCR)
parameters were set as follow for 18 cycles: initial denaturation in
\SI{95}{\celsius} for 30 seconds, sequential denaturation in \SI{95}{\celsius}
for 30 seconds, annealing in \SI{55}{\celsius} for 1 minute, and extension in
\SI{68}{\celsius} for 4 minutes. The mixture was then incubated
\SI{37}{\celsius} overnight with DpnI to digest methylated parent DNA strands,
which lack the desired mutation. DNA sequence was further confirmed by Eurofins
MWG Operon. (See appendix for plasmid map)

\subsection{Protein Expression}
\label{sec:protein-expression-method}

Mutant and wild type plasmids were transformed into electro-competent \latin{E.
coli} phenylalanine auxotrophic strains (AF-IQ cells)[5]. Electroporation was
done at \SI{25}{\micro\farad}, \SI{100}{\ohm}, 2.5 kV (Biorad Gene Pulser II).
Cells were plated on agar plates containing \SI{200}{\ug\per\mL} ampicillin,
\SI{34}{\ug\per\mL} chloramphenicol. A single colony was picked and grown in
medium (M9 medium supplemented with 0.2 wt \% glucose, \SI{35}{\mg\per\L}
thiamine, \SI{1}{\milli\Molar} \ch{MgSO4}, \SI{0.1}{\milli\Molar} \ch{CaCl2},
\SI{200}{\ug\per\mL} ampicillin, and \SI{34}{\ug\per\mL} chloramphenicol) with
\SI{20}{\mg\per\L} of 20 amino acids at \SI{37}{\celsius}, 300 r.p.m for 16
hours \SI{37}{\celsius} incubation.  Afterwards, \SI{250}{\mL} of M9 medium for
large-scale expression was innoculated 1:50 with the overnight culture.  After
optical density reached 1.0 at 600 nm, media shift was carried out by washing
the cells three times with \SI{4}{\celsius} 0.9\% \ch{NaCl}. Cells were then
transferred to M9 minimal medium containing either 20 amino acids or 19 amino
acids (-Phe). Ten minutes of \SI{37}{\celsius} incubation was adapted for
depriving of extra phenylalanine in the medium. \emph{p}FF-PTE and
\emph{p}FF-104A expression media were supplemented with and
\SI{3}{\milli\Molar} of \emph{p}FF and \SI{1}{\milli\Molar}
isopropyl-$\beta$-D-thiogalactopyranoside (IPTG) to induce protein expression.
\SI{1}{\milli\Molar} of \ch{CoCl2} was added in each post-induction medium.
After three hours incubation at \SI{37}{\celsius}, 300 r.p.m., the cells were
harvested by using 4000 r.p.m centrifugation (Beckman Coulter, Jersey City, NJ.
F10 rotor) at \SI{4}{\celsius} for 15 minutes and then resuspended with
\SI{20}{\milli\Molar} Tris-HCl, \SI{500}{\milli\Molar} \ch{NaCl},
\SI{5}{\milli\Molar} imidazole, 10\% glycerol (pH 8.0) and \SI{1}{\micro\Molar}
\ch{CoCl2}. Cell lysate was immediately sonicated for 1.5 minutes at
\SI{4}{\celsius} and then a clarification spin was performed (20, 000 g,
\SI{4}{\celsius}, 30 minutes).  Clarified supernatants were loaded into a
\SI{5}{\mL} His Trap column (G.E Healthcare, Piscataway, NJ) using AKTA FPLC
purifier (G.E.  Healthcare, Piscataway, NJ).  Protein elution was generated
using elution buffer B (\SI{20}{\milli\Molar} Tris-HCl, \SI{500}{\milli\Molar}
sodium chloride, \SI{500}{\milli\Molar} imidazole (pH 8.0)).  The purified
samples were then transferred into 3.5K MWCO dialysis SnakeSkin (Life
Technologies, Carlsbad, CA) for buffer exchange using \SI{12}{\L}
\SI{20}{\milli\Molar} phosphate buffer (pH 8.0).  Dialyzed protein was
subjected to kinetic assays immediately. The purify of protein was determined
by sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE). The
protein concentration was measured by Nano-Drop Thermo
Scientific (Waltham, MA). 

\subsection{Rosetta Design}
\label{sec:pyrosetta-method}

Rosetta\cite{Leaver-Fay2011,DiMaio2011a} was used to generate a symmetric,
\emph{p}FF-incorporated PTE structure used by all simulations. The structure
(PDB code: 1HZY) of wild type PTE was used as the input. In addition to the
phenylalanine positions being mutated to \emph{p}FF, three positions in the
wild-type PTE sequence were mutated (K185R, D208G, and R319S) based on S5PTE to
generate \emph{p}FF-PTE \cite{Roodveldt2005}. Mutations were made using the
Rosetta \emph{fixbb} application and were followed by side chain repacking and
minimization. As amino acids directly interacting with the \ch{Co^{2+}} ions
are crucial in binding the necessary divalent cation for PTE activity [3], 
they were fixed in their native rotamers during repacking and minimization.
PyRosetta, a python interface to the Rosetta libraries[4], was used to make and
characterize point mutations. Every \emph{p}FF position was individually
mutated into any natural amino acid minus phenylalanine. To simulate a
mutation, a single \emph{p}FF position was mutated and neighboring amino acid
within \SI{10}{\angstrom} (as measured by C$\alpha$-C$\alpha$ atom distance) was
allowed to repack and minimize to accommodate the point mutation to fill in
potential high-cost-energy voids or to supplement the hydrophobicity, polarity,
or charge in the vicinity. For each \emph{p}FF position, 500 decoys were
generated. After the mutations were made, representative structures of each
mutation were chosen based on the overall stability of the enzyme, reflected by
the total score. The binding energy is the total energy minus the energy of
both monomers separated by \SI{1000}{\angstrom}. Point mutations were chosen
based on the difference between relative total and predicted binding energies
of the mutant and \emph{p}FF-PTE sequence. As above, amino acids directly
interacting with the \ch{Co^{2+}} ions were fixed in their native rotamers
during repacking and minimization. All Rosetta and PyRosetta calculations were
done using the \emph{score12} score function, and possessed extra rotamer
sampling, including the native rotamers.

\subsection{Thermo-stability and Secondary Structure of Phosphotriesterase}
\label{sec:thermo}

\subsubsection{Nano-DSC}
\label{sec:dsc-method}

Differential scanning calorimetry (Nano-DSC, TA instrument, USA) was performed
by using \SI{600}{\micro\L} (\SI{0.1}{\mg\per\mL}) of protein right after
dialysis into \SI{20}{\milli\Molar} sodium phosphate buffer (pH 8.0).
Measurements were conducted at a scan rate of \SI{1}{\celsius\per\minute} from
\SI{20}{\celsius} to \SI{70}{\celsius}.  Signals was blanked with buffer under
the same condition.  The observed diagram was then analyzed by using
two-scaled model in NanoAnalyze software (TA instrument, USA). Cp and
T\textsubscript{m} were both determined by fitting the two state scaled model
\cite{Privalov1986}. The equation is shown below:
\begin{equation}
    C\textsubscript{p} = \kappa\textsubscript{B}
    \Bigg(\frac{\epsilon}{\kappa\textsubscript{B}T}\Bigg) \times
    \frac{e^{\beta\kappa}}{\big[e^{\beta\kappa}+1\big]^{2}} \label{eqn:dsc}
\end{equation}

\subsubsection{Circular Dichroism}
\label{sec:cd-method}

Circular Dichroism (CD) spectra were recorded on a JASCO J-815 Spectropolarimeter
(Easton, MD) using Spectra Manager software \cite{Kataev1985}. Temperature was
controlled at \SI{25}{\celsius} using a Fisher Isotemp Model 3016S water bath.
Proteins concentrations were \SI{10}{\micro\Molar} in \SI{20}{\milli\Molar}
phosphate buffer (pH 8.0) \SI{600}{\micro\liter}.  \SI{20}{\milli\Molar}
phosphate buffer was used for blanking signals. To calculate mean residue molar
ellipticities (deg $\times$ cm\textsuperscript{2} $\times$
dmol\textsuperscript{-1}), the following formula \cite{Kelly2005} was
used(Eq.~\ref{eqn:CD-chap1}): 
\begin{equation}
    \big[\theta\big] = MRW \times \theta obs / 10 \times c \times l
    \label{eqn:CD-chap1}
\end{equation}
where MRW is the mean residue weight of phosphotriesterase
(\si{\gram\per\mol}), $\theta$obs is the observed ellipticities (mdeg),
\emph{l} is the path length (cm), \emph{c} is the concentration in
\SI{}{\Molar}. Spectra was recorded from \SIrange{190}{250}{\nm} with a scan
speed of \SI{1}{\nano\meter\per\minute}.  The data presented is an average of
three trials.

\subsection{Enzyme Kinetics}
\label{sec:kinetics-method}

The protein was diluted to a final concentration of \SI{30}{\nano\Molar} in
\SI{20}{\milli\Molar} sodium phosphate (pH 8.0) by using the extinction
coefficient \SI{29575}{\per\Molar\per\cm} for all proteins\cite{Gasteiger2005,
Pace1995}.  Reactions were monitored spectrophotometrically (Synergy H1,
BioTek, Winooski VT) at \SI{405}{\nm} for paraoxon (coefficient =
\SI{17000}{\per\Molar\per\cm}) \cite{Baker2011b} in a 96-well plate. Reactions for paraoxon
(\SIrange{13}{104}{\micro\Molar}) was done in 0.2\% methanol.
K\textsubscript{M} and k\textsubscript{cat} values were determined by a
Lineweaver-Burk plot \cite{Baker2011b}. The equation used is shown below
(Eq.~\ref{eqn:MM-chap1}): 
\begin{equation} 
    \frac{1}{v} =
    \frac{K\textsubscript{M}}{V\textsubscript{max}}\times\frac{1}{S} +
    \frac{1}{V\textsubscript{max}} 
    \label{eqn:MM-chap1}
\end{equation}
where S represents substrate concentration; K\textsubscript{M} represents the
substrate concentration at which the reaction rate is half of
V\textsubscript{max}. The data reported is the average of three trials and the
error represents the standard deviation of those trials. Residual activities
and shelf life measurements were conducted with the same batch of proteins
(\SI{30}{\nano\Molar} in \SI{20}{\milli\Molar} sodium phosphate, pH 8.0). For
residual activity assays, proteins at \SI{35}{\celsius}, \SI{45}{\celsius}, and
\SI{55}{\celsius} were cooled back to room temperature for one hour and then
for activity on paraoxon (\SIrange{13}{104}{\micro\Molar}). Half-live
experiments were carried out using proteins (\SI{30}{\nano\Molar} in
\SI{20}{\milli\Molar} sodium phosphate, pH 8.0) that kept under room
temperature for 1, 2, 3, and 7 days.  

\subsection{MALDI-TOF Mass Spectrometry}

To determine level of \emph{p}FF incorporation, \SI{20}{\micro\liter} of
purified PTE, \emph{p}FF-PTE, F104A, or \emph{p}FF-104A was incubated with
\SI{12.5}{\ng\per\uL} of trypsin solution (in \SI{50}{\milli\Molar} of ammonium
bicarbonate) at \SI{37}{\celsius} overnight. \SI{2}{\uL} of 10\%
trifluoroacetic acid (TFA) was used to quench each reaction. Reaction was then
purified with a C\textsubscript{18} packed zip-tip (Millipore, Billerica, MA).
Tips were wet in 50\% acetonitirile (ACN), equilibrated in 0.1\% TFA, and
eluted with 0.1\% TFA in 75\% ACN. Matrix was prepared by dissolving \SI{10}{\mg\per\mL}
$\alpha$-cyano-4-hydrocinnamic acid (CCA) in 50\% ACN, 0.05\% TFA. Theoretical
trypsin digest were calculated from Peptide Mass
(www.expasy.org/tools/peptide-mass.html). Samples were added to the matrix at a
1:1 ratio and spotted on MALDI (Matrix-assisted laser desorption/ionization)
384-well plate. Five standards were spotted separately for calibration:
angiotensin I (MW = \SI{1295.69}{\g\per\mole}), neurotensin (MW =
\SI{1671.92}{\g\per\mole}), ACTH (1-17) (MW = \SI{2092.09}{\g\per\mole}), ACTH
(18-39) (MW = \SI{2464.20}{\g\per\mole}), and ACTH (7-38) (MW =
\SI{3656.93}{\g\per\mole}). Samples were analyzed on Bruker UltrafleXtreme
(Bruker, Billerica, MA). 1000 shots per sample were collected through linear
mode. Compass 1.4 for flex software was then used to analyze the MALDI spectra
(www.bruker.com/). Percentage of incorporation was calculated by the following
Equation \ref{eqn:maldi} \cite{Voloshchuk2009}:
\begin{equation} 
    \% of incorporation =
    \frac{50 \times Ha + 100 \times Hb}{Ha + Hb +Hc}
    \label{eqn:maldi}
\end{equation}
where Ha represented the peak hight of fragments with \emph{p}FF fully
incorporated; Hb represented the peak height of one with partial \emph{p}FF
incorporated; Hc represented the peak height of wild-type fragment. Values
calculated for each peptide fragment on the mass spectrum were averaged to
estimate the total percent of incorporation for the protein.

\section{Results}

\subsection{PyRosetta Design of Phosphotriesterase}

Using PyRosetta (a python interface to the Rosetta
libraries)\cite{Leaver-Fay2011}, every \emph{p}FF position in the model was
examined. Each was allowed to mutate to any amino acid except phenylalanine.
The mutated residue relative to the native \emph{p}FF was evaluated, based on
the total energy and predicted binding energy of the two chains (Section
\ref{sec:pyrosetta-method}). This protocol was used to predict stabilizing
site-specific mutations as well as mutations that would remove phenylalanine
positions that would not tolerate \emph{p}FF substitution. In our
residue-specific \emph{p}FF incorporation system we cannot control which
phenylalanine positions are fluorinated, and thus removing phenylalanine
positions that cannot tolerate fluorination could improve stability if
alternative natural amino acids score well at these positions. For each
mutation, every neighboring amino acid within \SI{10}{\angstrom} of the C$\alpha$
atom of \emph{p}FF was allowed to repack to accommodate the point mutation, by
filling in potential high-cost-energy voids or to supplement the
hydrophobicity, polarity, or charge in the vicinity (Figure
\ref{fig:rosetta-pte-chart}).
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig1_30}
    \caption[The energy scores of the individually mutated \emph{p}FF positions
        within \emph{p}FF-PTE with minimization. Star indicates pFF 104 which
        is known to be in the dimer interface.]{The energy scores of the
            individually mutated \emph{p}FF positions within \emph{p}FF-PTE
            with minimization. Star indicates \emph{p}FF 104 which is known to
        be in the dimer interface.}
    \label{fig:rosetta-pte-chart}
\end{figure}
% --------------------------

Replacing the phenylalanine at position 104 with \emph{p}FF created
energy-costing clashes with neighboring amino acids at the dimerization
interface (Figure \ref{fig:rosetta-pte}). As a dimer is known to be crucial for
function \cite{Baker2011b}, we sought to restabilize the interface with
incorporated \emph{p}FF. Upon mutation of F104 to alternative natural amino
acids, we identified the variant \emph{p}FF-F104A, which exhibited improved
packing in the absence of the \emph{p}FF (Figure \ref{fig:rosetta-pte}).
% --------------------------fig
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig1_11}
    \caption[Structure of \emph{p}FF-F104A identified from Rosetta,
        highlighting \emph{p}FFs and F104A. Comparison of \emph{p}FF-F104A and
        \emph{p}FF-PTE with energy scores shows that \emph{p}FF-PTE exhibits
        steric clash with a neighboring residue.  The original structure (PDB
        ID: 1HZY) was mutated and modified by Rosetta and rendered by UCSF
    Chimera.]{Structure of \emph{p}FF-F104A identified from Rosetta,
    highlighting \emph{p}FFs and F104A. Comparison of \emph{p}FF-F104A and
    \emph{p}FF-PTE with energy scores shows that \emph{p}FF-PTE exhibits steric
    clash with a neighboring residue. The original structure (PDB ID: 1HZY) was
    mutated and modified by Rosetta and rendered by UCSF Chimera.}
    \label{fig:rosetta-pte}
\end{figure}
% --------------------------*

\subsection{Biosynthesis of Proteins}

The \emph{p}FF-F104A variant and the \emph{p}FF-PTE parent were biosynthesized
by residue-specific incorporation with the phenylalanine auxotrophic
\emph{Escherichia coli} strain AFIQ \cite{Yang2014a}. As controls, the
non-fluorinated counterparts, PTE and F104A, were expressed under conventional
conditions. As expected, all four proteins exhibited over-expression in the
presence of phenylalanine or \emph{p}FF as demonstrated by SDS-PAGE gel results(Figure
\ref{fig:sds-gel}). After FPLC purification, SDS-PAGE analysis revealed that
the non-fluorinated protein PTE and F104A yielded \SI{0.94}{\mg} and
\SI{1.86}{\mg}, respectively, where \emph{p}FF-PTE and \emph{p}FF-F104A resulted in
\SI{0.20}{\mg} and \SI{0.4}{\mg}, respectively. (Table \ref{tab:protein-yield})
% --------------------------sds-page-fig
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig1_04}
    \caption[12\% SDS-PAGE of (A) expression of PTE and
    F104A; (B) purified PTE and F104A; and (C) purified \emph{p}FF-PTE and
\emph{p}FF-F104A.]{12\% SDS-PAGE of (A) expression of PTE
    and F104A; (B) purified PTE and F104A; and (C) purified \emph{p}FF-PTE and
    \emph{p}FF-F104A.} 
    \label{fig:sds-gel}
\end{figure}
% --------------------------*

% --------------------------table
\begin{table}[htbp]
\centering
\begin{tabular}{ lll }
  \hline
  Protein & Purified (mg) & Lysate (mg) \\
  \hline
  PTE & 0.94 & 1.87  \\
  F104A & 1.86 & 1.94  \\
  \emph{p}FF-PTE & 0.20 & 0.92  \\
  \emph{p}FF-F104A & 0.40 & 1.27  \\
  \hline

\end{tabular}
\caption[Protein yield of PTE, 104A, \emph{p}FF-PTE, and \emph{p}FF-104A. All
expressed in \SI{250}{\mL} of medium.]{Protein yield of PTE, 104A,
    \emph{p}FF-PTE, and \emph{p}FF-104A. All expressed in \SI{250}{\mL} of
medium.} 
    \label{tab:protein-yield} 
\end{table}
% --------------------------

\emph{p}FF-F104A and \emph{p}FF-PTE exhibited 80\% and 92\%
incorporation, respectively, as determined by MALDI-TOF
mass spectrometry. (Figure \ref{fig:MALDI-fig}) Notably, purified yields of
\emph{p}FF-F104A were two-fold higher than for \emph{p}FF-PTE, thus indicating
more soluble protein yield (Figure \ref{fig:sds-gel}, Table
\ref{tab:protein-yield}).
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=1.0\textwidth]{fig1_05}
    \caption[MALDI-TOF mass spectra of tryptic peptide fragments of (A)
    AWPEFFGSR; and (B) ATPFQELVLK.]{MALDI-TOF mass spectra of tryptic peptide
    fragments of (A) AWPEFFGSR; and (B) ATPFQELVLK.}
    \label{fig:MALDI-fig}
\end{figure}
% --------------------------*

\subsection{Secondary Structure and Stability}

Circular dichroism (CD) was performed to determine whether the mutation had an
impact on the overall secondary structure and stability. Far-UV wavelength
scans of \emph{p}FF-F104A and \emph{p}FF-PTE showed a double minimum at
\SIlist{208;222}{\nm} (\SI{25}{\celsius}), as expected for a
($\beta$/$\alpha$)\textsubscript{8}-barrel protein, thus suggesting that the
mutation did not affect the overall structure (Figure \ref{fig:CD-fig}).
Surprisingly, comparison of the non-fluorinated counterparts revealed that
F104A was less structured than PTE (Figure \ref{fig:CD-fig}). 
% --------------------------
\begin{figure}[htbp] \centering \includegraphics[width=0.7\textwidth]{fig1_06}
    \caption[CD wavelength scans of \SI{10}{\micro\Molar} PTE (circles), 104A (triangles),
    \emph{p}FF-PTE (diamonds) and \emph{p}FF-104A (squares) at
\SI{25}{\celsius}. Data represent an average of three trials.]{CD wavelength
    scans of \SI{10}{\micro\Molar} PTE (circles), 104A (triangles),
    \emph{p}FF-PTE (diamonds) and \emph{p}FF-104A (squares) at
    \SI{25}{\celsius}. Data represent an average of three trials.}
    \label{fig:CD-fig} 
\end{figure}
% --------------------------*

To assess the stability, differential scanning calorimetry (DSC) was performed
(Figure \ref{fig:DSC-fig}). Upon heating the sample from
\SIrange{20}{70}{\celsius}, \emph{p}FF-PTE exhibited two transitions
(T\textsubscript{m}1: 42.0 $\pm$ \SI{0.1}{\celsius}; T\textsubscript{m}2 : 48.6
$\pm$ \SI{0.2}{\celsius}); this was consistent with our previous studies
\cite{Baker2011b} (Figure \ref{fig:DSC-fig}, Table \ref{tab:DSC}). This
biphasic unfolding was also observed by Grimsley \latin{et al.}  in a study of
organophosphorus hydrolase, and was attributed to the presence of a dimeric
unfolded intermediate\cite{Grimsley1997b}. In contrast, \emph{p}FF-F104A
exhibited a single transition at 49.7 $\pm$ \SI{0.2}{\celsius}, which was
higher than both \emph{p}FF-PTE values (by 7.7 and \SI{1.1}{\celsius},
respectively) (Figure \ref{fig:DSC-fig}, Table \ref{tab:DSC}). Remarkably,
after heating, \emph{p}FF-F104A retained the single T\textsubscript{m} of 49.2
$\pm$ \SI{0.1}{\celsius}, thus demonstrating regaining of structure after
undergoing thermal unfolding.  In the absence of \emph{p}FF, F104A demonstrated
two transitions similar to \emph{p}FF-PTE, thus suggesting that fluorination
was critical for stability (Figure \ref{fig:DSC-fig}, Table \ref{tab:DSC}).
These data demonstrate the overall thermodynamic stability of \emph{p}FF-F104A.
% --------------------------DSC-fig
\begin{figure}[htbp] \centering \includegraphics[width=0.9\textwidth]{fig1_07}
    \caption[Differential scanning calorimetry thermograms of (A) F104A and (B)
    \emph{p}FF-F104A.]{Differential scanning calorimetry thermograms of (A)
        F104A and (B) \emph{p}FF-F104A.} \label{fig:DSC-fig} 
\end{figure}
% --------------------------*

% --------------------------DSC-table
\begin{table}[htbp]
\centering
\begin{tabular}{ lll }
  \hline
  Protein & T\textsubscript{m}1 (\si{\celsius}) & T\textsubscript{m}2 (\si{\celsius}) \\
  \hline
  PTE & 41.3 $\pm$ 0.2 & 48.0 $\pm$ 0.2 \\
  F104A & 41.5 $\pm$ 0.2 & 48.2 $\pm$ 0.3 \\
  \emph{p}FF-PTE & 42.0 $\pm$ 0.2 & 48.6 $\pm$ 0.2  \\
  \emph{p}FF-F104A & - & 49.7 $\pm$ 0.2  \\
  \hline
\end{tabular}
\caption[Melting temperatures of PTE, 104A, \emph{p}FF-PTE, and
\emph{p}FF-104A]{Melting temperatures of PTE, 104A, \emph{p}FF-PTE, and
\emph{p}FF-104A.} 
    \label{tab:DSC} 
\end{table}
% --------------------------

\subsection{Kinetics of Phosphotriesterase}

To assess function, we determined the Michaelis-Menten kinetics of
\emph{p}FF-F104A, \emph{p}FF-PTE, F104A, and PTE with paraoxon. At
\SI{25}{\celsius}, \emph{p}FF-PTE exhibited the highest activity
(k\textsubscript{cat}/K\textsubscript{M} = \SI{327000}{\per\Molar\per\second};
Table \ref{tab:kinetics-result}); \emph{p}FF-F104A was slightly lower
(k\textsubscript{cat}/K\textsubscript{M} = \SI{223000}{\per\Molar\per\second}).
Non-fluorinated PTE exhibited k\textsubscript{cat}/K\textsubscript{M} of
\SI{200000}{\per\Molar\per\second}, similar to those both fluorinated proteins;
however, F104A was dramatically less active
(k\textsubscript{cat}/K\textsubscript{M} = \SI{23000}{\per\Molar\per\second},
Table \ref{tab:kinetics-result}).  Thus, the fluorinated amino acids appear to
be necessary for \emph{p}FF-F104A activity. 
% --------------------------
\begin{table}[htbp]
\centering
    \begin{tabular}{llllll}
    \hline
%%
    protein                 &  & \SI{25}{\celsius} & \SI{35}{\celsius} &
    \SI{45}{\celsius} & \SI{55}{\celsius} \\ 
    \hline
%%
    \multirow{2}{*}{PTE}    & k\textsubscript{cat}/K\textsubscript{M} & 2.00$
    \pm$ 0.13 & 0.76 $\pm$ 0.11 & 0.72 $\pm$ 0.12 & 0.46 $\pm$ 0.18 \\
    
    & k\textsubscript{cat} & 2.1 $\pm$ 0.4 & 1.3 $\pm$ 0.1 & 1.4 $\pm$ 0.1 & 0.9
    $\pm$ 0.1 \\
%%
    \multirow{2}{*}{F104A}  & k\textsubscript{cat}/K\textsubscript{M} & 3.27
    $\pm$ 0.11 & 2.42 $\pm$ 0.10 & 1.84 $\pm$ 0.21 & 0.80 $\pm$ 0.09 \\ 
    
    & k\textsubscript{cat} & 6.0 $\pm$ 1.1 & 5.6 $\pm$ 0.1 & 4.0 $\pm$ 1.1 &
    2.0 $\pm$ 0.9 \\
%%
    \multirow{2}{*}{\emph{p}FF-PTE} & k\textsubscript{cat}/K\textsubscript{M} &
    0.23 $\pm$ 0.04 & 0.21 $\pm$ 0.03 & n.a & n.a \\ 
    
    & k\textsubscript{cat} & 0.01 $\pm$ 0.0 & 0.1 $\pm$ 0.0 & n.a & n.a \\
%%
    \multirow{2}{*}{\emph{p}FF-F104A} & k\textsubscript{cat}/K\textsubscript{M}
    & 2.23 $\pm$ 0.15 & 1.94 $\pm$ 0.18 & 1.49 $\pm$ 0.20 & 1.11 $\pm$ 0.09 \\
    & k\textsubscript{cat} & 3.3 $\pm$ 0.3 & 3.3 $\pm$ 0.6 & 2.6 $\pm$ 1.0 &
    2.0 $\pm$ 0.7 \\ 
    
    \hline
    \multicolumn{6}{l}{n.a = not available; 
        k\textsubscript{cat}/K\textsubscript{M}:
        $\times$10\textsuperscript{5}\SI{}{\per\Molar\per\second};
        k\textsubscript{cat}: \SI{}{\per\second}.}            
    \end{tabular}
    \caption[Paraoxon hydrolysis efficiency summary of PTE, F104A,
    \emph{p}FF-PTE, and \emph{p}FF-F104A. Residual activities were preformed
after incubation at \SIlist{35;45;55}{\celsius}.]{Paraoxon hydrolysis
    efficiency summary of PTE, F104A, \emph{p}FF-PTE, and \emph{p}FF-F104A.
    Residual activities were preformed after incubation at
    \SIlist{35;45;55}{\celsius}.} 
    \label{tab:kinetics-result}
\end{table}
% --------------------------*

\subsection{Thermo-activity and Stability Over Time}

Proteins were then incubated at \SIlist{35;45;55}{\celsius} for one hour, and
then cooled to room temperature for an hour to determine residual activity. A
decline in residual activity was observed for all proteins as a function of
elevated temperature. \emph{p}FF-F104A, which was designed to stabilize the
fluorinated protein, exhibited 50\% retention of activity at \SI{55}{\celsius}
(Figure \ref{fig:kinetics-fig}, Table \ref{tab:kinetics-result}). In contrast,
at \SI{55}{\celsius}, \emph{p}FF-PTE and PTE exhibited 24\% and 23\% initial
activity, respectively; F104A exhibited a significant loss in activity at or
above \SI{45}{\celsius} (Figure \ref{fig:kinetics-fig}, Table
\ref{tab:kinetics-result}). 
% --------------------------
\begin{table}[htbp]
\centering
    \begin{tabular}{llllll}
    \hline
%%
    protein                 &  & Day 1 & Day 2 & Day 3 & Day 7\\ 
    \hline
%%
    \multirow{2}{*}{PTE}    & k\textsubscript{cat}/K\textsubscript{M} & 2.06$
    \pm$ 0.23 & 1.89 $\pm$ 0.23 & 1.53 $\pm$ 0.18 & 0.89 $\pm$ 0.10 \\
    
    & k\textsubscript{cat} & 2.5 $\pm$ 0.5 & 0.97 $\pm$ 0.03 & 0.23 $\pm$ 0.01 & 0.10
    $\pm$ 0.01 \\
%%
    \multirow{2}{*}{F104A}  & k\textsubscript{cat}/K\textsubscript{M} & 3.53
    $\pm$ 0.12 & 2.01 $\pm$ 0.28 & 1.62 $\pm$ 0.21 & 1.13 $\pm$ 0.09 \\ 
    
    & k\textsubscript{cat} & 6.23 $\pm$ 0.97 & 3.23 $\pm$ 0.88 & 2.13 $\pm$ 0.94 &
    1.46 $\pm$ 0.97 \\
%%
    \multirow{2}{*}{\emph{p}FF-PTE} & k\textsubscript{cat}/K\textsubscript{M} &
    0.18 $\pm$ 0.02 & n.a & n.a & n.a \\ 
    
    & k\textsubscript{cat} & 0.11 $\pm$ 0.02 & n.a & n.a & n.a \\
%%
    \multirow{2}{*}{\emph{p}FF-F104A} & k\textsubscript{cat}/K\textsubscript{M}
    & 1.83 $\pm$ 0.21 & 1.72 $\pm$ 0.20 & 1.43 $\pm$ 0.13 & 1.20 $\pm$ 0.10 \\

    & k\textsubscript{cat} & 2.25 $\pm$ 0.32 & 2.03 $\pm$ 0.31 & 1.61 $\pm$ 0.25 &
    1.43 $\pm$ 0.25 \\ 

    \hline
    \multicolumn{6}{l}{n.a = not available; 
        k\textsubscript{cat}/K\textsubscript{M}:
        $\times$10\textsuperscript{5}\SI{}{\per\Molar\per\second};
        k\textsubscript{cat}: \SI{}{\per\second}.}            
    \end{tabular}
    \caption[Kinetics of paraoxon hydrolysis as a function of time.]{Kinetics
    of paraoxon hydrolysis as a function of time.} \label{tab:kinetics-day-result} 
\end{table}
% --------------------------*
As \emph{p}FF-F104A retained substantial activity
after elevated temperatures, we then investigated the half-life of the
activity. The parent \emph{p}FF-PTE exhibited more than 50\% loss of activity
after three days, whereas the non-fluorinated PTE showed more than 50\%
activity reduction after seven days (Figure \ref{fig:kinetics-fig}, Table
\ref{tab:kinetics-day-result}).  Remarkably, \emph{p}FF-F104A retained 66\%
activity after seven days. The non-fluorinated counterpart F104A failed to
exhibit activity after one day.  Together these data confirm that
\emph{p}FF-F104A is able to delay heat inactivation while maintaining function
after one week.
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=1.0\textwidth]{fig1_08}
    \caption[(A) Residual activity profile of all proteins, (B) Activity
    measured as a function of days. The non-fluorinated counterpart F104A
failed to exhibit activity after one day.]{(A) Residual activity profile of all
    proteins, (B) Activity measured as a function of days. The non-fluorinated
    counterpart F104A failed to exhibit activity after one day.} 
    \label{fig:kinetics-fig} 
\end{figure}
% --------------------------*

\section{Discussion}

We demonstrate the use of Rosetta to engineer phosphotriesterase bearing
\emph{p}FF. Based on the score function, F104 is identified as a
crucial residue for \emph{p}FF-PTE stabilization. With the higher
hydrophobicity and longer bond length properties from \emph{p}FF, this analog
may introduce unfavorable interactions.  As we simulate the \emph{p}FF-PTE
structure, we discover the steric clash between F104 and the neighboring
residue (Figure \ref{fig:rosetta-pte}). To remove this unfavorable interaction,
we generate F104A with site-directed mutagenesis, and further incorporate
\emph{p}FF into both wild-type PTE and F104A via residue-specific incorporation.
After the removal of unfavorable clash, the \emph{p}FF-F104A leads to a 2-fold
increase of protein yield relative to \emph{p}FF-PTE, while further extending
the half-life of PTE.

\subsection{Significance of Stability}

The \emph{p}FF-F104A mutant exhibits enhanced thermo-activity and half-life.
Investigation of the thermodynamics from DSC provides insights into changes.
DSC data suggest that the unfolding model has been altered from a 3-state to
2-state transition, and that the energy requirement to attaining the unfolded
intermediate is increased, thereby resulting in a more cooperative transition.
Mutations resulting in this particular transformation have been observed for
other proteins; for example, Fan \latin{et al.} showed that removal of a
C-terminal domain of the oligomeric \latin{E. coli} trigger factor protein
resulted in the transformation of an otherwise n-state unfolding process to a
distinct two-state unfolding process, indicative of pronounced stabilization of
the native structure by interdomain interactions. We propose that the
\emph{p}FF-F104A mutation might also be stabilizing the native structure of the
overall protein (in effect the reverse of the mutation observed by Fan
\latin{et al.}) \cite{Fan2008}. That is, \emph{p}FF-F104A unfolds cooperatively
in a single step, concurrent with its dissociation into monomeric
species. Although it was expected that the \emph{p}FF-F104A mutation would have
an effect on interdomain stability (such that neighboring residues would be
allowed to repack and energy would be minimized between monomers at the dimer
interface), the apparent stabilization of the native structure concluded from
the 3-state to 2-state transition transformation through these new interdomain
interactions were indeed unanticipated. Prior examples of this transformation
exist in cases involving subdomains of similar stabilities or where strong
coupling exists between subdomains \cite{Tsytlonok2013}.

\section{Conclusion}

Although methods enabling the biosynthesis of artificial proteins bearing UAAs
are abundant \cite{Voloshchuk2009}, tools to help further improve the overall
activity and stability are needed. Mutagenesis and evolutionary approaches have
been employed successfully to identify variants with enhanced function;
however, these rely heavily on testing or screening several to millions of
constructs \cite{Voloshchuk2007b,Montclare2006b,Yoo2007}. We demonstrate the use
of computational methods to identify a fluorinated protein variant that
exhibits superior heat stability and half-life. Notably, the \emph{p}FF-F104A
variant is only functional in the fluorinated form, thus validating
Rosetta-based design with \emph{p}FF. This provides another useful tool for
protein design and could be employed in conjunction with the above mentioned
approaches.

\printbibliography[heading=subbibliography]
\end{refsection}
% deleted content
% ------------------------- table-protein-engineering
%%\begin{table}[h!]
%%    \centering
%%    \begin{tabular}{ ll }
%%        \hline
%%        Method & Reference(s) \\
%%        \hline
%%        
%%        Rational design & \cite{Arnold1993}, Section\ref{sec:rational-design} \\
%%        Site-directed mutagenesis & \cite{Antikainen2005a} \\
%%        Random mutagenesis & \cite{Wong2006, Jackson2006, Labrou2010} \\
%%        DNA shuffling & \cite{ Jackson2006, Antikainen2005a} \\
%%        Molecular dynamics & \cite{Anthonsen1994} \\
%%        Peptidomimetics & \cite{Venkatesan2002} \\
%%        Phage display technology & \cite{Sidhu2007,Chaput2008} \\
%%        \emph{De novo} enzyme engineering & \cite{Golynskiy2010} \\
%%
%%        \hline
%%    \end{tabular}
%%    \caption[Examples of methods used for protein engineering.]{Examples of
%%        methods used for protein engineering\cite{Antikainen2005a}.}
%%        \label{tab:protein-engineering}
%%\end{table}
% ------------------------- table*
%
% --------------------------deleted-content
%Photo-responsive unnatural amino acids that crosslinked with nearby
%molecules have been developed for enabling the \latin{in vivo} manipulation of
%proteins \cite{Chin2002}. Chin \latin{et al.} used an evolved orthogonal
%aminoacyl-tRNA synthetase/tRNA pair to incorporate of
%\emph{p}-benzoyl-phenylalanine (\emph{p}Bpa) into protein glutathione
%\emph{S}-transferase \cite{Chin2002}. The efficient crosslinking (50\%) proves
%the usefulness for defining protein interactions \latin{in vivo} and \latin{in
%vitro}. 
% --------------------------
%
% Overall, proteins play a crucial role within cells and make up ~55\% the dry
% weight of an \emph{Escherichia coli} cell\cite{Neidhardt1990}. 
%
% --------------------------deleted content
%% Methods to incorporate amino acid analogues site-specifically into proteins
%% \latin{in vivo} greatly expand research of unnatural amino acids. We are not
%%only able to synthesize large amounts of protein, but capable of overcoming
%% potential problems including post-translational modifications.
% --------------------------
%
%%In addition to re-design, algorithms can also be employed in different
%%applications, including alanine enzyme–substrate \cite{Bolon2001,Jiang2008} and
%%protein–nucleic acid interactions \cite{Ashworth2006}. Numerous computational
%%tools have already shown promising results for the protein design.
%
% -------------------------- In chpater 2
% OPs are a synthetic class of small molecule
% that irreversibly inactivate acetylcholinesterase (AChE), disrupting neural
% transmission. AChE is an enzyme that degrades the neurotransmitter,
% acetylcholine, at the neuromuscular junction in the cholinergic nervous system.
% After the acetylcholine is hydrolyzed, the synaptic transmission would be
% terminated. Inhibition of AChE lead to hyper-stimulation from toxic
% accumulation of acetylcholine\cite{Soreq2001}. Army also adapted this protein
% for chemical weapons neutralization \cite{Yang2014a}.
% --------------------------
%
% Rosetta simulation explains the clash at the dimer interface, and we would like to
% investigate how this single mutation affects the overall PTE stability from
% different perspective. CD spectrum indicates the loss of secondary structure in
% the presence of \emph{p}FF. However, the differences between \emph{p}FF-PTE and
% \emph{p}FF-F104A are merely marginal. 
%
% --------------------------deleted content
%% One of UAAs, fluorinated analogs, has been studied extensively in the hope that it
%% will provide benefits to protein engineering, including properties of increased
%% hydrophobicity \cite{Biffinger2004}, thermostability
%% \cite{Voloshchuk2010,Baker2011b}, and chemical denaturant stability
%% \cite{Voloshchuk2007b}. The replacement of hydrophobic amino acids like leucine
%% and phenylalanine with their fluorinated analogs has provided content-dependent
%% results \cite{Tang2001}.
% --------------------------*

