\chapter{Improved Stability and Half-Life of Fluorinated Phosphotriesterase
Using Rosetta} 
\label{chap:rosetta}

\begin{refsection}

\section{Introduction}

\subsection{Rosetta and Protein Engineering}
\label{sec:rosetta}

% Introduce Rosetta and protein engineering. Also focus on protein engineering.
Computational tools are widely used for protein engineering [ref]. Rosetta
suite was first developed in University of Washington [ref]. Baker et al.
adapted this suite for prediction of three: dimensional structure of proteins.
This suite provides a handful of protocols for analyzing and mutating protein
structures. The simulation replies heavily on knowledge-based potentials. It is
a suite of libraries and tools for macromolecular ligand docking, to
thermo-stabilize proteins, to design a hydrogen-bond network, to design novel
protein folds, to create novel protein interfaces, and to design enzymes,
including some containing unnatural amino acids in the active sites. (see
Figure \ref{fig:test})
% --------------------------
\begin{figure}[h!] \centering \includegraphics[width=0.7\textwidth]{fig1_01}
    \caption[This is test]{Rosetta Test}\label{fig:test}

\end{figure}
% --------------------------
% fig1_01: need a real sketch of Rosetta

\subsection{Phosphotriesterase} 
\label{sec:pte}

% This section focuses on PTE.
PTE is a homodimeric protein composed of two monomers, each of which contains a
metallo-active site.Phosphotriesterase (PTE) are enzymes, which hydrolyze
organophosphates (OPs) as well as synthetic esters.\cite{Ghanem2005a} OPs are a
synthetic class of small molecule that irreversibly inactivate
acetylcholinesterase (AChE), disrupting neural transmission. AChE is an enzyme
that degrades the neurotransmitter, acetylcholine, at the neuromuscular
junction in the cholinergic nervous system. After the acetylcholine is
hydrolyzed, the synaptic transmission would be terminated. Inhibition of AChE
lead to hyper-stimulation from toxic accumulation of
acetylcholine.\cite{Soreq2001} Army also adapted this protein for chemical
weapons neutralization. \cite{Yang2014a}
% fig1_02: insert of crystal structure of PTE. Maybe add synapse photo?

\subsection{Incorporation of Non-natural Amino Acids}
\label{sec:rsi}

Several methods have been developed for the incorporation of unnatural amino
acids into proteins: solid-phase synthesis (SPPS)\cite{Mahto2011}, in vivo and
in vitro site-specific incorporation,\cite{Cellitti2008,Hassan2008} and
residue-specific incorporation (Fig. 1)1d.\cite{Johnson2010} In SPPS, activated
amino acids are immobilized on a solid support and synthesized step-by-step in
the reactant solution. This method is convenient for the introduction of
functional groups into peptides, but it is still restricted to the yield and
the expense of peptides. For example, if each coupling step has 99\% yield, a
26-amino acid peptide would be synthesized in 77\% final yield. To synthesis
longer chain peptides and proteins bearing UAAs, biosynthetic methods have been
developed.  There exists two contemporary methods to biosynthetically
incorporate non-natural amino acids into proteins: site-specific incorporation
and residue-specific incorporation. Bain \latin{et al.} have developed a
general approach for the in vitro synthesis of proteins.\cite{Bain1991} The
approach relies on the suppression of an amber termination codon (UAG) in the
mRNA by an amber suppressor tRNA charged with the amino acid analog. This
method has been well studied and developed in research of protein structures
and functions.\cite{Martoglio1995,Eichler1997} 

Methods to incorporate amino acid analogues site-specifically into proteins in
vivo greatly expand research of unnatural amino acids. We are not only able to
synthesize large amounts of protein, but capable of overcoming potential
problems including post translational modifications. An in vivo site-specific
method to incorporation UAAs was developed by Schultz and
coworkers.\cite{Wang2001,Wang2002} A stop codon at the position of interest is
encoded in the mRNA. For in vivo site-specific UAA incorporation, an orthogonal
aminoacyl-tRNA synthetase charges an orthogonal tRNA with particular UAA, and
the suppressor tRNA would help the incorporation of UAA with recognition of a
stop codon. As cells contain 20 aminoacyl-tRNA synthetase/suppressor tRNA
pairs, a new one is required for the incorporation. An orthogonal
aminoacyl-tRNA synthetase/suppressor tRNA pair based on a TyrRS/tRNATyr pair in
the \emph{Methanococcus jannaschii} has been engineered for use in \emph{E.
coli} for the incorporation of tyrosine analogs.\cite{Wang2001}
% fig1_03: add scheme of rsi

As an alternative to site-specific incorporation, residue-specific
incorporation has been developed in which a natural amino acid is replaced with
an UAA. Auxotrophic strains or organisms that cannot biosynthesize a particular
natural amino acid, has been used to introduce multiple UAAs throughout the
protein sequence. UAAs that are isosteric to natural amino acids are capable of
being recognized by the natural aminoacyl-tRNA synthetase (aaRS), charging the
appropriate tRNA enabling the introduction of UAA into the protein sequence
without alteration of the biosynthetic machinery. However, to introduce UAAs
with gross differences from the natural amino acids, further engineering of the
aaRS is required. To incorporate refractory methionine analogs, Tirrell and
coworkers engineered additional copied of the methionyl-tRNA synthetase (MetRS)
by adding the MetRS gene under constitutive promotor.\cite{Kiick2000}
Alternatively, Schimmel and coworkers mutated editing pocket of valyl-tRNA
synthetase (ValRS) to facilitate the incorporation of analogs that normally
would not be accepted by endogenous aaRS.\cite{Doring2001} Finally, Kast and
coworkers generated a mutated phenylalanyl-tRNA synthetase (PheRS), ePheRS*
under a constitutive promoter, with a large binding pocket (T251G) and showed
relaxed specificity.\cite{Kast1991}

\subsection{Fluorinated Amino Acids In Proteins} 
\label{sec:faa}

% Describe the properties of faa in the proteins.
Fluorinated amino acids (FAAs), represent a unique class of UAAs. They have
different bond energies, electron distributions, and hydrophobzgicity 26 as
compared to their hydrogenated counterparts. As we compare the structure of
fluorocarbon groups, the C-F bond is highly dipolar while the hydrocarbon is
less. The C-F bond is roughly 0.24 {\AA} longer than C-H bond.\cite{Tang2001}
While in some cases the global replacement of hydrophobic amino acids with
fluorinated analogs has led to the stabilization of protein
structure\cite{Biffinger2004}, it has all been shown that in some cases they
can reduce the thermodynamic stability.\cite{Panchenko2006b} The expansion of
the genetic code has led to the biosynthetic incorporation of a wide range of
noncanonical amino acids (NCAAs) into proteins.\cite{Voloshchuk2010} In
particular, fluorinated amino acids (FAAs) have been integrated into small
coiled-coil proteins,\cite{Montclare2009b,Tang2001}a range of
enzymes,\cite{Voloshchuk2009,Panchenko2006b,Voloshchuk2007b,Mehta2011,Hammill2007}
and biomaterials.\cite{Yuvienco2012b} Although incorporation of FAAs into a
target protein can lead to enhanced function or stability, in some cases loss
of activity or stability occurs, and further improvements to the artificial
protein have been made by rational mutagenesis\cite{Voloshchuk2007b} and
directed evolution strategies.\cite{Montclare2006b}
% fig1_04: add comparison of phe vs pFF
% table1_01: bond length comparison

\subsection{Scope of Work}

The primary goals of this work were to adapt Rosetta for phosphotriesterase.
Overall, with incorporation of \emph{p}FF into protein, we will be able
evaluate the performance of scoring function. In advance, we would evaluate the
shelf life and thermo-stability of phosphotriesterase.

\section{Methods}

\subsection{General}

All chemicals, reagents, and substrate were purchased from Sigma. T4 DNA ligase
was purchased from Roche. DNA sequence was confirmed by Eurofins MWG Operon.
96-well plates were purchased from Thermo Fisher Scientific (Waltham, MA).

\subsection{Recombinant Gene Construction}

pQE30-S5 was used as described before.\cite{Baker2011b} The pQE30-104A plasmid
was prepared with forward primers (5’-GATGTGTCGACTGCCGATATCGGTCG-3’, Fisher
Scientific), reverse primers (5’-CGACCGATATCGGCAGTCGACACA-3’, Fisher
Scientific). The PCR parameters were set as follow for 18 cycles: initial
denaturation in \SI{95}{\celsius} for 30 seconds, sequential denaturation in
\SI{95}{\celsius} for 30 seconds, annealing in \SI{55}{\celsius} for 1 minute,
and extension in \SI{68}{\celsius} for 4 minutes. The mixture was then
incubated \SI{37}{\celsius} overnight with DpnI to digest methylated parent DNA
strands, which lack the desired mutation. DNA sequence was further confirmed by
Eurofins MWG Operon.

\subsection{Protein Expression}

Mutant and wild type plasmids were transformed into \latin{E. coli} phenylalanine
auxotrophic strains (AF-IQ cells).[5] Electroporation was done at
\SI{25}{\micro\farad}, \SI{100}{\ohm}, 2.5 kV (Biorad Gene Pulser II). Cells were
plated on agar plates containing \SI{200}{\ug\per\mL} ampicillin, \SI{34}{\ug\per\mL}
chloramphenicol. A Single colony was picked and grown in medium (M9 medium
supplemented with 0.2 wt \% glucose, \SI{35}{\mg\per\L} thiamine, \SI{1}{\milli\Molar}
\ch{MgSO4}, \SI{0.1}{\milli\Molar}\ch{CaCl2}, \SI{200}{\ug\per\mL} ampicillin, and
\SI{34}{\ug\per\mL} chloramphenicol) with \SI{20}{\mg\per\L} of 20 amino acids
at \SI{37}{\celsius}, 300 r.p.m.  Afterwards, \SI{250}{\mL} of M9 medium for
large-scale expression was innoculated 1:50 with an overnight culture.  After
optical density reached 1.0 at 600 nm, media shift was carried out by washing
the cells three times with 0.9\% \SI{4}{\celsius} \ch{NaCl}.  Cells were then
transferred to M9 minimal medium containing either 20 amino acids or 19 amino
acids (-Phe). \emph{p}FF-PTE and \emph{p}FF-104A expression media were
supplemented with and \SI{3}{\milli\Molar} of \emph{p}FF and
\SI{1}{\milli\Molar} isopropyl-$\beta$-D-thiogalactopyranoside (IPTG) to induce
protein expression.  \SI{1}{\milli\Molar} of \ch{CoCl2} was added in each
post-induction medium. After three hours incubation at \SI{37}{\celsius}, 300
r.p.m., the cells were harvested and then resuspended with
\SI{20}{\milli\Molar} Tris-HCl, \SI{500}{\milli\Molar} \ch{NaCl},
\SI{5}{\milli\Molar} imidazole, 10\% glycerol (pH 8.0) and \SI{1}{\micro\Molar}
\ch{CoCl2}. Cell lysate was sonicated on ice for 1.5 minutes and then a
clarification spin was performed (20, 000 g, \SI{4}{\celsius}, 30 min).
Clarified supernatants were loaded into a His Trap column (G.E Healthcare,
Piscataway, NJ) using ÄKTA FPLC purifier (G.E.  Healthcare, Piscataway, NJ).
Protein elution was generated using elution buffer B (\SI{20}{\milli\Molar}
Tris-HCl, \SI{500}{\milli\Molar} sodium chloride, \SI{500}{\milli\Molar}
imidazole (pH 8.0)).  The purified samples were then transferred for buffer
exchange using \SI{12}{\L} \SI{20}{\milli\Molar} phosphate buffer (pH 8.0).
Dialyzed protein was subjected to kinetic assays immediately.

\subsection{PyRosetta Design}
\label{sec:rosetta}

Rosetta[1] was used to generate a symmetric, \emph{p}FF-incorporated PTE
structure used by all simulations. The Holden and coworkers structure (PDB
code: 1HZY) of wild type PTE was used as the input. In addition to the
phenylalanine positions being mutated to \emph{p}FF, three positions in the
wild-type PTE sequence were mutated (K185R, D208G, and R319S) to generate
\emph{p}FF-PTE.[2] Mutations were made using the Rosetta \emph{fixbb}
application and were followed by side chain repacking and minimization. The
amino acids directly interacting with the \ch{Co^{2+}} ions are crucial in
binding the necessary divalent cation for PTE activity,[3] so they were fixed
in their native rotamers during repacking and minimization. PyRosetta, a python
interface to the Rosetta libraries,[4] was used to make and characterize point
mutations. Every \emph{p}FF position was individually mutated into any natural
amino acid minus phenylalanine. To simulate a mutation, a single \emph{p}FF
position would be mutated and neighboring amino acid within 10\AA (as measured
by C$\alpha$-C$\alpha$ atom distance) was allowed to repack and minimize to
accommodate the point mutation to fill in potential high-cost-energy voids or
to supplement the hydrophobicity, polarity, or charge in the vicinity. For each
\emph{p}FF position, 500 decoys were generated. After the mutations were made,
representative structures of each mutation were chosen based on the overall
stability of the enzyme, reflected by the total score. The binding energy is
the total energy minus the energy of both monomers separated by 1000\AA. Point
mutations were chosen based on the difference between relative total and
predicted binding energies of the mutant and \emph{p}FF-PTE sequence. As above,
amino acids directly interacting with the \ch{Co^{2+}} ions were fixed in their
native rotamers during repacking and minimization. All Rosetta and PyRosetta
calculations were done using the \emph{score12} score function, and included
extra rotamer sampling, including the native rotamers.

\subsection{Thermo-stability and Secondary Structure of Phosphotriesterase}
\label{sec:thermo}

\subsubsection{Nano-DSC}
\label{sec:dsc}

DSC (Nano-DSC, TA instrument, USA) was preformed by using \SI{600}{\micro\L}
(\SI{0.1}{\mg\per\mL}) of protein right after dialysis. Measurements were
conducted at a scan rate of \SI{1}{\celsius\per\minute}. Signals was blanked with
buffer under the same condition.  The observed diagram was then analyzed by
using NanoAnalyze software.

\subsubsection{Circular Dichroism}
\label{sec:cd}

CD spectra were recorded on a JASCO J-815 Spectropolarimeter (Easton, MD) using
Spectra Manager software. Temperature was controlled using a Fisher Isotemp
Model 3016S water bath. Proteins concentrations were \SI{10}{\micro\Molar} in
\SI{20}{\milli\Molar} phosphate buffer (pH 8.0). \SI{20}{\milli\Molar}
phosphate buffer was used for blanking signals. To calculate ellipticities, the
following formula was used(Eq.~\ref{eqn:CD}): 
\begin{equation}
    θmrw = MRW(θobs) / (10 * c * l) 
    \label{eqn:CD}
\end{equation}
where \emph{MRW} is the mean residue weight of the specific phosphotriesterase,
θobs is the observed ellipticities (mdeg), \emph{l} is the path length (cm),
\emph{c} is the concentration in \SI{}{\micro\Molar}. Spectra was recorded from
\SIrange{190}{250}{\nm} with a scan speed of \SI{1}{\nano\meter\per\minute}.

\subsection{Enzyme Kinetics}
\label{sec:kinetics}

The protein was diluted to a final concentration of \SI{30}{\nano\Molar} in
\SI{20}{\milli\Molar} sodium phosphate (pH 8.0) by using the extinction
coefficient \SI{29280}{\per\Molar\per\cm}. Reactions were monitored
spectrophotometrically (Synergy H1, BioTek, Winooski VT) at \SI{405}{\nm} for
paraoxon (coefficient = \SI{17000}{\per\Molar\per\cm}).  Reactions for paraoxon
(\SIrange{13}{104}{\micro\Molar}) was done in 0.4\% methanol.
K\textsubscript{M} and k\textsubscript{cat} values were determined by a
Lineweaver-Burk plot.\cite{Baker2011b} The equation used is shown below
(Eq.~\ref{eqn:MM}): 
\begin{equation} 
    \frac{1}{v} =
    \frac{K\textsubscript{M}}{V\textsubscript{max}}\times\frac{1}{S} +
    \frac{1}{V\textsubscript{max}} 
    \label{eqn:MM}
\end{equation}
where S represents substrate concentration; K\textsubscript{M} represents the
substrate concentration at which the reaction rate is half of
V\textsubscript{max}. The data reported is the average of three trials and the
error represents the standard deviation of those trials.

\subsection{MALDI-TOF Mass Spectrometry}

To determine level of \emph{p}FF incorporation, \SI{20}{\micro\liter} of
purified PTE \emph{p}FF-PTE, F104A, or \emph{p}FF-104A was incubated with
\SI{12.5}{\ng\per\uL} of trypsin solution (in \SI{50}{\milli\Molar} of ammonium
bicarbonate) at \SI{37}{\celsius} overnight. \SI{2}{\uL} of 10\%
trifluoroacetic acid (TFA) was used to quench each reaction. Reaction was then
purified with a C\textsubscript{18} packed zip-tip (Millipore, Billerica, MA).
Tips were wetted in 50\% acetonitirile (ACN), equilibrated in 0.1\% TFA, and
eluted with 0.1\% TFA in 75\% ACN. Matrix was dissolved in \SI{10}{\mg\per\mL}
$\alpha$-cyano-4-hydrocinnamic acid (CCA) in 50\% ACN, 0.05\% TFA. Theoretical
trypsin digest were calculated from Peptide Mass
(www.expasy.org/tools/peptide-mass.html). Samples were added to the matrix at a
1:1 ratio and spotted on MALDI plate. Five standards were spotted separately
for calibration: angiotensin I (MW = \SI{1295.69}{\g\per\mole}), neurotensin
(MW = \SI{1671.92}{\g\per\mole}), ACTH (1-17) (MW = \SI{2092.09}{\g\per\mole}),
ACTH (18-39) (MW = \SI{2464.20}{\g\per\mole}), and ACTH (7-38) (MW =
\SI{3656.93}{\g\per\mole}).  Compass 1.4 for flex software was then used to
analyze the MALDI spectra (www.bruker.com/).

\section{Results and Discussion}

\subsection{Biosynthesis of phosphotriesterase}

% add gel of PTE.
The \emph{p}FF-F104A variant and the \emph{p}FF-PTE parent were biosynthesized
by residue-specific incorporation with the phenylalanine auxotrophic
\emph{Escherichia coli} strain AFIQ.[6] As controls, the non-fluorinated
counterparts, PTE and F104A, were expressed under conventional conditions. As
expected, all four proteins exhibited good expression in the presence of
phenylalanine or \emph{p}FF. \emph{p}FF-F104A and \emph{p}FF-PTE exhibited 80\%
and 92\% incorporation, respectively, as determined by MALDI-TOF mass
spectrometry. Notably, purified yields of \emph{p}FF-F104A were twofold higher
than for \emph{p}FF-PTE, thus indicating more soluble protein yield.

\subsection{Thermo-stability And Secondary Structure}

% add figure of DSC and CD.
Circular dichroism (CD) was performed to determine whether the mutation had an
impact on the overall secondary structure and stability. Far-UV wavelength
scans of \emph{p}FF-F104A and \emph{p}FF-PTE showed a double minimum at
\SIlist{208;222}{\nm} (\SI{25}{\celsius}), as expected for a
($\beta$/$\alpha$)\textsubscript{8}-barrel protein, thus suggesting that the
mutation did not affect the overall structure (Figure S3).  Surprisingly,
comparison of the non-fluorinated counterparts revealed that F104A was less
structured than PTE (Figure S3). To assess the stability, differential scanning
calorimetry (DSC) was performed (Figure 2). Upon heating the sample from
\SIrange{0}{70}{\celsius}, \emph{p}FF-PTE exhibited two transitions
(T\textsubscript{m}1: 42.0 $\pm$ \SI{0.1}{\celsius}; T\textsubscript{m}2 : 48.6
$\pm$ \SI{0.2}{\celsius}; Table S1); this is consistent with our previous
studies.\cite{Baker2011b} This biphasic unfolding was also observed by Grimsley
\latin{et al.}  in a study of organophosphorus hydrolase, and was attributed to
the presence of a dimeric unfolded intermediate.\cite{Grimsley1997b} In
contrast, \emph{p}FF-F104A exhibited a single transition at 49.7 $\pm$
\SI{0.2}{\celsius}, which was higher than both \emph{p}FF-PTE values (by 7.7
and \SI{1.1}{\celsius}); Figure 2B, Table S1).  Remarkably, after heating,
\emph{p}FF-F104A retained the single T\textsubscript{m} of 49.2 $\pm$
\SI{0.1}{\celsius}, thus demonstrating regaining of structure after undergoing
thermal unfolding.  In the absence of \emph{p}FF, F104A demonstrated two
transitions similar to \emph{p}FF-PTE (Figure 2A), thus suggesting that
fluorination was critical for stability.  These data demonstrate the overall
thermodynamic stability of \emph{p}FF-F104A.

Moreover, the data further suggest that the unfolding model has been altered
from a 3-state to 2-state transition, and that the energy requirement to
attaining the unfolded intermediate was increased, thereby resulting in a more
cooperative transition. Mutations resulting in this particular transformation
have been observed for other proteins; for example, Fan \latin{et al.} showed
that removal of a C-terminal domain of the oligomeric \latin{E. coli} trigger
factor protein resulted in the transformation of an otherwise n-state unfolding
process to a distinct two-state unfolding process, indicative of pronounced
stabilization of the native structure by interdomain interactions.[18] We
propose that the \emph{p}FF-F104A mutation might also be stabilizing the native
structure of the overall protein (in effect the reverse of the mutation
observed by Fan \latin{et al.}).\cite{Fan2008} That is, \emph{p}FF-F104A unfolds
cooperatively in a single step, concurrent with its dissociation into monomeric
species. Although it was expected that the \emph{p}FF-F104A mutation would have an
effect on interdomain stability (such that neighboring residues would be
allowed to repack and energy would be minimized between monomers at the dimer
interface), the apparent stabilization of the native structure concluded from
the 3-state to 2-state transition transformation through these new interdomain
interactions were indeed unanticipated. Prior examples of this transformation
exist in cases involving subdomains of similar stabilities or where strong
coupling exists between subdomains.\cite{Tsytlonok2013}

\subsection{Enzymatic Kinetics of PTE}

% add table 
To assess function, we determined the Michaelis–Menten kinetics of
\emph{p}FF-F104A, \emph{p}FF-PTE, F104A, and PTE with paraoxon. At
\SI{25}{\celsius}, \emph{p}FF-PTE exhibited the highest activity
(k\textsubscript{cat}/K\textsubscript{M} = \SI{327000}{\per\Molar\per\second};
Table 1); \emph{p}FF-F104A was slightly lower
(k\textsubscript{cat}/K\textsubscript{M} = \SI{223000}{\per\Molar\per\second});
non-fluorinated PTE exhibited k\textsubscript{cat}/K\textsubscript{M} of
\SI{200000}{\per\Molar\per\second},similar to those both fluorinated proteins;
however, F104A was dramatically less active
(k\textsubscript{cat}/K\textsubscript{M} = \SI{23000}{\per\Molar\per\second}).
Thus, the fluorinated amino acids appear to be necessary for \emph{p}FF-F104A
activity.  Proteins were then incubated at \SIlist{35;45;55}{\celsius} for one
hour, and then cooled to room temperature to determine residual activity. A
decline in residual activity was observed for all proteins as a function of
elevated temperature. \emph{p}FF-F104A, which was designed to stabilize the
fluorinated protein, exhibited 50\% retention of activity at \SI{55}{\celsius}
(Figure 3A, Table 1). In contrast, at \SI{55}{\celsius}, \emph{p}FF-PTE and PTE
exhibited 24\% and 23\% initial activity, respectively; F104A exhibited a
significant loss in activity at or above \SI{45}{\celsius} (Figure 3A, Table
1). As \emph{p}FF-F104A retained substantial activity after elevated
temperatures, we then investigated the half-life of the activity. The parent
\emph{p}FF-PTE exhibited more than 50\% loss of activity after three days,
whereas the non-fluorinated PTE showed more than 50\% activity reduction after
seven days (Figure 3B, Table 2). Remarkably, \emph{p}FF-F104A retained 66\%
activity after seven days. The non-fluorinated counterpart F104A failed to
exhibit activity after one day. Together these data confirm that
\emph{p}FF-F104A is able to delay heat inactivation while maintaining function
after one week.

\subsection{Protein Design}

Although methods enabling the biosynthesis of artificial proteins bearing NCAAs
are abundant,\cite{Voloshchuk2009} tools to help further improve the overall
activity and stability are needed. Mutagenesis and evolutionary approaches have
been employed successfully to identify variants with enhanced function;
however, these rely heavily on testing or screening several to millions of
constructs.\cite{Voloshchuk2007b,Montclare2006b,Yoo2007} We demonstrate the use
of computational methods to identify a fluorinated protein variant that
exhibits superior heat stability and half-life. Notably, the \emph{p}FF-F104A
variant is only functional in the fluorinated form, thus validating
Rosetta-based design with \emph{p}FF. This provides another useful tool for
protein design and could be employed in conjunction with the above mentioned
approaches.

\printbibliography[heading=subbibliography]

\end{refsection}
